<html>
    <head>
        <meta charset="utf-8">
        
            <script src="lib/bindings/utils.js"></script>
            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/dist/vis-network.min.css" integrity="sha512-WgxfT5LWjfszlPHXRmBWHkV2eceiWTOBvrKCNbdgDYTHrT2AeLCGbF4sZlZw3UMN3WtL0tGUoIAKsu8mllg/XA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
            <script src="https://cdnjs.cloudflare.com/ajax/libs/vis-network/9.1.2/dist/vis-network.min.js" integrity="sha512-LnvoEWDFrqGHlHmDD2101OrLcbsfkrzoSpvtSQtxK3RMnRV0eOkhhBN2dXHKRrUU8p2DGRTk35n4O8nWSVe1mQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
            
        
<center>
<h1></h1>
</center>

<!-- <link rel="stylesheet" href="../node_modules/vis/dist/vis.min.css" type="text/css" />
<script type="text/javascript" src="../node_modules/vis/dist/vis.js"> </script>-->
        <link
          href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css"
          rel="stylesheet"
          integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6"
          crossorigin="anonymous"
        />
        <script
          src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js"
          integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf"
          crossorigin="anonymous"
        ></script>


        <center>
          <h1></h1>
        </center>
        <style type="text/css">

             #mynetwork {
                 width: 100%;
                 height: 750px;
                 background-color: #ffffff;
                 border: 1px solid lightgray;
                 position: relative;
                 float: left;
             }

             
             #loadingBar {
                 position:absolute;
                 top:0px;
                 left:0px;
                 width: 100%;
                 height: 750px;
                 background-color:rgba(200,200,200,0.8);
                 -webkit-transition: all 0.5s ease;
                 -moz-transition: all 0.5s ease;
                 -ms-transition: all 0.5s ease;
                 -o-transition: all 0.5s ease;
                 transition: all 0.5s ease;
                 opacity:1;
             }

             #bar {
                 position:absolute;
                 top:0px;
                 left:0px;
                 width:20px;
                 height:20px;
                 margin:auto auto auto auto;
                 border-radius:11px;
                 border:2px solid rgba(30,30,30,0.05);
                 background: rgb(0, 173, 246); /* Old browsers */
                 box-shadow: 2px 0px 4px rgba(0,0,0,0.4);
             }

             #border {
                 position:absolute;
                 top:10px;
                 left:10px;
                 width:500px;
                 height:23px;
                 margin:auto auto auto auto;
                 box-shadow: 0px 0px 4px rgba(0,0,0,0.2);
                 border-radius:10px;
             }

             #text {
                 position:absolute;
                 top:8px;
                 left:530px;
                 width:30px;
                 height:50px;
                 margin:auto auto auto auto;
                 font-size:22px;
                 color: #000000;
             }

             div.outerBorder {
                 position:relative;
                 top:400px;
                 width:600px;
                 height:44px;
                 margin:auto auto auto auto;
                 border:8px solid rgba(0,0,0,0.1);
                 background: rgb(252,252,252); /* Old browsers */
                 background: -moz-linear-gradient(top,  rgba(252,252,252,1) 0%, rgba(237,237,237,1) 100%); /* FF3.6+ */
                 background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,rgba(252,252,252,1)), color-stop(100%,rgba(237,237,237,1))); /* Chrome,Safari4+ */
                 background: -webkit-linear-gradient(top,  rgba(252,252,252,1) 0%,rgba(237,237,237,1) 100%); /* Chrome10+,Safari5.1+ */
                 background: -o-linear-gradient(top,  rgba(252,252,252,1) 0%,rgba(237,237,237,1) 100%); /* Opera 11.10+ */
                 background: -ms-linear-gradient(top,  rgba(252,252,252,1) 0%,rgba(237,237,237,1) 100%); /* IE10+ */
                 background: linear-gradient(to bottom,  rgba(252,252,252,1) 0%,rgba(237,237,237,1) 100%); /* W3C */
                 filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#fcfcfc', endColorstr='#ededed',GradientType=0 ); /* IE6-9 */
                 border-radius:72px;
                 box-shadow: 0px 0px 10px rgba(0,0,0,0.2);
             }
             

             

             
             /* position absolute is important and the container has to be relative or absolute as well. */
          div.popup {
                 position:absolute;
                 top:0px;
                 left:0px;
                 display:none;
                 background-color:#f5f4ed;
                 -moz-border-radius: 3px;
                 -webkit-border-radius: 3px;
                 border-radius: 3px;
                 border: 1px solid #808074;
                 box-shadow: 3px 3px 10px rgba(0, 0, 0, 0.2);
          }

          /* hide the original tooltip */
          .vis-tooltip {
            display:none;
          }
             
        </style>
    </head>


    <body>
        <div class="card" style="width: 100%">
            
            
            <div id="mynetwork" class="card-body"></div>
        </div>

        
            <div id="loadingBar">
              <div class="outerBorder">
                <div id="text">0%</div>
                <div id="border">
                  <div id="bar"></div>
                </div>
              </div>
            </div>
        
        

        <script type="text/javascript">

              // initialize global variables.
              var edges;
              var nodes;
              var allNodes;
              var allEdges;
              var nodeColors;
              var originalNodes;
              var network;
              var container;
              var options, data;
              var filter = {
                  item : '',
                  property : '',
                  value : []
              };

              

              

              // This method is responsible for drawing the graph, returns the drawn network
              function drawGraph() {
                  var container = document.getElementById('mynetwork');

                  

                  // parsing and collecting nodes and edges from the python
                  nodes = new vis.DataSet([{"color": "lightblue", "font": {"color": "black"}, "id": "Comparison of Measurable Residual Disease in Pediatric B-Lymphoblastic Leukemia Using Multiparametric Flow Cytometry and Next-Generation Sequencing", "label": "Comparison of Measurable Residual Disease in Pediatric B-Lymphoblastic Leukemia Using Multiparametric Flow Cytometry and Next-Generation Sequencing", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eTitle:\u003c/strong\u003e Comparison of Measurable Residual Disease in Pediatric B-Lymphoblastic Leukemia Using Multiparametric Flow Cytometry and Next-Generation Sequencing\u003c/p\u003e\n\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.3343/alm.2023.0412\"\u003ehttps://doi.org/10.3343/alm.2023.0412\u003c/a\u003e\u003c/p\u003e\n\n\u003cp\u003e\u003cstrong\u003eKey Findings:\u003c/strong\u003e\u003c/p\u003e\n\u003col\u003e\n  \u003cli\u003e\n    \u003cstrong\u003eHigh Concordance Between MFC-MRD and NGS-MRD Assays:\u003c/strong\u003e\n    In a cohort of 139 samples from 54 pediatric B-ALL patients, the concordance rate between MFC-MRD and NGS-MRD was 79.9%, showing substantial agreement in measurable residual disease detection.\n  \u003c/li\u003e\n  \u003cli\u003e\n    \u003cstrong\u003eNGS-MRD Detected More MRD-Positive Cases Than MFC-MRD:\u003c/strong\u003e\n    NGS-MRD identified MRD in 39.6% of samples compared to 23.7% by MFC-MRD, indicating that NGS-MRD has higher sensitivity in detecting low levels of disease.\n  \u003c/li\u003e\n  \u003cli\u003e\n    \u003cstrong\u003eStatistically Significant Correlation Between Methods:\u003c/strong\u003e\n    MRD values from both assays showed strong correlation based on total nucleated cells (r = 0.736, \u003cem\u003eP\u003c/em\u003e \u0026lt; 0.001) and B-cell percentages (r = 0.748, \u003cem\u003eP\u003c/em\u003e \u0026lt; 0.001), supporting their complementary quantitative validity.\n  \u003c/li\u003e\n  \u003cli\u003e\n    \u003cstrong\u003eNGS-MRD Identified MRD Below MFC Detection Limits:\u003c/strong\u003e\n    In 14.3% of discordant samples (MFC\u2212NGS+MRD), NGS-MRD detected MRD below the limit of detection for MFC-MRD, explaining the discrepancy.\n  \u003c/li\u003e\n  \u003cli\u003e\n    \u003cstrong\u003ePost-Therapy Immunophenotypic Changes May Impact MFC-MRD:\u003c/strong\u003e\n    Some MFC\u2212NGS+MRD samples fell within MFC detection range but were missed, likely due to phenotypic shifts in leukemic cells that MFC could not recognize post-treatment.\n  \u003c/li\u003e\n  \u003cli\u003e\n    \u003cstrong\u003eNGS-MRD Reports Significantly Higher MRD Quantification:\u003c/strong\u003e\n    Among MRD-positive samples, NGS-MRD reported significantly higher values than MFC-MRD, with a median difference of 0.120% (\u003cem\u003eP\u003c/em\u003e \u0026lt; 0.001).\n  \u003c/li\u003e\n  \u003cli\u003e\n    \u003cstrong\u003eHematogone Percentage Influences Assay Concordance:\u003c/strong\u003e\n    The concordant cases had significantly higher hematogone percentages than discordant ones. A 0.0555% hematogone threshold predicted concordance with AUC = 0.637 (\u003cem\u003eP\u003c/em\u003e \u0026lt; 0.001).\n  \u003c/li\u003e\n  \u003cli\u003e\n    \u003cstrong\u003eHemodilution Affects MFC-MRD More Than NGS-MRD:\u003c/strong\u003e\n    Sample quality influenced MFC-MRD more than NGS-MRD. Hematogone percentage was used to assess hemodilution impact, reinforcing the robustness of NGS.\n  \u003c/li\u003e\n  \u003cli\u003e\n    \u003cstrong\u003eMFC-MRD Remains Essential for Patients Without IGH Clones:\u003c/strong\u003e\n    In cases where IGH clonality is not detected at diagnosis, NGS-MRD cannot be applied, making MFC-MRD the only viable method for residual disease monitoring.\n  \u003c/li\u003e\n  \u003cli\u003e\n    \u003cstrong\u003eNGS-MRD May Provide Earlier MRD Detection Than MFC-MRD:\u003c/strong\u003e\n    In several cases, MRD positivity appeared in NGS-MRD results before being detected by MFC-MRD, suggesting NGS-MRD could enable earlier intervention.\n  \u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD Detection Methods", "label": "MRD Detection Methods", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Assay Concordance", "label": "Assay Concordance", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sensitivity of NGS-MRD", "label": "Sensitivity of NGS-MRD", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Discordant MRD Results", "label": "Discordant MRD Results", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Quantitative Correlation Metrics", "label": "Quantitative Correlation Metrics", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Immunophenotypic Shifts", "label": "Immunophenotypic Shifts", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Hemodilution Effects", "label": "Hemodilution Effects", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Hematogone Analysis", "label": "Hematogone Analysis", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical Applicability of MRD Assays", "label": "Clinical Applicability of MRD Assays", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Early Detection of Relapse", "label": "Early Detection of Relapse", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "The effectiveness of blinatumomab in clearing measurable residual disease in pediatric B-cell acute lymphoblastic leukemia patients detected by next-generation sequencing", "label": "The effectiveness of blinatumomab in clearing measurable residual disease in pediatric B-cell acute lymphoblastic leukemia patients detected by next-generation sequencing", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e The effectiveness of blinatumomab in clearing measurable residual disease in pediatric B-cell acute lymphoblastic leukemia patients detected by next-generation sequencing\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1002/cam4.6771\"\u003e10.1002/cam4.6771\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eBlinatumomab successfully cleared next-generation sequencing (NGS)-detectable measurable residual disease (MRD) in 13 out of 19 (68%) pediatric B-ALL patients who were already negative for MRD by multiparameter flow cytometry (MFC), demonstrating the drug\u2019s ability to achieve deeper molecular remission.\u003c/li\u003e\n\n  \u003cli\u003eAmong the patients who did not achieve NGS-MRD clearance after blinatumomab, three experienced relapse within 1.8 months, indicating a potential link between persistent NGS-MRD and early relapse.\u003c/li\u003e\n\n  \u003cli\u003ePatients harboring persistent IGH clones after blinatumomab treatment were more likely to relapse than those with persistent IGK or IGKDE clones, suggesting IGH as a more reliable predictor of disease recurrence.\u003c/li\u003e\n\n  \u003cli\u003eOverall, the clearance rates of specific clonal rearrangements after blinatumomab were: IGH (75%), IGK (67%), IGKDE (83%), IGH-DJ (100%), and IGL (33%), highlighting variable sensitivity of different immunoglobulin gene types to the therapy.\u003c/li\u003e\n\n  \u003cli\u003eDespite having hematologic complete remission and MFC-MRD negativity, all patients had detectable MRD by NGS before blinatumomab, supporting the superior sensitivity of NGS (down to 10\u207b\u2076) compared to MFC (10\u207b\u2074).\u003c/li\u003e\n\n  \u003cli\u003eOne-year overall survival (OS) and event-free survival (EFS) rates post-blinatumomab were 94.7% and 78.9%, respectively, underscoring the treatment\u2019s favorable short-term outcomes.\u003c/li\u003e\n\n  \u003cli\u003eSix patients underwent hematopoietic stem cell transplantation (HSCT) following blinatumomab; all remained in remission at the last follow-up, suggesting a viable treatment sequence for high-risk or relapsed patients.\u003c/li\u003e\n\n  \u003cli\u003eThis is the first study to confirm blinatumomab\u0027s efficacy in eliminating extremely low levels of MRD detectable only via NGS in pediatric patients, highlighting its potential role in deep molecular remission strategies.\u003c/li\u003e\n\n  \u003cli\u003eThe study supports the prognostic utility of NGS-based MRD monitoring over traditional methods, aligning with emerging clinical guidelines that favor NGS for relapse risk assessment in B-ALL.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Blinatumomab efficacy in pediatric B-ALL", "label": "Blinatumomab efficacy in pediatric B-ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Measurable residual disease (MRD) monitoring", "label": "Measurable residual disease (MRD) monitoring", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Next-generation sequencing (NGS)-based MRD detection", "label": "Next-generation sequencing (NGS)-based MRD detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison between NGS and MFC sensitivity", "label": "Comparison between NGS and MFC sensitivity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic significance of clonal immunoglobulin gene rearrangements (IGH, IGK, IGL)", "label": "Prognostic significance of clonal immunoglobulin gene rearrangements (IGH, IGK, IGL)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Early relapse prediction and risk stratification", "label": "Early relapse prediction and risk stratification", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Short-term survival outcomes (OS/EFS) post-treatment", "label": "Short-term survival outcomes (OS/EFS) post-treatment", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Role of blinatumomab in MRD-directed therapy", "label": "Role of blinatumomab in MRD-directed therapy", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Integration of NGS-MRD into clinical decision-making", "label": "Integration of NGS-MRD into clinical decision-making", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Hematopoietic stem cell transplantation (HSCT) after blinatumomab", "label": "Hematopoietic stem cell transplantation (HSCT) after blinatumomab", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Multi-target measurable residual disease assessed by error-corrected sequencing in patients with acute myeloid leukemia: An ALFA study", "label": "Multi-target measurable residual disease assessed by error-corrected sequencing in patients with acute myeloid leukemia: An ALFA study", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Multi-target measurable residual disease assessed by error-corrected sequencing in patients with acute myeloid leukemia: An ALFA study\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1038/s41408-024-01078-8\"\u003e10.1038/s41408-024-01078-8\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003ePersistence of non-DTA mutations (i.e., not in DNMT3A, TET2, ASXL1) detected via NGS-MRD in complete remission was significantly associated with worse relapse-free survival (RFS) and overall survival (OS), with hazard ratios (HR) of 2.23 and 2.26, respectively.\u003c/li\u003e\n\n  \u003cli\u003ePersistence of DTA mutations alone was also associated with poorer overall survival (HR = 2.16), indicating these mutations should not be routinely excluded from MRD analysis.\u003c/li\u003e\n\n  \u003cli\u003ePatients with two or more persistent mutations had a significantly higher cumulative incidence of relapse (CIR, HR = 3.71), and worse RFS (HR = 3.36) and OS (HR = 3.81), compared to those with zero or one mutation detected in remission.\u003c/li\u003e\n\n  \u003cli\u003eDetection of only one mutation in remission did not significantly impact prognosis, suggesting limited prognostic value for single persisting mutations alone.\u003c/li\u003e\n\n  \u003cli\u003eIn the subset of patients with NPM1 mutations, both NPM1-MRD positivity and multi-target NGS-MRD positivity were independently associated with increased relapse risk (HR = 4.16 and HR = 3.37, respectively), indicating complementary prognostic value.\u003c/li\u003e\n\n  \u003cli\u003eWT1-MRD positivity was a stronger independent predictor of poor RFS and OS than NGS-MRD in patients with low WT1 expression at baseline, suggesting WT1 monitoring may outperform NGS-MRD in some settings.\u003c/li\u003e\n\n  \u003cli\u003eThe prognostic utility of NGS-MRD persisted regardless of whether patients underwent allogeneic stem cell transplantation (allo-SCT), indicating that NGS-MRD status should not alone dictate transplant decisions.\u003c/li\u003e\n\n  \u003cli\u003eThe study confirms that multi-target error-corrected NGS-MRD is a powerful prognostic tool in AML, with mutation number and type both influencing outcomes, and supports its potential integration into broader MRD assessment strategies.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Measurable Residual Disease (MRD) in Acute Myeloid Leukemia (AML)", "label": "Measurable Residual Disease (MRD) in Acute Myeloid Leukemia (AML)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Error-corrected Next-Generation Sequencing (NGS-MRD)", "label": "Error-corrected Next-Generation Sequencing (NGS-MRD)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "DNMT3A, TET2, and ASXL1 (DTA) mutations", "label": "DNMT3A, TET2, and ASXL1 (DTA) mutations", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic impact of persistent mutations in remission", "label": "Prognostic impact of persistent mutations in remission", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison between NGS-MRD, WT1-MRD, and NPM1-MRD", "label": "Comparison between NGS-MRD, WT1-MRD, and NPM1-MRD", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Role of mutation burden (number of mutations) in prognosis", "label": "Role of mutation burden (number of mutations) in prognosis", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Influence of NGS-MRD on allogeneic stem cell transplantation decisions", "label": "Influence of NGS-MRD on allogeneic stem cell transplantation decisions", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Integration of molecular MRD strategies in AML clinical management", "label": "Integration of molecular MRD strategies in AML clinical management", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Quantification of Measurable Residual Disease Detection by Next-Generation Sequencing-Based Clonality Testing in B-Cell and Plasma Cell Neoplasms", "label": "Quantification of Measurable Residual Disease Detection by Next-Generation Sequencing-Based Clonality Testing in B-Cell and Plasma Cell Neoplasms", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cdiv\u003e\n  \u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Quantification of Measurable Residual Disease Detection by Next-Generation Sequencing-Based Clonality Testing in B-Cell and Plasma Cell Neoplasms\u003c/p\u003e\n  \u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1016/j.jmoldx.2023.11.009\"\u003e10.1016/j.jmoldx.2023.11.009\u003c/a\u003e\u003c/p\u003e\n\n  \u003col\u003e\n    \u003cli\u003eNGS-based MRD assay demonstrated 91% concordance with flow cytometry (FC) in MRD detection across 519 successfully sequenced samples, validating its reliability in B-cell and plasma cell neoplasms.\u003c/li\u003e\n    \u003cli\u003eNormalized MRD quantification using a spike-in calibrator showed high correlation with FC quantitation (R = 0.67 overall), with particularly strong correlation in CLL and B-ALL cases (R = 0.85 for both).\u003c/li\u003e\n    \u003cli\u003eFor plasma cell neoplasms, the correlation between NGS and FC quantitation was weaker (R = 0.57), likely due to underestimation by FC and biologic or technical biases affecting both methods.\u003c/li\u003e\n    \u003cli\u003eNGS assay showed robust sensitivity, detecting disease down to at least 1 \u00d7 10\u207b\u2075 in 99.6% of runs and 1 \u00d7 10\u207b\u2076 in 72% of runs using standardized control samples.\u003c/li\u003e\n    \u003cli\u003eThe spike-in calibrator did not significantly impact sequencing efficiency or clonal read proportions, supporting its use for normalization without compromising assay performance.\u003c/li\u003e\n    \u003cli\u003eIntra- and inter-assay reproducibility of MRD quantification was high, with low variability (SD: 0.003%\u20130.007%) in replicate testing of six MRD-positive samples across separate runs.\u003c/li\u003e\n    \u003cli\u003eInterlaboratory testing of 35 samples between MSKCC and Invivoscribe showed 100% concordance in MRD status and near-perfect correlation in MRD quantitation (R = 0.97).\u003c/li\u003e\n    \u003cli\u003eAmong discordant cases, 50% of FC-undetectable/NGS-positive cases later relapsed, compared to 35% of FC-positive/NGS-undetectable cases, suggesting NGS may have higher clinical sensitivity.\u003c/li\u003e\n    \u003cli\u003eSequencing failure occurred in only 4.9% of samples, all due to very low cellularity or B/plasma cell content, with corresponding FC results also being suboptimal.\u003c/li\u003e\n    \u003cli\u003eNGS-based MRD monitoring is particularly advantageous for detecting disease in samples with non-viable or disrupted plasma cells, due to the stability of DNA over time.\u003c/li\u003e\n  \u003c/ol\u003e\n\u003c/div\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD Detection Technologies", "label": "MRD Detection Technologies", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparative Accuracy of MFC-MRD vs. NGS-MRD", "label": "Comparative Accuracy of MFC-MRD vs. NGS-MRD", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sensitivity and Detection Limits", "label": "Sensitivity and Detection Limits", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Discordance Analysis Between Assays", "label": "Discordance Analysis Between Assays", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Quantitative Correlation and Statistical Agreement", "label": "Quantitative Correlation and Statistical Agreement", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Effect of Immunophenotypic Changes Post-Therapy", "label": "Effect of Immunophenotypic Changes Post-Therapy", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Impact of Hemodilution and Sample Quality", "label": "Impact of Hemodilution and Sample Quality", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Use of Hematogones as a Marker of Sample Integrity", "label": "Use of Hematogones as a Marker of Sample Integrity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical Scenarios Favoring One Assay Over the Other", "label": "Clinical Scenarios Favoring One Assay Over the Other", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Implications for Early Detection and Monitoring Strategy", "label": "Implications for Early Detection and Monitoring Strategy", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": " Corrigendum to \u201cMeasurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways?", "label": " Corrigendum to \u201cMeasurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways?", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Corrigendum to \u201cMeasurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways?\u201d\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1016/j.blre.2024.101226\"\u003e10.1016/j.blre.2024.101226\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eIn 22 studies of multiparameter flow cytometry (MPFC)-based MRD testing in acute myeloid leukemia (AML), the MRD cut-off thresholds ranged from 0.004% to 0.2%, indicating significant variability in sensitivity across assays.\u003c/li\u003e\n  \u003cli\u003eAmong 14 studies using PCR-based MRD testing in AML, cut-off thresholds ranged from 0.0001% to 1%, with a median value of 0.01%, reflecting higher assay sensitivity compared to flow cytometry.\u003c/li\u003e\n  \u003cli\u003eIn studies of MPFC-based MRD testing for AML, median MRD cut-off was reported as 0.1%, further confirming it as a common standard for flow cytometry-based detection.\u003c/li\u003e\n  \u003cli\u003eThe corrected MRD cut-off threshold for RUNX1::RUNX1T1-positive AML using PCR is a 3-log reduction in transcript levels.\u003c/li\u003e\n  \u003cli\u003eNPM1-mutated AML is assessed using PCR with either a 4-log reduction threshold or an absolute transcript level of 0/102 ABL1 copies, suggesting different practices across labs.\u003c/li\u003e\n  \u003cli\u003eFor AML in older persons, MPFC-based MRD threshold is 0.01%, which is relatively stringent and indicates detection of very low disease burden.\u003c/li\u003e\n  \u003cli\u003eFor core binding factor (CBF) AML, quantitative PCR (qPCR) MRD testing uses a 3-log reduction threshold.\u003c/li\u003e\n  \u003cli\u003eIn BCR::ABL1-positive acute lymphoblastic leukemia (ALL), MPFC-based MRD thresholds ranged from 0.01% to 0.04%, emphasizing sensitivity in detecting residual disease in high-risk ALL subtypes.\u003c/li\u003e\n  \u003cli\u003eA study using next-generation sequencing (NGS)-based MRD testing reported a limit of detection (LoD) as low as 0.0001%, showcasing the superior sensitivity of NGS-based approaches.\u003c/li\u003e\n  \u003cli\u003eIn 14 studies of MPFC-based MRD testing in various hematologic cancers, thresholds ranged from 0.001% to 1% (median = 0.1%), underscoring the broad range of sensitivity and lack of standardization.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD (Measurable Residual Disease) testing thresholds", "label": "MRD (Measurable Residual Disease) testing thresholds", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MPFC (Multiparameter Flow Cytometry) assay standards", "label": "MPFC (Multiparameter Flow Cytometry) assay standards", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "PCR-based MRD assay performance", "label": "PCR-based MRD assay performance", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "NGS (Next-Generation Sequencing) in MRD detection", "label": "NGS (Next-Generation Sequencing) in MRD detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "AML (Acute Myeloid Leukemia) subtypes and MRD evaluation", "label": "AML (Acute Myeloid Leukemia) subtypes and MRD evaluation", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "ALL (Acute Lymphoblastic Leukemia) MRD sensitivity", "label": "ALL (Acute Lymphoblastic Leukemia) MRD sensitivity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Methodological variability across studies", "label": "Methodological variability across studies", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Assay sensitivity and limit of detection (LoD)", "label": "Assay sensitivity and limit of detection (LoD)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Standardization of MRD thresholds in hematological cancers", "label": "Standardization of MRD thresholds in hematological cancers", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Measurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways", "label": "Measurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Measurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways?\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1016/j.blre.2024.101226\"\u003e10.1016/j.blre.2024.101226\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eMRD-testing has prognostic value in haematological cancers, but its accuracy at the individual level is limited, with sensitivities often below 50% and considerable rates of false negatives and false positives.\u003c/li\u003e\n  \n  \u003cli\u003eAcross 42 studies in AML, patients with a positive MRD test after consolidation therapy or before transplant showed a substantially higher relapse risk compared to those with negative MRD tests (median 66% vs. 22% relapse risk over 3\u20135 years).\u003c/li\u003e\n  \n  \u003cli\u003eMRD-testing in ALL also shows prognostic significance, especially after induction and before or after transplant. However, many patients with negative MRD tests still relapsed, indicating its limited predictive reliability.\u003c/li\u003e\n  \n  \u003cli\u003eMRD testing thresholds and assay sensitivities vary widely across studies\u2014MPFC assays typically use thresholds between 0.01\u20130.2%, while PCR-based tests use 0.0001\u201310%, complicating cross-study comparisons and clinical interpretations.\u003c/li\u003e\n  \n  \u003cli\u003eDespite technological advancements such as NGS and ddPCR improving assay sensitivity, discordances remain high between MRD techniques (e.g., only 69% concordance between MPFC and NGS in AML), which can impact clinical decisions.\u003c/li\u003e\n  \n  \u003cli\u003eMRD is frequently used as a regulatory endpoint, such as for accelerated approval of therapies in plasma cell myeloma, yet there is limited evidence from randomized trials supporting MRD-guided treatment decisions that alter long-term outcomes.\u003c/li\u003e\n  \n  \u003cli\u003eUse of MRD to guide therapy\u2014such as escalation or de-escalation of treatment intensity or transplant decisions\u2014is recommended in guidelines, but these recommendations lack support from randomized controlled trials and are often based on observational data prone to bias.\u003c/li\u003e\n  \n  \u003cli\u003eSome studies show that adding MRD results to conventional clinical variables (e.g., cytogenetics, mutations) yields minimal improvement in relapse prediction models, with only a 0.01 increase in the C-statistic in some cases.\u003c/li\u003e\n  \n  \u003cli\u003eIn pediatric and adult ALL, MRD-guided therapy decisions (e.g., recommending transplant based on MRD status) may influence treatment paths but are not reliably associated with improved survival or reduced relapse rates in the absence of strong supporting data.\u003c/li\u003e\n  \n  \u003cli\u003eThe authors recommend caution in using MRD as a sole determinant of treatment decisions and emphasize the need for multi-dimensional prognostic models that integrate biological, clinical, and dynamic treatment response data.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Accuracy and limitations of MRD-testing", "label": "Accuracy and limitations of MRD-testing", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD-testing techniques (MPFC, PCR, dPCR, NGS)", "label": "MRD-testing techniques (MPFC, PCR, dPCR, NGS)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic value of MRD in AML and ALL", "label": "Prognostic value of MRD in AML and ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sampling challenges and assay sensitivity", "label": "Sampling challenges and assay sensitivity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Concordance between MRD-testing methods", "label": "Concordance between MRD-testing methods", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical utility and thresholds for MRD interpretation", "label": "Clinical utility and thresholds for MRD interpretation", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD-guided therapy and lack of RCT-based evidence", "label": "MRD-guided therapy and lack of RCT-based evidence", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Role of MRD in regulatory approval and guidelines", "label": "Role of MRD in regulatory approval and guidelines", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Integration of MRD with multi-dimensional risk models", "label": "Integration of MRD with multi-dimensional risk models", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Recommendations for cautious use of MRD in clinical decisions", "label": "Recommendations for cautious use of MRD in clinical decisions", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Comparison of next-generation sequencing (NGS) and next-generation flow (NGF) for minimal residual disease (MRD) assessment in multiple myeloma", "label": "Comparison of next-generation sequencing (NGS) and next-generation flow (NGF) for minimal residual disease (MRD) assessment in multiple myeloma", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Comparison of next-generation sequencing (NGS) and next-generation flow (NGF) for minimal residual disease (MRD) assessment in multiple myeloma\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1038/s41408-020-00377-0\"\u003ehttps://doi.org/10.1038/s41408-020-00377-0\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eNGS and NGF showed a high correlation in MRD detection (R\u00b2 = 0.905), with both methods identifying undetectable MRD in a substantial proportion of patients post-transplantation.\u003c/li\u003e\n  \u003cli\u003ePatients with undetectable MRD by either method demonstrated significantly higher 3-year progression-free survival (PFS) (NGS: 88.7%, NGF: 91.4%) and overall survival (OS) (NGS: 96.2%, NGF: 96.6%) compared to MRD-positive patients (p \u0026lt; 0.01).\u003c/li\u003e\n  \u003cli\u003eNGF achieved MRD detection in all cases, while NGS was limited by sample quality and DNA concentration, requiring ethanol precipitation in some cases, which may have compromised sensitivity.\u003c/li\u003e\n  \u003cli\u003eFifteen cases were discordant between NGS and NGF results, mostly at low MRD levels (\u0026lt;10\u003csup\u003e\u22125\u003c/sup\u003e), yet only 3 of these patients experienced relapse, suggesting clinical relevance even at low detection levels.\u003c/li\u003e\n  \u003cli\u003eMRD status was a more powerful predictor of outcome than traditional response categories or cytogenetic risk\u2014MRD-negative patients in VGPR had similar PFS as those in CR/sCR.\u003c/li\u003e\n  \u003cli\u003eAmong MRD-positive patients, those with high-risk cytogenetics had significantly worse outcomes (PFS: 15.4%\u201328.6%) than those with standard-risk features (PFS: 61.8%\u201367.1%).\u003c/li\u003e\n  \u003cli\u003eMultivariate analysis confirmed MRD negativity by NGF and low R-ISS stage as independent predictors of prolonged PFS and OS, with hazard ratios as low as 0.20 for PFS and 0.21 for OS (p \u0026lt; 0.05).\u003c/li\u003e\n  \u003cli\u003eNGS offered a more user-friendly and scalable workflow but was more affected by preanalytical conditions; NGF, while requiring more technical expertise, provided faster results and higher cell input consistency.\u003c/li\u003e\n  \u003cli\u003eThe study supports using either NGS (LymphoTrack\u00ae) or NGF as valid MRD assessment tools, with both providing prognostically significant information, reinforcing MRD as a key endpoint in multiple myeloma trials.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Minimal Residual Disease (MRD) detection in multiple myeloma", "label": "Minimal Residual Disease (MRD) detection in multiple myeloma", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison between Next-Generation Sequencing (NGS) and Next-Generation Flow (NGF)", "label": "Comparison between Next-Generation Sequencing (NGS) and Next-Generation Flow (NGF)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic significance of MRD status on progression-free survival (PFS) and overall survival (OS)", "label": "Prognostic significance of MRD status on progression-free survival (PFS) and overall survival (OS)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Technical and clinical performance characteristics of NGS vs. NGF", "label": "Technical and clinical performance characteristics of NGS vs. NGF", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Influence of cytogenetics and response categories on MRD outcomes", "label": "Influence of cytogenetics and response categories on MRD outcomes", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Multivariate modeling of survival predictors", "label": "Multivariate modeling of survival predictors", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Applicability and limitations of MRD assessment techniques in clinical trials", "label": "Applicability and limitations of MRD assessment techniques in clinical trials", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Next-Generation Sequencing in Acute Lymphoblastic Leukemia", "label": "Next-Generation Sequencing in Acute Lymphoblastic Leukemia", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cdiv\u003e\n\u003cb\u003eArticle Title:\u003c/b\u003e Next-Generation Sequencing in Acute Lymphoblastic Leukemia\u003cbr\u003e\n\u003cb\u003eDOI:\u003c/b\u003e \u003ca href=\"https://doi.org/10.3390/ijms20122929\" target=\"_blank\"\u003e10.3390/ijms20122929\u003c/a\u003e\n\u003c/div\u003e\n\n\u003cdiv\u003e\n\u003col\u003e\n  \u003cli\u003eNGS has enabled the identification of new B-ALL subtypes such as DUX4/ERG and ETV6-RUNX1-like, with implications for diagnosis and treatment stratification.\u003c/li\u003e\n  \u003cli\u003eDUX4-rearranged B-ALL is associated with ERG deregulation and favorable prognosis; it is more frequent in adolescents and young adults, and mouse models confirm its leukemogenic potential.\u003c/li\u003e\n  \u003cli\u003eMEF2D and ZNF384 fusion genes define distinct subtypes of B-ALL with poor prognosis and unique therapeutic vulnerabilities, such as sensitivity to HDAC inhibitors or JAK-STAT pathway inhibitors.\u003c/li\u003e\n  \u003cli\u003eTCF3-HLF-positive B-ALL shows stem cell-like characteristics and is sensitive to venetoclax, a BCL2 inhibitor, suggesting targeted therapy potential for this high-risk group.\u003c/li\u003e\n  \u003cli\u003eMLL-rearranged ALL in infants presents with low mutational burden but frequent RAS pathway mutations, indicating MLL fusion proteins as primary oncogenic drivers and therapeutic targets.\u003c/li\u003e\n  \u003cli\u003eB-NEG ALL cases lacking known fusion transcripts harbor frequent RAS/RTK and JAK/STAT mutations, which are age-dependent and potentially druggable.\u003c/li\u003e\n  \u003cli\u003eEPOR rearrangements in Ph-like ALL result in JAK-STAT pathway activation and are responsive to JAK inhibitors, highlighting a new molecular target in high-risk patients.\u003c/li\u003e\n  \u003cli\u003eIn hyperdiploid B-ALL, chromosomal gains appear to be early driver events, while hypodiploid ALL is associated with specific mutations such as TP53 and FLT3, influencing prognosis and relapse patterns.\u003c/li\u003e\n  \u003cli\u003eInherited predisposition to ALL is rare but includes germline mutations in ETV6 and PAX5, with WES studies identifying these in familial cases.\u003c/li\u003e\n  \u003cli\u003eT-ALL cases show frequent mutations in NOTCH1, FBXW7, and JAK3, and novel mutations in epigenetic regulators and transcription factors, indicating potential targets for precision therapy.\u003c/li\u003e\n  \u003cli\u003eFunctional non-coding mutations in regulatory elements of TAL1, LMO1, and LMO2 contribute to aberrant gene expression in T-ALL, particularly in early T-cell progenitor cases.\u003c/li\u003e\n  \u003cli\u003eNovel gene fusions such as ZBTB16-ABL1 with PTK activity have been identified in T-ALL, which are potentially targetable with tyrosine kinase inhibitors.\u003c/li\u003e\n  \u003cli\u003eETP-ALL, a high-risk T-ALL subtype, shows mutations in DNMT3A, FLT3, and NOTCH1, and whole-genome sequencing can aid in MRD monitoring and relapse prediction.\u003c/li\u003e\n  \u003cli\u003eNGS-based MRD monitoring using IG/TCR rearrangements has demonstrated superior sensitivity and prognostic value compared to conventional PCR, with applications in both diagnosis and relapse prediction.\u003c/li\u003e\n  \u003cli\u003eThe ClonoSEQ assay, based on NGS, has been FDA-approved for MRD detection in ALL, showing greater depth and sensitivity for residual disease monitoring than traditional methods.\u003c/li\u003e\n  \u003cli\u003eThird-generation nanopore sequencing technologies such as MinION allow rapid, low-cost detection of mutations and fusion genes (e.g., BCR-ABL1) and are beginning to show promise in clinical applications despite higher error rates.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "B-cell Acute Lymphoblastic Leukemia (B-ALL) subtypes and gene fusions", "label": "B-cell Acute Lymphoblastic Leukemia (B-ALL) subtypes and gene fusions", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "T-cell Acute Lymphoblastic Leukemia (T-ALL) genetic alterations", "label": "T-cell Acute Lymphoblastic Leukemia (T-ALL) genetic alterations", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Next-Generation Sequencing (NGS) technologies and methodologies", "label": "Next-Generation Sequencing (NGS) technologies and methodologies", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Minimal Residual Disease (MRD) monitoring using NGS", "label": "Minimal Residual Disease (MRD) monitoring using NGS", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Third-generation sequencing (e.g., nanopore sequencing) in leukemia", "label": "Third-generation sequencing (e.g., nanopore sequencing) in leukemia", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Risk stratification and targeted therapy in ALL", "label": "Risk stratification and targeted therapy in ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Inherited genetic predispositions to ALL", "label": "Inherited genetic predispositions to ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Molecular classification and prognostic biomarkers in ALL", "label": "Molecular classification and prognostic biomarkers in ALL", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Is Next-Generation Sequencing the Way to Go for Residual Disease Monitoring in Acute Lymphoblastic Leukemia?", "label": "Is Next-Generation Sequencing the Way to Go for Residual Disease Monitoring in Acute Lymphoblastic Leukemia?", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cb\u003eArticle Title:\u003c/b\u003e Is Next-Generation Sequencing the Way to Go for Residual Disease Monitoring in Acute Lymphoblastic Leukemia?\u003c/p\u003e\n\u003cp\u003e\u003cb\u003eDOI:\u003c/b\u003e \u003ca href=\"https://doi.org/10.1007/s40291-017-0277-9\"\u003e10.1007/s40291-017-0277-9\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eMinimal residual disease (MRD) is the most important independent prognostic factor in acute lymphoblastic leukemia (ALL), influencing treatment decisions and long-term outcomes in both pediatric and adult patients.\u003c/li\u003e\n  \u003cli\u003eReal-time quantitative PCR (RQ-PCR) using allele-specific primers for clonal immunoglobulin (Ig) and T-cell receptor (TR) gene rearrangements is currently the gold standard for MRD detection due to its high sensitivity and specificity.\u003c/li\u003e\n  \u003cli\u003eNext-generation sequencing (NGS) of Ig/TR loci allows for deeper sequencing and potentially higher sensitivity (up to 10\u003csup\u003e-7\u003c/sup\u003e) compared to RQ-PCR, enabling earlier and more precise MRD detection.\u003c/li\u003e\n  \u003cli\u003eNGS enables simultaneous detection and quantification of MRD without patient-specific primer design, making it more efficient and scalable than RQ-PCR.\u003c/li\u003e\n  \u003cli\u003eIn head-to-head comparisons, NGS demonstrated better specificity for relapse prediction than RQ-PCR, with fewer false positives, especially post stem-cell transplantation (SCT).\u003c/li\u003e\n  \u003cli\u003eNGS-based MRD monitoring can be effectively performed on peripheral blood, potentially reducing the need for invasive bone marrow aspirations.\u003c/li\u003e\n  \u003cli\u003eClonal evolution and the presence of multiple evolved Ig/TR rearrangements during treatment can be tracked using NGS, offering insights into leukemic heterogeneity and evolution.\u003c/li\u003e\n  \u003cli\u003eNGS-based assays also provide information on the physiological B- and T-cell repertoire, where reduced diversity post-treatment has been associated with higher relapse risk.\u003c/li\u003e\n  \u003cli\u003eNGS faces challenges such as contamination (e.g., from post-PCR processing or run-to-run carryover) and requires stringent protocols including unique barcoding to mitigate this risk.\u003c/li\u003e\n  \u003cli\u003eAccurate quantification of MRD using NGS requires correction for sequencing depth and input DNA; methods like spike-in standards and synthetic polyclonal backgrounds are used but not yet standardized.\u003c/li\u003e\n  \u003cli\u003eThe EuroClonality-NGS Consortium is actively working on standardizing the entire NGS MRD workflow including primer design, data analysis, and interpretation guidelines to ensure inter-lab reproducibility.\u003c/li\u003e\n  \u003cli\u003eCost-effectiveness of NGS compared to RQ-PCR depends on high levels of sample multiplexing per sequencing run to maintain feasibility in clinical diagnostics.\u003c/li\u003e\n  \u003cli\u003eBioinformatic tools like IMGT/V-QUEST, Vidjil, and ARResT/Interrogate are essential for analyzing and interpreting NGS data, with challenges still remaining in clonotype assignment and error filtering.\u003c/li\u003e\n  \u003cli\u003eNGS has strong potential to become the new gold standard for MRD monitoring, but widespread clinical adoption hinges on overcoming current limitations in standardization, contamination control, and cost.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic role of Minimal Residual Disease (MRD) in ALL", "label": "Prognostic role of Minimal Residual Disease (MRD) in ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Traditional MRD detection methods (e.g., RQ-PCR, flow cytometry)", "label": "Traditional MRD detection methods (e.g., RQ-PCR, flow cytometry)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Advantages and limitations of Next-Generation Sequencing (NGS) for MRD", "label": "Advantages and limitations of Next-Generation Sequencing (NGS) for MRD", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparative sensitivity and specificity of NGS vs. RQ-PCR", "label": "Comparative sensitivity and specificity of NGS vs. RQ-PCR", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "NGS-based marker identification and clonal tracking", "label": "NGS-based marker identification and clonal tracking", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "B- and T-cell repertoire analysis using NGS", "label": "B- and T-cell repertoire analysis using NGS", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Technical and bioinformatic challenges in NGS (e.g., contamination, quantification)", "label": "Technical and bioinformatic challenges in NGS (e.g., contamination, quantification)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Standardization efforts (EuroClonality-NGS Consortium)", "label": "Standardization efforts (EuroClonality-NGS Consortium)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Cost and scalability considerations for clinical implementation", "label": "Cost and scalability considerations for clinical implementation", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Future perspectives and clinical integration of NGS in MRD monitoring", "label": "Future perspectives and clinical integration of NGS in MRD monitoring", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Simple deep sequencing-based post-remission MRD surveillance predicts clinical relapse in B-ALL", "label": "Simple deep sequencing-based post-remission MRD surveillance predicts clinical relapse in B-ALL", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Simple deep sequencing-based post-remission MRD surveillance predicts clinical relapse in B-ALL\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1186/s13045-018-0652-y\"\u003e10.1186/s13045-018-0652-y\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eThe NGS-MRD assay demonstrated an analytical sensitivity of approximately 0.0001%, enabling detection of as few as 1 leukemic cell among 1 million normal leukocytes with high reproducibility and linearity.\u003c/li\u003e\n  \u003cli\u003eThe assay showed high intra-run and inter-run precision, with minimal variation even near the lower limit of detection, supporting its robustness for clinical use.\u003c/li\u003e\n  \u003cli\u003eCompared to multi-color flow cytometry (MCF), the NGS method detected MRD in 22 additional samples that were MCF-negative, suggesting superior sensitivity and fewer false negatives.\u003c/li\u003e\n  \u003cli\u003eNGS-MRD surveillance was able to detect conversion to positive MRD (CPMRD) in post-remission patients up to 6.4 months before clinical relapse, while MCF failed to detect early relapse in all surveilled patients.\u003c/li\u003e\n  \u003cli\u003eAll patients with detectable CPMRD by NGS eventually experienced clinical relapse, demonstrating that CPMRD serves as a reliable early biomarker of impending disease recurrence.\u003c/li\u003e\n  \u003cli\u003eAmong surveilled patients, the median time from CPMRD detection to clinical relapse was 4.7 months, compared to a 20-month median gap in non-surveilled patients where relapse occurred without earlier MRD detection.\u003c/li\u003e\n  \u003cli\u003ePatients with post-remission CPMRD had significantly shorter relapse-free survival than those with negative MRD conversion (NMRDC), with 100% sensitivity, specificity, PPV, and NPV for predicting clinical relapse.\u003c/li\u003e\n  \u003cli\u003eIn contrast, NGS-based MRD positivity at the end of induction was not significantly predictive of relapse in this small cohort, with several MRD-positive patients not relapsing within the follow-up window.\u003c/li\u003e\n  \u003cli\u003eThe platform, based on LymphoTrack-Miseq, offers a practical, scalable, and clinically implementable alternative to centralized proprietary NGS-MRD assays.\u003c/li\u003e\n  \u003cli\u003eThe study highlights that rising MRD levels (CPMRD), rather than single positive MRD results, are likely more predictive of relapse, and advocates for serial MRD monitoring to improve early intervention strategies.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "NGS-based MRD detection assay development", "label": "NGS-based MRD detection assay development", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Analytical sensitivity and reproducibility of MRD testing", "label": "Analytical sensitivity and reproducibility of MRD testing", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison between NGS and flow cytometry for MRD", "label": "Comparison between NGS and flow cytometry for MRD", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Post-remission MRD surveillance in B-ALL", "label": "Post-remission MRD surveillance in B-ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prediction of clinical relapse using CPMRD", "label": "Prediction of clinical relapse using CPMRD", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical utility of dynamic MRD monitoring", "label": "Clinical utility of dynamic MRD monitoring", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Implementation feasibility in routine clinical labs", "label": "Implementation feasibility in routine clinical labs", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Preemptive therapy timing based on MRD trends", "label": "Preemptive therapy timing based on MRD trends", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Clinical utility of next-generation sequencing-based minimal residual disease in paediatric B-cell acute lymphoblastic leukaemia", "label": "Clinical utility of next-generation sequencing-based minimal residual disease in paediatric B-cell acute lymphoblastic leukaemia", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Clinical utility of next-generation sequencing-based minimal residual disease in paediatric B-cell acute lymphoblastic leukaemia\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1111/bjh.14420\"\u003e10.1111/bjh.14420\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eNGS-based MRD was successfully implemented in 91% of pediatric B-cell ALL patients (72 out of 79) by identifying leukemia-specific CDR3 sequences, demonstrating strong feasibility of the technique in clinical practice.\u003c/li\u003e\n  \u003cli\u003eThe detection rate of MRD using NGS declined over time, from 51% at day 33 to 25% at day 80, 19% at 4\u20135 months, and 7.4% at 24 months, indicating effective disease clearance in most patients through treatment.\u003c/li\u003e\n  \u003cli\u003eNGS-MRD positivity at day 80, 4\u20135 months, and 24 months was significantly associated with worse leukemia-free survival, with hazard ratios of 7.44, 10.24, and 19.26 respectively (all p \u0026lt; 0.001), establishing its prognostic value.\u003c/li\u003e\n  \u003cli\u003eStratifying patients based on NGS-MRD sensitivity at 10\u003csup\u003e\u22126\u003c/sup\u003e (as opposed to 10\u003csup\u003e\u22124\u003c/sup\u003e) better discriminated standard-risk from intermediate-risk patients, resulting in significantly different 5-year leukemia-free survival outcomes (100% vs. 61.7%, p = 0.005).\u003c/li\u003e\n  \u003cli\u003eEven low-positive NGS-MRD results (\u0026lt;10\u003csup\u003e\u22124\u003c/sup\u003e) at day 80 and 4\u20135 months were significantly predictive of poor outcomes, suggesting that deep sequencing can detect clinically relevant residual disease levels overlooked by conventional methods.\u003c/li\u003e\n  \u003cli\u003eAll four patients with MRD positivity at 24 months relapsed shortly afterward, indicating that any detectable disease at this time point should be considered a high-risk feature warranting aggressive intervention.\u003c/li\u003e\n  \u003cli\u003eMultivariate Cox analysis confirmed that both low and high positive NGS-MRD results at day 80 and 4\u20135 months were independent predictors of relapse, even after adjusting for known risk factors.\u003c/li\u003e\n  \u003cli\u003eAmong 15 patients with relapse samples, clonal evolution was observed in 3 cases, while the original clones remained detectable at relapse in 12 cases, showing that relapses were mostly driven by the same dominant leukemic clones present at diagnosis.\u003c/li\u003e\n  \u003cli\u003eNGS-MRD showed greater sensitivity (10\u003csup\u003e\u22126\u003c/sup\u003e) than real-time PCR or flow cytometry (typically 10\u003csup\u003e\u22124\u003c/sup\u003e), and was capable of detecting low-level disease missed by other modalities, potentially reducing false negatives.\u003c/li\u003e\n  \u003cli\u003eThe study suggested MRD re-monitoring at 4\u20135 months may be redundant for patients negative at day 80, but warranted for those positive at day 80, refining timing recommendations for MRD surveillance.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Next-generation sequencing (NGS) for MRD detection", "label": "Next-generation sequencing (NGS) for MRD detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic value of MRD at multiple treatment time points", "label": "Prognostic value of MRD at multiple treatment time points", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Risk stratification in pediatric B-cell ALL", "label": "Risk stratification in pediatric B-cell ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of NGS with traditional MRD methods (RQ-PCR, FCM)", "label": "Comparison of NGS with traditional MRD methods (RQ-PCR, FCM)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sensitivity thresholds for MRD detection", "label": "Sensitivity thresholds for MRD detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clonal evolution in relapsed leukemia", "label": "Clonal evolution in relapsed leukemia", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Treatment implications based on MRD status", "label": "Treatment implications based on MRD status", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Timing and frequency of MRD monitoring", "label": "Timing and frequency of MRD monitoring", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical outcomes and relapse prediction", "label": "Clinical outcomes and relapse prediction", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Utility of deep sequencing in personalized treatment planning", "label": "Utility of deep sequencing in personalized treatment planning", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Association of Minimal Residual Disease With Clinical Outcome in Pediatric and Adult Acute Lymphoblastic Leukemia: A Meta-analysis", "label": "Association of Minimal Residual Disease With Clinical Outcome in Pediatric and Adult Acute Lymphoblastic Leukemia: A Meta-analysis", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Association of Minimal Residual Disease With Clinical Outcome in Pediatric and Adult Acute Lymphoblastic Leukemia: A Meta-analysis\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1001/jamaoncol.2017.0580\"\u003e10.1001/jamaoncol.2017.0580\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eMinimal residual disease (MRD) negativity after induction therapy was strongly associated with improved event-free survival (EFS) in both pediatric and adult acute lymphoblastic leukemia (ALL) patients, with hazard ratios (HR) of 0.23 and 0.28, respectively.\u003c/li\u003e\n  \u003cli\u003eMRD negativity was also associated with significantly improved overall survival (OS) in both populations, with an HR of 0.28 in pediatric and adult patients alike.\u003c/li\u003e\n  \u003cli\u003eThe beneficial association of MRD negativity with outcomes was consistent across MRD detection methods (PCR and flow cytometry), detection timepoints (post-induction vs. post-consolidation), cutoff thresholds, phenotypes (B-cell vs. T-cell), and Philadelphia chromosome status.\u003c/li\u003e\n  \u003cli\u003ePatients with MRD negativity had markedly better 10-year survival: pediatric EFS was 77% vs. 32% for MRD-positive; adult EFS was 64% vs. 21%.\u003c/li\u003e\n  \u003cli\u003eMRD cutoff level sensitivity analysis showed that both \u22640.01% and \u003e0.01% thresholds were predictive of outcome, although smaller HRs were paradoxically observed with higher cutoffs in some subgroups.\u003c/li\u003e\n  \u003cli\u003eSubgroup analyses confirmed robustness of MRD\u2019s prognostic value, showing no significant modification of MRD\u0027s effect on outcome by patient age, phenotype, cytogenetics, or MRD detection method.\u003c/li\u003e\n  \u003cli\u003eMRD negativity is proposed as a surrogate biomarker for therapeutic response and is recommended as an early end point in clinical trials to support accelerated drug approval.\u003c/li\u003e\n  \u003cli\u003eNext-generation sequencing is highlighted as a promising future method for MRD detection due to its higher sensitivity and potential for standardization compared to flow cytometry or PCR.\u003c/li\u003e\n  \u003cli\u003eThe study suggests adaptive phase 3 trial designs incorporating MRD rates and EFS projections may improve trial efficiency and predictive accuracy for evaluating new therapies.\u003c/li\u003e\n  \u003cli\u003eAlthough MRD is a strong prognostic marker, the study cautions that its use as a surrogate end point in drug trials must be validated against traditional outcomes like EFS and OS due to possible dissociation in newer therapies.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Minimal residual disease (MRD) as a prognostic biomarker", "label": "Minimal residual disease (MRD) as a prognostic biomarker", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Event-free survival (EFS) and overall survival (OS) in ALL", "label": "Event-free survival (EFS) and overall survival (OS) in ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD detection methods (PCR vs flow cytometry vs NGS)", "label": "MRD detection methods (PCR vs flow cytometry vs NGS)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Risk stratification in pediatric and adult ALL", "label": "Risk stratification in pediatric and adult ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD cutoff thresholds and sensitivity", "label": "MRD cutoff thresholds and sensitivity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical trial design and drug development using MRD", "label": "Clinical trial design and drug development using MRD", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Subgroup analyses (B-cell/T-cell, Ph+/Ph\u2212, age groups)", "label": "Subgroup analyses (B-cell/T-cell, Ph+/Ph\u2212, age groups)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Role of MRD in regulatory approval and surrogate endpoints", "label": "Role of MRD in regulatory approval and surrogate endpoints", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Adaptive trial methodologies using MRD-guided endpoints", "label": "Adaptive trial methodologies using MRD-guided endpoints", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Limitations and future directions in MRD standardization4", "label": "Limitations and future directions in MRD standardization4", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Measurable residual disease in acute lymphoblastic leukemia: methods and clinical context in adult patients", "label": "Measurable residual disease in acute lymphoblastic leukemia: methods and clinical context in adult patients", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Measurable residual disease in acute lymphoblastic leukemia: methods and clinical context in adult patients\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.3324/haematol.2022.280638\"\u003e10.3324/haematol.2022.280638\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eMRD is the strongest independent predictor of relapse and long-term survival in both adult and pediatric ALL patients, regardless of detection method.\u003c/li\u003e\n  \u003cli\u003eNext-generation sequencing (NGS) offers the highest sensitivity (up to 10\u003csup\u003e\u22126\u003c/sup\u003e) among MRD detection methods and is FDA-approved via ClonoSEQ, with potential for peripheral blood monitoring.\u003c/li\u003e\n  \u003cli\u003eFlow cytometry (MFC) and qPCR are widely used, but MFC is less reliable in patients treated with CD19-directed therapies due to immunophenotypic shifts or antigen loss.\u003c/li\u003e\n  \u003cli\u003eMRD-guided treatment strategies are becoming standard, with MRD positivity prompting therapy intensification or allogeneic hematopoietic cell transplant (HCT).\u003c/li\u003e\n  \u003cli\u003ePersistent MRD at levels \u003e10\u003csup\u003e\u22124\u003c/sup\u003e before or after allogeneic HCT is associated with significantly increased relapse risk, underscoring the need for MRD clearance pre-transplant.\u003c/li\u003e\n  \u003cli\u003eMRD monitoring at regular intervals (e.g., every 3 months) can detect molecular relapse 3\u20134 months ahead of clinical relapse, enabling timely interventions.\u003c/li\u003e\n  \u003cli\u003eNGS-based MRD detected additional cases missed by flow cytometry, and NGS-MRD negativity post-consolidation correlated with better survival outcomes.\u003c/li\u003e\n  \u003cli\u003eMRD negativity after novel therapies (e.g., blinatumomab, inotuzumab ozogamicin, CAR T cells) correlates with superior survival, and has been used to guide regulatory approval.\u003c/li\u003e\n  \u003cli\u003eIn T-ALL, effective MRD-targeted therapies are still lacking; MRD positivity in this subtype remains an unmet need in therapeutic decision-making.\u003c/li\u003e\n  \u003cli\u003eEmerging methods like digital droplet PCR and EuroClonality-NGS panels may enhance MRD detection, but require further clinical validation for ALL.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic value of measurable residual disease (MRD) in ALL", "label": "Prognostic value of measurable residual disease (MRD) in ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD detection methods (MFC, qPCR, NGS, ddPCR)", "label": "MRD detection methods (MFC, qPCR, NGS, ddPCR)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Next-generation sequencing (NGS) and ClonoSEQ", "label": "Next-generation sequencing (NGS) and ClonoSEQ", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD-informed treatment strategies and risk stratification", "label": "MRD-informed treatment strategies and risk stratification", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Role of MRD in guiding allogeneic hematopoietic cell transplant (HCT)", "label": "Role of MRD in guiding allogeneic hematopoietic cell transplant (HCT)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD detection post-novel therapies (e.g., blinatumomab, inotuzumab, CAR T)", "label": "MRD detection post-novel therapies (e.g., blinatumomab, inotuzumab, CAR T)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Limitations in MRD tracking for T-cell ALL", "label": "Limitations in MRD tracking for T-cell ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Emerging MRD technologies and diagnostic platforms", "label": "Emerging MRD technologies and diagnostic platforms", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical relevance of low-level MRD and early molecular relapse", "label": "Clinical relevance of low-level MRD and early molecular relapse", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: an European Research Initiative on CLL study", "label": "A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: an European Research Initiative on CLL study", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: an European Research Initiative on CLL study\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1038/leu.2015.313\"\u003e10.1038/leu.2015.313\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eAn optimized six-marker core panel (CD19, CD20, CD5, CD43, CD79b, CD81) was identified for detecting MRD in chronic lymphocytic leukemia (CLL) with high sensitivity (down to 0.0010%).\u003c/li\u003e\n  \u003cli\u003eThis panel showed excellent reproducibility and concordance with expected values across multiple laboratories, supporting its suitability for widespread clinical use.\u003c/li\u003e\n  \u003cli\u003eCompared to older ERIC-standardized 4-color (4-tube) and 6-color (2-tube) flow cytometry methods, the one-tube six-marker panel reduced sample handling and improved sensitivity and quantification.\u003c/li\u003e\n  \u003cli\u003eHigh-throughput sequencing (HTS) using the ClonoSEQ platform demonstrated a superior detection limit down to 0.00010% (1 in 1 million cells) and good concordance with flow cytometry at the 0.010% MRD threshold.\u003c/li\u003e\n  \u003cli\u003eHTS identified dominant productive IGH rearrangements in 98% of CLL samples and provided more complete clonal information than Sanger sequencing, especially in challenging or complex rearrangements.\u003c/li\u003e\n  \u003cli\u003eHTS detected MRD in 22% of samples where flow cytometry did not, underscoring its superior analytical sensitivity, although with more variability in quantitative results compared to flow cytometry.\u003c/li\u003e\n  \u003cli\u003eInter-operator variability using the six-marker panel was within acceptable limits (\u00b10.27 log), confirming ease of implementation even by users with minimal prior MRD-specific experience.\u003c/li\u003e\n  \u003cli\u003eThe panel supports platform-independent implementation by specifying minimum fluorescence intensity thresholds for each marker, ensuring consistent performance across labs and reagents.\u003c/li\u003e\n  \u003cli\u003eFlow cytometry remains more practical for fresh sample analysis and real-time MRD assessment, whereas HTS is advantageous for archived samples and ultra-sensitive detection.\u003c/li\u003e\n  \u003cli\u003eThe combination of both methods (flow cytometry and HTS) offers a robust and complementary approach to MRD detection in CLL, enabling both accessibility and high sensitivity depending on clinical needs.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Minimal Residual Disease (MRD) detection in Chronic Lymphocytic Leukemia (CLL)", "label": "Minimal Residual Disease (MRD) detection in Chronic Lymphocytic Leukemia (CLL)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Development and validation of a six-marker core flow cytometry panel", "label": "Development and validation of a six-marker core flow cytometry panel", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of flow cytometry with high-throughput sequencing (HTS) for MRD", "label": "Comparison of flow cytometry with high-throughput sequencing (HTS) for MRD", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Assay sensitivity, reproducibility, and inter-laboratory standardization", "label": "Assay sensitivity, reproducibility, and inter-laboratory standardization", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Limitations and advantages of Sanger sequencing versus HTS", "label": "Limitations and advantages of Sanger sequencing versus HTS", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Application of ClonoSEQ HTS platform for IGH clonotype detection", "label": "Application of ClonoSEQ HTS platform for IGH clonotype detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Platform-independent reagent specifications for MRD testing", "label": "Platform-independent reagent specifications for MRD testing", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical trial integration and real-world implementation feasibility", "label": "Clinical trial integration and real-world implementation feasibility", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Impact of marker selection and reagent signal intensity on CLL detection", "label": "Impact of marker selection and reagent signal intensity on CLL detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Complementary roles of flow cytometry and HTS in CLL treatment monitoring", "label": "Complementary roles of flow cytometry and HTS in CLL treatment monitoring", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "label": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1182/bloodadvances.2022007378\"\u003e10.1182/bloodadvances.2022007378\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eAmong adult patients with acute lymphoblastic leukemia (ALL), 46% of samples that were MRD-negative by multiparameter flow cytometry (MFC) were found to be MRD-positive by next-generation sequencing (NGS), indicating that standard MFC may miss clinically significant disease.\u003c/li\u003e\n  \u003cli\u003eEarly MRD negativity detected by NGS at a sensitivity of 10\u003csup\u003e-6\u003c/sup\u003e (after one induction cycle) was associated with a 0% relapse rate and a 5-year overall survival of 90%, compared to 45% relapse and 61% survival in MRD-positive patients.\u003c/li\u003e\n  \u003cli\u003ePatients who were MRD-negative by MFC but MRD-positive by NGS had relapse risks comparable to those MRD-positive by both methods, highlighting the added prognostic power of NGS.\u003c/li\u003e\n  \u003cli\u003eIn non-transplanted patients, early NGS MRD negativity still predicted excellent outcomes, suggesting that such patients might avoid allogeneic stem cell transplantation (alloSCT) without compromising prognosis.\u003c/li\u003e\n  \u003cli\u003eAssessment of MRD using NGS at later time points (1.5 to 4.5 months post-treatment) was less predictive of relapse than early post-induction MRD testing.\u003c/li\u003e\n  \u003cli\u003eNGS MRD negativity at a lower sensitivity threshold (e.g., 10\u003csup\u003e-4\u003c/sup\u003e or 10\u003csup\u003e-5\u003c/sup\u003e) was less discriminative, confirming that the 10\u003csup\u003e-6\u003c/sup\u003e cutoff provides superior risk stratification.\u003c/li\u003e\n  \u003cli\u003eThe study supports the potential de-escalation of therapy for patients achieving deep early MRD responses and suggests exploring use of blinatumomab or other agents in patients with low-level MRD detected only by NGS.\u003c/li\u003e\n  \u003cli\u003eLow-intensity chemotherapy regimens combined with monoclonal antibodies (e.g., inotuzumab ozogamicin) achieved similar MRD negativity rates as intensive regimens, suggesting their efficacy for deep remission in select patients.\u003c/li\u003e\n  \u003cli\u003eThe study emphasizes that early and deep MRD response, not just MRD status alone, should be the basis for clinical decision-making in frontline ALL therapy.\u003c/li\u003e\n  \u003cli\u003eThese findings underscore the need for prospective trials to define the role of high-sensitivity NGS MRD testing in guiding treatment intensity and transplantation decisions.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD detection in acute lymphoblastic leukemia (ALL)", "label": "MRD detection in acute lymphoblastic leukemia (ALL)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of NGS vs. multiparameter flow cytometry (MFC)", "label": "Comparison of NGS vs. multiparameter flow cytometry (MFC)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical significance of early MRD negativity by NGS", "label": "Clinical significance of early MRD negativity by NGS", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic implications of MRD discordance between NGS and MFC", "label": "Prognostic implications of MRD discordance between NGS and MFC", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Risk stratification using high-sensitivity NGS assays", "label": "Risk stratification using high-sensitivity NGS assays", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Impact of MRD status on relapse and overall survival", "label": "Impact of MRD status on relapse and overall survival", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Role of MRD in guiding allogeneic stem cell transplantation (alloSCT) decisions", "label": "Role of MRD in guiding allogeneic stem cell transplantation (alloSCT) decisions", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Efficacy of low-intensity regimens combined with monoclonal antibodies", "label": "Efficacy of low-intensity regimens combined with monoclonal antibodies", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Timing of MRD assessment and its prognostic value", "label": "Timing of MRD assessment and its prognostic value", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Therapeutic implications and de-escalation strategies based on MRD", "label": "Therapeutic implications and de-escalation strategies based on MRD", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "label": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1186/s12885-020-07077-9\"\u003e10.1186/s12885-020-07077-9\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eThe clonoSEQ Assay demonstrated a limit of detection (LoD) of 1.903 malignant cells and a limit of quantitation (LoQ) of 2.390 cells, enabling detection of MRD levels as low as 6.77 \u00d7 10\u003csup\u003e\u22127\u003c/sup\u003e with high sensitivity across various DNA input levels.\u003c/li\u003e\n  \u003cli\u003ePrecision analysis showed coefficient of variation (%CV) ranging from 18% at high MRD levels to 68% at the LoD, indicating robust performance even at low disease burdens.\u003c/li\u003e\n  \u003cli\u003eLinearity was confirmed across a wide range of MRD frequencies and DNA input amounts (200 ng to 40 \u03bcg), with observed vs. expected slopes ranging from 0.95 to 1.03 across ALL, CLL, and MM, confirming the assay\u0027s quantitative reliability.\u003c/li\u003e\n  \u003cli\u003eQuantitation accuracy was within \u00b125% of expected values across tested cell inputs when compared to multiparametric flow cytometry (mpFC), with a strong correlation (R\u00b2 = 0.98), supporting reliable MRD quantification.\u003c/li\u003e\n  \u003cli\u003eSequence accuracy was extremely high, with an error rate of approximately 3.5 parts per 100,000 nucleotides, corresponding to Phred quality scores around 44.5.\u003c/li\u003e\n  \u003cli\u003eThe limit of blank (LoB) was effectively zero, with a false detection rate below 1%, confirming the assay\u0027s high specificity and minimal background signal in healthy samples.\u003c/li\u003e\n  \u003cli\u003eReproducibility across operators, instruments, reagent lots, and assay runs showed minimal variability (0\u20133% CV), highlighting the assay\u2019s robustness under routine laboratory conditions.\u003c/li\u003e\n  \u003cli\u003eMultiple receptor sequences from the same clonal population can be tracked, improving sensitivity and reducing the impact of Poisson sampling errors near the LoD.\u003c/li\u003e\n  \u003cli\u003eThe assay assigns a \"uniqueness score\" to each sequence, preventing tracking of sequences likely to appear in healthy repertoires and minimizing false positives in MRD detection.\u003c/li\u003e\n  \u003cli\u003eclonoSEQ is validated as a standardized, accurate, and sensitive assay suitable for MRD detection in ALL, CLL, and MM, offering advantages over conventional methods such as flow cytometry and ASO-PCR.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Analytical validation of the clonoSEQ Assay", "label": "Analytical validation of the clonoSEQ Assay", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD detection sensitivity, specificity, and quantification", "label": "MRD detection sensitivity, specificity, and quantification", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Precision and reproducibility across DNA input levels", "label": "Precision and reproducibility across DNA input levels", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Linearity and accuracy of MRD measurements", "label": "Linearity and accuracy of MRD measurements", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison with multiparameter flow cytometry (mpFC)", "label": "Comparison with multiparameter flow cytometry (mpFC)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sequence fidelity and tracking of clonal rearrangements", "label": "Sequence fidelity and tracking of clonal rearrangements", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Application in ALL, CLL, and MM", "label": "Application in ALL, CLL, and MM", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Evaluation of detection thresholds: LoD, LoQ, and LoB", "label": "Evaluation of detection thresholds: LoD, LoQ, and LoB", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Impact of DNA input quantity on MRD assay performance", "label": "Impact of DNA input quantity on MRD assay performance", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Standardization and reliability for clinical implementation", "label": "Standardization and reliability for clinical implementation", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "label": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cb\u003eArticle Title:\u003c/b\u003e Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study\u003c/p\u003e\n\u003cp\u003e\u003cb\u003eDOI:\u003c/b\u003e \u003ca href=\"https://doi.org/10.1200/JCO.2023.41.16_suppl.7033\"\u003e10.1200/JCO.2023.41.16_suppl.7033\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eThis study compared MRD detection using next-generation sequencing (NGS) via the clonoSEQ assay and multiparameter flow cytometry (MFC) in adult ALL patients who achieved complete remission (CR).\u003c/li\u003e\n  \u003cli\u003eAmong 52 patients assessed by both methods at CR, 50% were concordantly MRD-negative by both NGS and MFC, while 25% were discordant\u2014MRD-negative by MFC but MRD-positive by NGS\u2014highlighting NGS\u2019s greater sensitivity.\u003c/li\u003e\n  \u003cli\u003eAnother 25% of patients were MRD-positive by both NGS and MFC, demonstrating overlap but also limitations in sensitivity and agreement between methods.\u003c/li\u003e\n  \u003cli\u003eResidual disease levels detected by NGS and MFC were significantly correlated (R\u00b2 = 0.6744, p \u003c 0.001), indicating that both methods reflect disease burden, but with NGS detecting lower levels.\u003c/li\u003e\n  \u003cli\u003eMRD status classification was significantly different between methods (p \u003c 0.001, Fisher\u2019s Exact Test), attributed to their differing limits of detection\u201410\u207b\u2076 for NGS versus 10\u207b\u2075 for MFC.\u003c/li\u003e\n  \u003cli\u003eMedian follow-up was 18 months post-MRD testing, during which 17.9% of patients relapsed, 5.4% died, and 35.7% underwent allogeneic transplant, offering context for prognostic value assessment.\u003c/li\u003e\n  \u003cli\u003eClonoSEQ assay results provided additional prognostic value over MFC, supporting its use in refining post-remission therapy decisions for adult ALL patients.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of NGS (clonoSEQ) and flow cytometry (MFC) for MRD detection", "label": "Comparison of NGS (clonoSEQ) and flow cytometry (MFC) for MRD detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sensitivity differences between NGS (10\u207b\u2076) and MFC (10\u207b\u2075)", "label": "Sensitivity differences between NGS (10\u207b\u2076) and MFC (10\u207b\u2075)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Concordance and discordance rates in MRD classification", "label": "Concordance and discordance rates in MRD classification", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Statistical correlation between NGS and MFC MRD levels", "label": "Statistical correlation between NGS and MFC MRD levels", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical outcome tracking (relapse, death, transplant) post-MRD assessment", "label": "Clinical outcome tracking (relapse, death, transplant) post-MRD assessment", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic value of NGS MRD detection in adult ALL", "label": "Prognostic value of NGS MRD detection in adult ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Implications for treatment decisions and post-remission risk stratification", "label": "Implications for treatment decisions and post-remission risk stratification", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "CDRH\u2019s Approach to Tumor Profiling Next Generation Sequencing Tests", "label": "CDRH\u2019s Approach to Tumor Profiling Next Generation Sequencing Tests", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cdiv\u003e\n\u003cb\u003eArticle Title:\u003c/b\u003e CDRH\u2019s Approach to Tumor Profiling Next Generation Sequencing Tests\u003cbr\u003e\n\u003cb\u003eDOI:\u003c/b\u003e Not available\n\u003c/div\u003e\n\n\u003cdiv\u003e\n\u003col\u003e\n  \u003cli\u003eThe FDA authorized three tumor profiling NGS tests\u2014Oncomine Dx Target Test, MSK-IMPACT, and FoundationOne CDx\u2014as significant advances in precision oncology.\u003c/li\u003e\n  \u003cli\u003eA three-tiered framework was introduced for reporting biomarkers: Level 1 (Companion Diagnostics), Level 2 (Cancer Mutations with Evidence of Clinical Significance), and Level 3 (Cancer Mutations with Potential Clinical Significance).\u003c/li\u003e\n  \u003cli\u003eLevel 1 biomarkers require analytical validity and either a clinical outcome study or concordance with a previously approved companion diagnostic.\u003c/li\u003e\n  \u003cli\u003eLevel 2 biomarkers are supported by analytical and clinical validity and are typically backed by clinical guidelines or peer-reviewed literature.\u003c/li\u003e\n  \u003cli\u003eLevel 3 biomarkers are supported by analytical validation and a mechanistic rationale but are considered exploratory or for trial-matching purposes.\u003c/li\u003e\n  \u003cli\u003eThe FDA permits a dynamic updating mechanism post-market, allowing test developers to report additional variants within validated genes and to reclassify mutations from Level 3 to Level 2 as clinical evidence accumulates\u2014without a new FDA submission.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "FDA authorization of tumor profiling NGS tests", "label": "FDA authorization of tumor profiling NGS tests", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Three-tiered biomarker classification system", "label": "Three-tiered biomarker classification system", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Companion diagnostics and therapeutic guidance", "label": "Companion diagnostics and therapeutic guidance", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Cancer mutations with established clinical significance", "label": "Cancer mutations with established clinical significance", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Cancer mutations with potential clinical relevance", "label": "Cancer mutations with potential clinical relevance", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Regulatory flexibility and post-market variant updates", "label": "Regulatory flexibility and post-market variant updates", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "ClonoSEQ assay for the detection of lymphoid malignancies", "label": "ClonoSEQ assay for the detection of lymphoid malignancies", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e ClonoSEQ assay for the detection of lymphoid malignancies\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1080/14737159.2019.1627877\"\u003e10.1080/14737159.2019.1627877\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eClonoSEQ was the first FDA-approved next-generation sequencing (NGS) assay for minimal residual disease (MRD) assessment in B-cell lineage acute lymphoblastic leukemia (B-ALL) and multiple myeloma (MM), offering higher sensitivity (up to 1 in 10\u003csup\u003e6\u003c/sup\u003e) than traditional methods like flow cytometry or ASO-PCR.\u003c/li\u003e\n  \u003cli\u003eClonoSEQ can detect clonal evolution and track minor subclones, enabling prediction of relapse and treatment optimization in lymphoid malignancies.\u003c/li\u003e\n  \u003cli\u003eThe assay incorporates synthetic internal controls and a bioinformatics pipeline to minimize amplification bias and improve quantification accuracy of MRD levels.\u003c/li\u003e\n  \u003cli\u003eIn multiple studies comparing ClonoSEQ with flow cytometry and ASO-PCR, ClonoSEQ consistently detected MRD at lower levels, and MRD positivity by ClonoSEQ was predictive of shorter progression-free survival (PFS) and overall survival (OS).\u003c/li\u003e\n  \u003cli\u003eIn pediatric B-ALL, ClonoSEQ-based MRD stratification improved outcome prediction, particularly among patients who were MRD-negative by flow cytometry but positive by ClonoSEQ.\u003c/li\u003e\n  \u003cli\u003ePost-transplant studies showed that MRD-negative patients by ClonoSEQ had significantly lower relapse rates and higher OS, reinforcing its utility for pre- and post-HSCT risk stratification.\u003c/li\u003e\n  \u003cli\u003eClonoSEQ was successfully applied to mature B-cell malignancies like chronic lymphocytic leukemia (CLL) and mantle cell lymphoma (MCL), demonstrating high concordance with flow cytometry and predicting relapse risk post-HSCT.\u003c/li\u003e\n  \u003cli\u003eApplication of ClonoSEQ to cell-free circulating tumor DNA (ctDNA) in plasma showed potential for non-invasive MRD monitoring and early relapse detection in diffuse large B-cell lymphoma (DLBCL).\u003c/li\u003e\n  \u003cli\u003eDespite its clinical promise, ClonoSEQ\u2019s limitations include high cost, need for pretreatment baseline samples, dependency on sufficient DNA input, and its status as a closed proprietary platform.\u003c/li\u003e\n  \u003cli\u003eOngoing efforts by consortia like EuroClonality-NGS aim to develop open, standardized HTS protocols for MRD analysis that may match or complement ClonoSEQ\u2019s capabilities at potentially lower cost.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Minimal Residual Disease (MRD) detection in lymphoid malignancies", "label": "Minimal Residual Disease (MRD) detection in lymphoid malignancies", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "ClonoSEQ assay design and technical features", "label": "ClonoSEQ assay design and technical features", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of ClonoSEQ with flow cytometry and ASO-PCR", "label": "Comparison of ClonoSEQ with flow cytometry and ASO-PCR", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "ClonoSEQ\u0027s clinical application in B-cell acute lymphoblastic leukemia (B-ALL)", "label": "ClonoSEQ\u0027s clinical application in B-cell acute lymphoblastic leukemia (B-ALL)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "ClonoSEQ\u0027s prognostic value in multiple myeloma (MM)", "label": "ClonoSEQ\u0027s prognostic value in multiple myeloma (MM)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Use of ClonoSEQ in chronic lymphocytic leukemia (CLL) and mantle cell lymphoma (MCL)", "label": "Use of ClonoSEQ in chronic lymphocytic leukemia (CLL) and mantle cell lymphoma (MCL)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Non-invasive MRD detection using circulating tumor DNA (ctDNA)", "label": "Non-invasive MRD detection using circulating tumor DNA (ctDNA)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Limitations and challenges of the ClonoSEQ platform", "label": "Limitations and challenges of the ClonoSEQ platform", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Standardization efforts in high-throughput sequencing for MRD (e.g., EuroClonality-NGS)", "label": "Standardization efforts in high-throughput sequencing for MRD (e.g., EuroClonality-NGS)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clonal evolution and its implications for MRD tracking", "label": "Clonal evolution and its implications for MRD tracking", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Analytical validation of the clonoSEQ assay", "label": "Analytical validation of the clonoSEQ assay", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Limit of detection (LoD), quantitation (LoQ), and blank (LoB) for MRD", "label": "Limit of detection (LoD), quantitation (LoQ), and blank (LoB) for MRD", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Precision and reproducibility across input levels and cancer types", "label": "Precision and reproducibility across input levels and cancer types", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Linearity of MRD quantification across DNA concentrations and disease burdens", "label": "Linearity of MRD quantification across DNA concentrations and disease burdens", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of clonoSEQ assay with multiparametric flow cytometry (mpFC)", "label": "Comparison of clonoSEQ assay with multiparametric flow cytometry (mpFC)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sequence accuracy and error rate metrics", "label": "Sequence accuracy and error rate metrics", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Tracking of multiple receptor sequences for enhanced sensitivity", "label": "Tracking of multiple receptor sequences for enhanced sensitivity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Impact of clonotype uniqueness on MRD specificity", "label": "Impact of clonotype uniqueness on MRD specificity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Standardization and robustness of NGS-based MRD assessment", "label": "Standardization and robustness of NGS-based MRD assessment", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Limitations of the assay and considerations for clinical implementation", "label": "Limitations of the assay and considerations for clinical implementation", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of MRD detection methods (NGS vs. MFC)", "label": "Comparison of MRD detection methods (NGS vs. MFC)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sensitivity and detection thresholds of clonoSEQ and flow cytometry", "label": "Sensitivity and detection thresholds of clonoSEQ and flow cytometry", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Concordance and discordance in MRD results between assays", "label": "Concordance and discordance in MRD results between assays", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical implications of MRD positivity and relapse prediction", "label": "Clinical implications of MRD positivity and relapse prediction", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic value of NGS-based MRD testing in adult ALL", "label": "Prognostic value of NGS-based MRD testing in adult ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Correlation between MRD burden measured by NGS and MFC", "label": "Correlation between MRD burden measured by NGS and MFC", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Use of MRD testing to guide treatment decisions and transplant timing", "label": "Use of MRD testing to guide treatment decisions and transplant timing", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Minimal Residual Disease Detection in Adults with Acute Lymphoblastic Leukemia By Next Generation Sequencing: Impact on Clinical Outcomes", "label": "Minimal Residual Disease Detection in Adults with Acute Lymphoblastic Leukemia By Next Generation Sequencing: Impact on Clinical Outcomes", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Minimal Residual Disease Detection in Adults with Acute Lymphoblastic Leukemia By Next Generation Sequencing: Impact on Clinical Outcomes\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1182/blood-2020-143416\"\u003e10.1182/blood-2020-143416\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eIn a real-world cohort of 42 adult ALL patients, 38% were MRD-negative by flow cytometry but MRD-positive by the clonoSEQ NGS assay, revealing higher sensitivity of NGS in early detection of residual disease.\u003c/li\u003e\n  \n  \u003cli\u003eAmong the patients with discordant results (Flow\u2212/clonoSEQ+), 31% later tested positive by flow and relapsed, demonstrating that early detection via NGS could have prognostic value and therapeutic implications.\u003c/li\u003e\n\n  \u003cli\u003eOf the patients with Flow\u2212/clonoSEQ+ results, 19% received blinatumomab followed by allo-HCT; two remained in remission and one died from unrelated complications, suggesting benefit from MRD-directed intervention.\u003c/li\u003e\n\n  \u003cli\u003eIn the subgroup of 11 Ph-like B-ALL patients, clonoSEQ-guided therapy decisions led to allo-HCT in 8 MRD+ patients (with 5 in remission and 3 relapsed) and continued chemotherapy in 3 MRD\u2212 patients (all in sustained remission).\u003c/li\u003e\n\n  \u003cli\u003eIn a comparison of post-induction MRD status in 12 patients, 25% were Flow+/clonoSEQ+, 25% Flow\u2212/clonoSEQ+, and 50% Flow\u2212/clonoSEQ\u2212. The corresponding 12-month disease-free survival (DFS) rates were 33%, 100%, and 50%, respectively.\u003c/li\u003e\n\n  \u003cli\u003ePatients who were MRD-negative by clonoSEQ after induction therapy and did not receive allo-HCT maintained molecular remission beyond one year, indicating that intensive therapy might be avoidable in this subset.\u003c/li\u003e\n\n  \u003cli\u003eThe study concludes that early MRD evaluation via NGS may provide superior prognostic guidance compared to flow cytometry and can inform tailored treatment strategies, especially in high-risk adult ALL populations.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sensitivity differences between NGS (clonoSEQ) and flow cytometry", "label": "Sensitivity differences between NGS (clonoSEQ) and flow cytometry", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Discordance in MRD detection and its clinical implications", "label": "Discordance in MRD detection and its clinical implications", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD-guided therapeutic decisions (e.g., blinatumomab, allo-HCT)", "label": "MRD-guided therapeutic decisions (e.g., blinatumomab, allo-HCT)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic value of clonoSEQ in predicting relapse and remission", "label": "Prognostic value of clonoSEQ in predicting relapse and remission", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Use of MRD status post-induction to stratify patients", "label": "Use of MRD status post-induction to stratify patients", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Outcomes of Ph-like B-ALL patients based on MRD status", "label": "Outcomes of Ph-like B-ALL patients based on MRD status", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Role of NGS in deferring intensive therapies in MRD-negative patients", "label": "Role of NGS in deferring intensive therapies in MRD-negative patients", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Facts About Measurable Residual Disease (MRD)", "label": "Facts About Measurable Residual Disease (MRD)", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Facts About Measurable Residual Disease (MRD)\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e Not available (Fact Sheet by The Leukemia \u0026 Lymphoma Society, Revised October 2023)\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eMultiparameter flow cytometry (MFC) can detect 1 cancer cell in 10,000 to 100,000 nucleated cells and is applicable to nearly all patients with AML, ALL, CLL, and MM.\u003c/li\u003e\n  \u003cli\u003eReal-time quantitative PCR (RQ-PCR) detects MRD at a sensitivity of 1 in 100,000 cells and is effective for patients with specific genetic aberrations such as the BCR-ABL1 fusion gene.\u003c/li\u003e\n  \u003cli\u003eNext-generation sequencing (NGS)\u2013based assays, such as ClonoSEQ\u00ae, provide the highest sensitivity (\u003e10\u003csup\u003e\u22126\u003c/sup\u003e) and are FDA-cleared for MRD monitoring in B-ALL, CLL, and MM.\u003c/li\u003e\n  \u003cli\u003eIn acute lymphoblastic leukemia (ALL), achieving MRD negativity at the 10\u003csup\u003e\u22124\u003c/sup\u003e level is the strongest predictor of overall survival and leukemia-free survival across both pediatric and adult populations.\u003c/li\u003e\n  \u003cli\u003eFor AML, MRD positivity after complete remission is associated with significantly higher relapse risk and lower survival, regardless of detection method.\u003c/li\u003e\n  \u003cli\u003eEuropean LeukemiaNet (ELN) recommends MRD monitoring before and after hematopoietic stem-cell transplantation (HSCT) in AML to guide prognosis and clinical trial enrollment.\u003c/li\u003e\n  \u003cli\u003eIn multiple myeloma (MM), MRD negativity determined by MFC, NGS, MRI, or PET-CT is highly predictive of both progression-free and overall survival.\u003c/li\u003e\n  \u003cli\u003eAdvanced next-generation flow (NGF) in MM, using 8\u201310 color panels, achieves higher sensitivity than conventional MFC and improves prognostic accuracy.\u003c/li\u003e\n  \u003cli\u003eMRD negativity in MM can outweigh high-risk genetic features in prognostic value, reinforcing its central role in treatment monitoring.\u003c/li\u003e\n  \u003cli\u003eIn chronic myeloid leukemia (CML), RQ-PCR for BCR-ABL1 remains the gold standard for MRD monitoring and is used to define major molecular response and assess eligibility for therapy discontinuation.\u003c/li\u003e\n  \u003cli\u003eMRD results must always be interpreted in the context of assay sensitivity (e.g., 10\u003csup\u003e\u22124\u003c/sup\u003e, 10\u003csup\u003e\u22126\u003c/sup\u003e) as different technologies yield different thresholds for \u201cMRD negativity.\u201d\u003c/li\u003e\n  \u003cli\u003eBone marrow sampling quality (e.g., avoiding hemodilution) is critical for accurate MRD detection, especially for flow cytometry and molecular testing.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD detection technologies", "label": "MRD detection technologies", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sensitivity thresholds for MRD assays", "label": "Sensitivity thresholds for MRD assays", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Multiparameter flow cytometry (MFC)", "label": "Multiparameter flow cytometry (MFC)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Real-time quantitative PCR (RQ-PCR)", "label": "Real-time quantitative PCR (RQ-PCR)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Next-generation sequencing (NGS)", "label": "Next-generation sequencing (NGS)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD sampling techniques and best practices", "label": "MRD sampling techniques and best practices", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD in acute lymphoblastic leukemia (ALL)", "label": "MRD in acute lymphoblastic leukemia (ALL)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD in acute myeloid leukemia (AML)", "label": "MRD in acute myeloid leukemia (AML)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD in multiple myeloma (MM)", "label": "MRD in multiple myeloma (MM)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD in chronic myeloid leukemia (CML)", "label": "MRD in chronic myeloid leukemia (CML)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical utility and prognostic value of MRD negativity", "label": "Clinical utility and prognostic value of MRD negativity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "FDA-cleared assays (e.g., ClonoSEQ\u00ae)", "label": "FDA-cleared assays (e.g., ClonoSEQ\u00ae)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Use of MRD in post-transplant monitoring and treatment planning", "label": "Use of MRD in post-transplant monitoring and treatment planning", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Limitations and challenges in MRD interpretation", "label": "Limitations and challenges in MRD interpretation", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Guidelines from organizations like ELN and NCCN", "label": "Guidelines from organizations like ELN and NCCN", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "NGS vs. MFC MRD Sensitivity", "label": "NGS vs. MFC MRD Sensitivity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic Value of Early NGS MRD Negativity", "label": "Prognostic Value of Early NGS MRD Negativity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Risk Stratification in MFC-Negative Patients", "label": "Risk Stratification in MFC-Negative Patients", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Timing of MRD Assessment and Clinical Outcomes", "label": "Timing of MRD Assessment and Clinical Outcomes", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Impact of Baseline Cytogenetics on MRD Interpretation", "label": "Impact of Baseline Cytogenetics on MRD Interpretation", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "NGS MRD and AlloSCT Decision-Making", "label": "NGS MRD and AlloSCT Decision-Making", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic Utility of NGS in Non-Transplanted Patients", "label": "Prognostic Utility of NGS in Non-Transplanted Patients", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD Response in Lower-Intensity Chemotherapy Regimens", "label": "MRD Response in Lower-Intensity Chemotherapy Regimens", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Limitations of Late MRD Negativity", "label": "Limitations of Late MRD Negativity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Reevaluating Clinical MRD Thresholds with NGS Data", "label": "Reevaluating Clinical MRD Thresholds with NGS Data", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Interlaboratory Analytical Validation of a Next-Generation Sequencing Strategy for Clonotypic Assessment and Minimal Residual Disease Monitoring in Multiple Myeloma", "label": "Interlaboratory Analytical Validation of a Next-Generation Sequencing Strategy for Clonotypic Assessment and Minimal Residual Disease Monitoring in Multiple Myeloma", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cdiv\u003e\n  \u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Interlaboratory Analytical Validation of a Next-Generation Sequencing Strategy for Clonotypic Assessment and Minimal Residual Disease Monitoring in Multiple Myeloma\u003c/p\u003e\n  \u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.5858/arpa.2021-0088-OA\"\u003e10.5858/arpa.2021-0088-OA\u003c/a\u003e\u003c/p\u003e\n\n  \u003col\u003e\n    \u003cli\u003eThe LymphoTrack NGS assay demonstrated a limit of detection (LOD) of 1.3 myeloma cells per reaction and could detect MRD levels as low as 10\u207b\u2075, supporting its high analytical sensitivity.\u003c/li\u003e\n    \u003cli\u003eClonality detection using LymphoTrack was successful in 94% of baseline samples (95 of 101), with 97.9% concordance to conventional Sanger sequencing when using FR1 and FR2 primers.\u003c/li\u003e\n    \u003cli\u003eNGS using LymphoTrack detected 7 biallelic rearrangements and 1 double-unproductive rearrangement not identified by Sanger, highlighting superior detection of complex immunoglobulin gene rearrangements.\u003c/li\u003e\n    \u003cli\u003eInterlaboratory reproducibility was high: among 82 cases sequenced in both centers, MRD concordance was 85.4%, with strong correlation (R\u00b2 = 0.869) between labs.\u003c/li\u003e\n    \u003cli\u003eComparison between NGS and next-generation flow cytometry (NGF) revealed strong agreement in MRD quantification, with Bland-Altman plots showing less than 2-fold mean difference.\u003c/li\u003e\n    \u003cli\u003eIn 19 relapsed patients, MRD was positive by both NGS and NGF prior to clinical progression (mean lead time: 11.6 months), confirming the prognostic utility of both methods.\u003c/li\u003e\n    \u003cli\u003eWhile NGF achieved higher sensitivity (10\u207b\u2076) compared to NGS (10\u207b\u2075), no significant survival differences were observed between MRD-negative groups across methods, validating NGS as clinically comparable.\u003c/li\u003e\n    \u003cli\u003eDifferences in quantification precision (LOQ) were attributed to spike-in variability, DNA template amplification, and technical variation, suggesting the need for improved quantification standards in NGS.\u003c/li\u003e\n    \u003cli\u003eSamples with low plasma cell infiltration or polyclonal backgrounds showed improved clonality detection using NGS compared to Sanger sequencing, indicating better suitability of NGS for challenging cases.\u003c/li\u003e\n    \u003cli\u003eThe study supports the use of LymphoTrack as a validated alternative to FDA-approved ClonoSEQ for MRD monitoring in multiple myeloma, though further standardization is recommended for clinical deployment.\u003c/li\u003e\n  \u003c/ol\u003e\n\u003c/div\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "LymphoTrack NGS assay validation", "label": "LymphoTrack NGS assay validation", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clonality detection in multiple myeloma", "label": "Clonality detection in multiple myeloma", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Minimal residual disease (MRD) monitoring", "label": "Minimal residual disease (MRD) monitoring", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison with Sanger sequencing", "label": "Comparison with Sanger sequencing", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison with next-generation flow cytometry (NGF)", "label": "Comparison with next-generation flow cytometry (NGF)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Interlaboratory reproducibility and concordance", "label": "Interlaboratory reproducibility and concordance", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Detection limits and quantification precision (LOD and LOQ)", "label": "Detection limits and quantification precision (LOD and LOQ)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Technical challenges in DNA amplification and spike-in normalization", "label": "Technical challenges in DNA amplification and spike-in normalization", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic value of MRD detection and relapse prediction", "label": "Prognostic value of MRD detection and relapse prediction", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Standardization of NGS methods for clinical implementation", "label": "Standardization of NGS methods for clinical implementation", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "How I investigate minimal residual disease in acute lymphoblastic leukemia", "label": "How I investigate minimal residual disease in acute lymphoblastic leukemia", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cdiv\u003e\n  \u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e How I investigate minimal residual disease in acute lymphoblastic leukemia\u003c/p\u003e\n  \u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1111/ijlh.13463\"\u003e10.1111/ijlh.13463\u003c/a\u003e\u003c/p\u003e\n\n  \u003cul\u003e\n    \u003cli\u003eMinimal Residual Disease (MRD) as a prognostic marker in acute lymphoblastic leukemia (ALL)\u003c/li\u003e\n    \u003cli\u003eMultiparametric Flow Cytometry (MFC) techniques for MRD detection\u003c/li\u003e\n    \u003cli\u003eMolecular MRD detection methods, including quantitative PCR (qPCR) and digital PCR (dPCR)\u003c/li\u003e\n    \u003cli\u003eHigh-throughput sequencing (HTS/NGS) for immunoglobulin and T-cell receptor gene rearrangements\u003c/li\u003e\n    \u003cli\u003eComparison of sensitivity, specificity, and applicability across MRD detection methods\u003c/li\u003e\n    \u003cli\u003eMRD marker panels and immunophenotypic strategies in T-ALL and B-ALL\u003c/li\u003e\n    \u003cli\u003eImpact of immunotherapy (e.g., anti-CD19, anti-CD20) on MRD detection accuracy\u003c/li\u003e\n    \u003cli\u003ePeripheral blood versus bone marrow as sources for MRD monitoring\u003c/li\u003e\n    \u003cli\u003eStandardization and interlaboratory harmonization of MRD assays\u003c/li\u003e\n    \u003cli\u003eRegulatory approval and clinical integration of NGS-based MRD testing (e.g., ClonoSEQ)\u003c/li\u003e\n  \u003c/ul\u003e\n\u003c/div\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Minimal Residual Disease (MRD) as a prognostic marker in Acute Lymphoblastic Leukemia (ALL)", "label": "Minimal Residual Disease (MRD) as a prognostic marker in Acute Lymphoblastic Leukemia (ALL)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Multiparametric Flow Cytometry (MFC) for MRD detection", "label": "Multiparametric Flow Cytometry (MFC) for MRD detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Molecular assays for MRD detection (qPCR, digital PCR)", "label": "Molecular assays for MRD detection (qPCR, digital PCR)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "High-throughput sequencing (HTS/NGS) approaches in MRD monitoring", "label": "High-throughput sequencing (HTS/NGS) approaches in MRD monitoring", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of MRD methodologies: sensitivity, specificity, and clinical utility", "label": "Comparison of MRD methodologies: sensitivity, specificity, and clinical utility", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Immunophenotypic challenges post immunotherapy (e.g., CD19, CD20 loss)", "label": "Immunophenotypic challenges post immunotherapy (e.g., CD19, CD20 loss)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Use of peripheral blood vs. bone marrow for MRD evaluation", "label": "Use of peripheral blood vs. bone marrow for MRD evaluation", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD marker panels and gating strategies for T-ALL and B-ALL", "label": "MRD marker panels and gating strategies for T-ALL and B-ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Standardization and validation of MRD assays across laboratories", "label": "Standardization and validation of MRD assays across laboratories", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "FDA approval and clinical implementation of NGS-based MRD assays (e.g., ClonoSEQ)", "label": "FDA approval and clinical implementation of NGS-based MRD assays (e.g., ClonoSEQ)", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": " A Novel Single-Tube Next Generation Sequencing Assay for B-Cell Receptor Clonality Testing", "label": " A Novel Single-Tube Next Generation Sequencing Assay for B-Cell Receptor Clonality Testing", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e A Novel Single-Tube Next Generation Sequencing Assay for B-Cell Receptor Clonality Testing\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.3390/jmp5010004\"\u003e10.3390/jmp5010004\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eThe Oncomine BCR Pan-Clonality Assay identified clonal BCR rearrangements in 25 of 26 (96%) B-cell neoplastic cell lines, demonstrating high sensitivity with both heavy and light chain detection.\u003c/li\u003e\n  \u003cli\u003eIn clinical samples from multiple myeloma (MM) patients, the assay successfully detected at least one clonotype lineage in 42 out of 45 (93%) cases, affirming its utility for clonality detection in high SHM contexts.\u003c/li\u003e\n  \u003cli\u003eThe assay maintained high reproducibility with a Pearson correlation \u0026gt;0.95 between replicate libraries, confirming robustness across operators and sequencing runs.\u003c/li\u003e\n  \u003cli\u003eThe lower detection limit of the assay was determined to be approximately 1 in 10\u003csup\u003e6\u003c/sup\u003e cells, as shown by serial dilution experiments using known clonal cell lines spiked into polyclonal backgrounds.\u003c/li\u003e\n  \u003cli\u003eAmong MM patients, 94% with IGH clonotype lineages showed somatic hypermutation (SHM), with 39% exhibiting high SHM (\u0026gt;7%) and 18% showing very high SHM (\u0026gt;10%), supporting a post-germinal center origin hypothesis.\u003c/li\u003e\n  \u003cli\u003eTwo MM cases exhibited unusually low SHM (\u0026lt;1.5%) across the full IGH V gene sequence, suggesting potential pre-germinal center origins, which is atypical for MM.\u003c/li\u003e\n  \u003cli\u003eClonotype lineage analysis revealed evidence of ongoing SHM/intraclonal diversity in 18% of MM patients, supporting a model of active clonal evolution in a subset of cases.\u003c/li\u003e\n  \u003cli\u003eBiased IGHV segment usage was observed, with IGHV3-11 and IGHV4-34 disproportionately represented among dominant clones in 20% of MM cases, indicating potential biological or prognostic relevance.\u003c/li\u003e\n  \u003cli\u003eThe assay demonstrated stable performance across DNA input concentrations from 1 ng to 500 ng, showing linear detection and frequency consistency even with low-input samples.\u003c/li\u003e\n  \u003cli\u003eNo false-positive clonotype detection was reported in polyclonal samples, highlighting high specificity, making it suitable for clinical and research applications in lymphoproliferative disorders.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "B-cell receptor (BCR) clonality detection", "label": "B-cell receptor (BCR) clonality detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Next-generation sequencing (NGS) assay development", "label": "Next-generation sequencing (NGS) assay development", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Multiple myeloma (MM) diagnostics", "label": "Multiple myeloma (MM) diagnostics", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Somatic hypermutation (SHM) analysis", "label": "Somatic hypermutation (SHM) analysis", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Intraclonal diversity and clonal evolution", "label": "Intraclonal diversity and clonal evolution", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "IGHV gene usage bias (e.g., IGHV3-11, IGHV4-34)", "label": "IGHV gene usage bias (e.g., IGHV3-11, IGHV4-34)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Assay sensitivity and specificity", "label": "Assay sensitivity and specificity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Single-tube multiplex PCR methods", "label": "Single-tube multiplex PCR methods", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Detection limits in clonality testing", "label": "Detection limits in clonality testing", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Reproducibility of NGS-based clonotyping", "label": "Reproducibility of NGS-based clonotyping", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": " LymphoTrack Is Equally Sensitive as PCR GeneScan and Sanger Sequencing for Detection of Clonal Rearrangements in ALL Patients", "label": " LymphoTrack Is Equally Sensitive as PCR GeneScan and Sanger Sequencing for Detection of Clonal Rearrangements in ALL Patients", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e LymphoTrack Is Equally Sensitive as PCR GeneScan and Sanger Sequencing for Detection of Clonal Rearrangements in ALL Patients\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.3390/diagnostics12061389\"\u003e10.3390/diagnostics12061389\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eLymphoTrack detected 100% of IGH clonal rearrangements (85/85) identified by PCR GeneScan and Sanger sequencing, demonstrating complete concordance for this marker.\u003c/li\u003e\n  \u003cli\u003eThe detection rate for IGK rearrangements was 96% (64/67), with discrepancies attributed to low peaks or polyclonal backgrounds in traditional methods.\u003c/li\u003e\n  \u003cli\u003eTRG rearrangements were detected in 98% (91/93) of cases by LymphoTrack, with some rearrangements confirmed using orthogonal assays due to heavy gene trimming affecting primer binding.\u003c/li\u003e\n  \u003cli\u003eLymphoTrack identified 97% (34/35) of TCRB rearrangements, with one rearrangement not meeting interpretation thresholds due to low amplification.\u003c/li\u003e\n  \u003cli\u003eIn 93% of IGH samples with \u22652 clones, the rearrangement with the highest %Total reads in LymphoTrack matched the highest RFU in PCR GeneScan, showing good correlation between NGS read counts and traditional signal intensity.\u003c/li\u003e\n  \u003cli\u003eLymphoTrack detected dual or ambiguous clonal rearrangements in several cases that were indistinguishable by GeneScan without cloning, showing NGS\u2019s resolution advantage.\u003c/li\u003e\n  \u003cli\u003eIn some cases, LymphoTrack identified clonal rearrangements missed by GeneScan, often due to primer mismatches or trimmed junctional regions preventing amplification.\u003c/li\u003e\n  \u003cli\u003eThe turnaround time for clonality assessment was reduced from ~2 weeks using traditional methods to 3\u20135 days with LymphoTrack, making it more suitable for clinical workflows.\u003c/li\u003e\n  \u003cli\u003eLymphoTrack showed consistent results across IGH, IGK, TRG, and TRB assays, reinforcing its applicability for comprehensive clonality screening in ALL.\u003c/li\u003e\n  \u003cli\u003eAlthough LymphoTrack failed to detect incomplete IGH or TCRD rearrangements due to current kit limitations, its performance on complete rearrangements supports its use as a primary screening method, supplemented by traditional methods when necessary.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clonal rearrangement detection in acute lymphoblastic leukemia (ALL)", "label": "Clonal rearrangement detection in acute lymphoblastic leukemia (ALL)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of LymphoTrack with PCR GeneScan and Sanger sequencing", "label": "Comparison of LymphoTrack with PCR GeneScan and Sanger sequencing", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sensitivity and specificity of next-generation sequencing (NGS) assays", "label": "Sensitivity and specificity of next-generation sequencing (NGS) assays", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Validation of IGH, IGK, TRG, and TCRB rearrangements", "label": "Validation of IGH, IGK, TRG, and TCRB rearrangements", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Correlation between %Total reads (NGS) and RFU (PCR GeneScan)", "label": "Correlation between %Total reads (NGS) and RFU (PCR GeneScan)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Identification of ambiguous and polyclonal rearrangements", "label": "Identification of ambiguous and polyclonal rearrangements", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Limitations of current LymphoTrack assays for incomplete IGH and TCRD", "label": "Limitations of current LymphoTrack assays for incomplete IGH and TCRD", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Advantages of NGS over traditional methods in turnaround time and resolution", "label": "Advantages of NGS over traditional methods in turnaround time and resolution", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Implementation considerations for LymphoTrack in clinical diagnostics", "label": "Implementation considerations for LymphoTrack in clinical diagnostics", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Utility of ASO design for MRD monitoring in ALL", "label": "Utility of ASO design for MRD monitoring in ALL", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Multiparameter Flow Cytometry and ClonoSEQ Correlation to Evaluate Precursor B-Lymphoblastic Leukemia Measurable Residual Disease", "label": "Multiparameter Flow Cytometry and ClonoSEQ Correlation to Evaluate Precursor B-Lymphoblastic Leukemia Measurable Residual Disease", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Multiparameter Flow Cytometry and ClonoSEQ Correlation to Evaluate Precursor B-Lymphoblastic Leukemia Measurable Residual Disease\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1007/s12308-023-00544-9\"\u003e10.1007/s12308-023-00544-9\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eCOG multiparameter flow cytometry (MFC) and ClonoSEQ results were concordant in 80% (59/74) of B-ALL bone marrow samples, showing high agreement in MRD detection.\u003c/li\u003e\n  \u003cli\u003eDiscordant results were seen in 20% of cases, with 19% being ClonoSEQ+/MFC\u2212 and only 1% MFC+/ClonoSEQ\u2212, indicating greater sensitivity of ClonoSEQ in low MRD settings.\u003c/li\u003e\n  \u003cli\u003eClonoSEQ detected MRD as low as 1 clonal cell per million nucleated cells, with 86% of discordant ClonoSEQ+/MFC\u2212 cases showing \u003c100 cells/million, underscoring its ultra-high sensitivity.\u003c/li\u003e\n  \u003cli\u003eClonoSEQ revealed newly emerging dominant Ig sequences in 6% (2/31) of patients during follow-up, suggesting possible clonal evolution in B-ALL over time.\u003c/li\u003e\n  \u003cli\u003eIn patients treated with blinatumomab, all 14 samples were MFC-negative for MRD, but 3 samples remained ClonoSEQ-positive, suggesting MFC may miss MRD in CD19-negative or low-burden scenarios post-immunotherapy.\u003c/li\u003e\n  \u003cli\u003eStrong correlation (r = 0.96) was found between tumor burden values measured by MFC and ClonoSEQ, supporting their use as complementary diagnostic tools.\u003c/li\u003e\n  \u003cli\u003eCD38 and CD10 expression levels were identified as key discriminators in MFC for distinguishing B-lymphoblasts from hematogones, particularly in MRD evaluation contexts.\u003c/li\u003e\n  \u003cli\u003eNGS-based MRD methods like ClonoSEQ offer greater sensitivity (up to 10\u207b\u2077) and the ability to track multiple receptor sequences, enhancing detection robustness compared to MFC.\u003c/li\u003e\n  \u003cli\u003eThe requirement of a diagnostic sample for clonotype identification remains a limitation for ClonoSEQ, as it may not detect MRD in cases lacking baseline dominant sequences.\u003c/li\u003e\n  \u003cli\u003eThe study concludes that both ClonoSEQ and MFC should be used in tandem for accurate MRD monitoring in B-ALL, especially under therapy-induced phenotypic shifts or clonal changes.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Measurable residual disease (MRD) monitoring in precursor B-ALL", "label": "Measurable residual disease (MRD) monitoring in precursor B-ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of multiparameter flow cytometry (MFC) and ClonoSEQ", "label": "Comparison of multiparameter flow cytometry (MFC) and ClonoSEQ", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sensitivity and specificity of NGS-based MRD detection", "label": "Sensitivity and specificity of NGS-based MRD detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Discordance between MFC and ClonoSEQ results", "label": "Discordance between MFC and ClonoSEQ results", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Impact of blinatumomab immunotherapy on MRD detection", "label": "Impact of blinatumomab immunotherapy on MRD detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clonal evolution and emergence of new dominant sequences", "label": "Clonal evolution and emergence of new dominant sequences", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Interpretation challenges in low MRD settings", "label": "Interpretation challenges in low MRD settings", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Phenotypic markers in B-ALL MRD evaluation (CD10, CD38, CD19)", "label": "Phenotypic markers in B-ALL MRD evaluation (CD10, CD38, CD19)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Diagnostic limitations of ClonoSEQ requiring baseline samples", "label": "Diagnostic limitations of ClonoSEQ requiring baseline samples", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Complementary use of MFC and NGS for comprehensive MRD assessment", "label": "Complementary use of MFC and NGS for comprehensive MRD assessment", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Next-generation sequencing\u2013based MRD in adults with ALL undergoing hematopoietic cell transplantation", "label": "Next-generation sequencing\u2013based MRD in adults with ALL undergoing hematopoietic cell transplantation", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cdiv\u003e\n  \u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Next-generation sequencing\u2013based MRD in adults with ALL undergoing hematopoietic cell transplantation\u003c/p\u003e\n  \u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1182/bloodadvances.2023009856\" target=\"_blank\"\u003e10.1182/bloodadvances.2023009856\u003c/a\u003e\u003c/p\u003e\n  \u003col\u003e\n    \u003cli\u003eEven low-level measurable residual disease (MRD) before hematopoietic cell transplantation (HCT), specifically below 10\u003csup\u003e\u20134\u003c/sup\u003e, was significantly associated with increased post-HCT relapse risk (HR = 3.56; 95% CI: 1.39\u20139.15).\u003c/li\u003e\n    \u003cli\u003eDetectable MRD after HCT (MRD\u003csub\u003epost\u003c/sub\u003e), regardless of the level, was the strongest predictor of relapse (HR = 4.60; 95% CI: 3.01\u20137.02) and leukemia-specific mortality (HR = 9.85; 95% CI: 4.97\u201319.54).\u003c/li\u003e\n    \u003cli\u003eIn B-cell ALL, detection of IgH clonotypes post-HCT (but not non-IgH such as Ig\u03ba/\u03bb) was strongly associated with relapse, suggesting IgH-specific MRD is more prognostically relevant.\u003c/li\u003e\n    \u003cli\u003ePatients with undetectable MRD both before and after HCT had the lowest relapse rates, supporting the value of deep MRD clearance to a sensitivity of 10\u003csup\u003e\u20136\u003c/sup\u003e prior to transplant.\u003c/li\u003e\n    \u003cli\u003eMRD monitoring using peripheral blood showed high concordance with bone marrow results (Pearson correlation coefficient = 0.78), supporting its clinical use as a less invasive and more frequent monitoring approach.\u003c/li\u003e\n    \u003cli\u003eAmong patients with detectable post-HCT IgH MRD, 61% relapsed, whereas only 21% of those with non-IgH MRD relapsed, highlighting the lower clinical significance of non-IgH clonotype detection.\u003c/li\u003e\n    \u003cli\u003eThe study demonstrated that even patients with moderate pre-HCT MRD levels (\u003c10\u003csup\u003e\u20134\u003c/sup\u003e to \u226410\u003csup\u003e\u20133\u003c/sup\u003e) had significantly increased relapse risk (HR = 1.81; 95% CI: 1.16\u20132.82), reaffirming the importance of achieving MRD-negativity.\u003c/li\u003e\n    \u003cli\u003ePatients with very high pre-HCT MRD (\u003e10\u003csup\u003e\u20133\u003c/sup\u003e) were at the highest risk of relapse post-HCT (HR = 3.38; 95% CI: 1.84\u20136.19), suggesting that MRD burden before HCT stratifies relapse risk quantitatively.\u003c/li\u003e\n    \u003cli\u003eNGS-based MRD assessments from peripheral blood allowed for more frequent monitoring and may enable earlier detection of relapse in clinical practice.\u003c/li\u003e\n    \u003cli\u003eFindings support integrating NGS-based MRD assessment in clinical decision-making both before and after HCT to guide interventions such as additional therapy or closer monitoring.\u003c/li\u003e\n  \u003c/ol\u003e\n\u003c/div\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic significance of pre- and post-HCT MRD in ALL", "label": "Prognostic significance of pre- and post-HCT MRD in ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Risk stratification based on MRD levels (\u003c10\u207b\u2074, \u226510\u207b\u00b3)", "label": "Risk stratification based on MRD levels (\u003c10\u207b\u2074, \u226510\u207b\u00b3)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "IgH vs non-IgH clonotypes in relapse prediction", "label": "IgH vs non-IgH clonotypes in relapse prediction", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Peripheral blood vs bone marrow MRD monitoring", "label": "Peripheral blood vs bone marrow MRD monitoring", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Post-HCT relapse and leukemia-specific mortality", "label": "Post-HCT relapse and leukemia-specific mortality", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical utility of MRD in treatment and monitoring decisions", "label": "Clinical utility of MRD in treatment and monitoring decisions", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Implications for MRD-guided therapeutic strategies before HCT", "label": "Implications for MRD-guided therapeutic strategies before HCT", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Next-generation sequencing for MRD monitoring in B-lineage malignancies: from bench to bedside", "label": "Next-generation sequencing for MRD monitoring in B-lineage malignancies: from bench to bedside", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Next-generation sequencing for MRD monitoring in B-lineage malignancies: from bench to bedside\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1186/s40164-022-00300-2\"\u003e10.1186/s40164-022-00300-2\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eNext-generation sequencing (NGS)-based immunoglobulin (IG) clonality assays achieve a higher sensitivity (up to 10\u003csup\u003e\u22126\u003c/sup\u003e) than conventional MRD monitoring methods such as ASO-PCR and multicolor flow cytometry.\u003c/li\u003e\n  \u003cli\u003eNGS-based MRD monitoring allows comprehensive tracking of clonal evolution and immunogenetic profiling in B-lineage malignancies through V(D)J recombination analysis.\u003c/li\u003e\n  \u003cli\u003eIGH rearrangement patterns differ by disease subtype: ALL displays unmutated IGH due to immature B cell origin, while MCL, CLL, and FL show biased VH usage and stereotyped BCRs with diagnostic and prognostic relevance.\u003c/li\u003e\n  \u003cli\u003eIn CLL, MRD and prognosis correlate with IGHV mutational status: unmutated CLL (U-CLL) shows more aggressive behavior compared to mutated CLL (M-CLL), with implications for therapy selection and outcome prediction.\u003c/li\u003e\n  \u003cli\u003eNGS-based MRD assays such as LymphoTrack and ClonoSeq have standardized workflows and are FDA-approved for monitoring MRD in ALL and MM, supporting their clinical utility.\u003c/li\u003e\n  \u003cli\u003eThe identification of \u201cindex clones\u201d (dominant V(D)J rearrangements in diagnostic samples) enables longitudinal MRD tracking and quantification of clonal burden during and after treatment.\u003c/li\u003e\n  \u003cli\u003eMRD negativity defined by a sensitivity of 10\u003csup\u003e\u22126\u003c/sup\u003e (1 cell in 1,000,000) correlates with improved overall and progression-free survival, surpassing older thresholds (e.g., 10\u003csup\u003e\u22125\u003c/sup\u003e).\u003c/li\u003e\n  \u003cli\u003eNGS-based MRD tracking is capable of distinguishing relapse patterns such as early vs. late divergent clonal evolution in diseases like DLBCL, offering mechanistic insights into resistance and recurrence.\u003c/li\u003e\n  \u003cli\u003eTechnical limitations affecting MRD sensitivity include DNA input quantity, bone marrow sample quality, and overestimation biases due to polyclonal background amplification.\u003c/li\u003e\n  \u003cli\u003eDespite clear advantages, NGS-based MRD monitoring faces adoption challenges including high cost, lack of inclusion in clinical trials for novel therapies, and variability in MRD negativity definitions across labs.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Minimal Residual Disease (MRD) monitoring in hematologic malignancies", "label": "Minimal Residual Disease (MRD) monitoring in hematologic malignancies", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Next-Generation Sequencing (NGS)-based clonality assays", "label": "Next-Generation Sequencing (NGS)-based clonality assays", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "V(D)J recombination and immunoglobulin gene rearrangements", "label": "V(D)J recombination and immunoglobulin gene rearrangements", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Disease-specific IG rearrangement patterns in B-lineage cancers (e.g., ALL, CLL, MCL, DLBCL, FL, MM)", "label": "Disease-specific IG rearrangement patterns in B-lineage cancers (e.g., ALL, CLL, MCL, DLBCL, FL, MM)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clonal evolution and its clinical implications", "label": "Clonal evolution and its clinical implications", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic significance of IGHV mutation status", "label": "Prognostic significance of IGHV mutation status", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Technical methodology and sensitivity thresholds for MRD detection", "label": "Technical methodology and sensitivity thresholds for MRD detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "FDA-approved NGS-based MRD platforms (e.g., ClonoSeq)", "label": "FDA-approved NGS-based MRD platforms (e.g., ClonoSeq)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Limitations and barriers to clinical adoption of NGS MRD monitoring", "label": "Limitations and barriers to clinical adoption of NGS MRD monitoring", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Integration of MRD data into risk stratification and treatment decisions", "label": "Integration of MRD data into risk stratification and treatment decisions", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Measurable Residual Disease (MRD) in Acute Lymphoblastic Leukemia (ALL)", "label": "Measurable Residual Disease (MRD) in Acute Lymphoblastic Leukemia (ALL)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of MRD detection methods: Flow Cytometry (MFC) vs. Next-Generation Sequencing (NGS)", "label": "Comparison of MRD detection methods: Flow Cytometry (MFC) vs. Next-Generation Sequencing (NGS)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic significance of MRD negativity at various time points", "label": "Prognostic significance of MRD negativity at various time points", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical utility of high-sensitivity NGS MRD assays", "label": "Clinical utility of high-sensitivity NGS MRD assays", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Risk stratification and relapse prediction in ALL", "label": "Risk stratification and relapse prediction in ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Impact of treatment regimens (hyper-CVAD vs. mini\u2013hyper-CVD with inotuzumab)", "label": "Impact of treatment regimens (hyper-CVAD vs. mini\u2013hyper-CVD with inotuzumab)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Implications for allogeneic stem cell transplantation (alloSCT) decisions", "label": "Implications for allogeneic stem cell transplantation (alloSCT) decisions", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Limitations of conventional MRD thresholds and timing", "label": "Limitations of conventional MRD thresholds and timing", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Treatment de-escalation potential based on early deep remission", "label": "Treatment de-escalation potential based on early deep remission", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD as a biomarker for therapy guidance in frontline ALL management", "label": "MRD as a biomarker for therapy guidance in frontline ALL management", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Perspective: Sensitive Detection of Residual Lymphoproliferative Disease by NGS and Clonal Rearrangements\u2014How Low Can You Go?", "label": "Perspective: Sensitive Detection of Residual Lymphoproliferative Disease by NGS and Clonal Rearrangements\u2014How Low Can You Go?", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cb\u003eArticle Title:\u003c/b\u003e Perspective: Sensitive Detection of Residual Lymphoproliferative Disease by NGS and Clonal Rearrangements\u2014How Low Can You Go?\u003c/p\u003e\n\u003cp\u003e\u003cb\u003eDOI:\u003c/b\u003e 10.1016/j.exphem.2021.03.005\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eNGS-based MRD assays can theoretically achieve sensitivity levels as low as 10\u003csup\u003e\u22127\u003c/sup\u003e, but reaching such levels in practice depends heavily on input DNA amount, sequencing depth, and amplification efficiency.\u003c/li\u003e\n  \u003cli\u003eThe randomness of subsampling introduces Poisson-based error, which significantly affects detection reliability when working with low target abundance, especially below 10\u003csup\u003e\u22126\u003c/sup\u003e MRD levels.\u003c/li\u003e\n  \u003cli\u003eUnlike PCR-based methods, NGS allows for multiplexed detection of multiple clonal rearrangements without patient-specific primers, enhancing specificity and scalability of MRD monitoring.\u003c/li\u003e\n  \u003cli\u003eBiological variables, such as the presence of immature or incomplete rearrangements, may limit the ability to detect a clonal marker, highlighting the need for clonotype identification at diagnosis.\u003c/li\u003e\n  \u003cli\u003eStandard sensitivity estimation methods using serial dilutions can overestimate practical sensitivity in clinical settings due to variance and material loss across assay steps.\u003c/li\u003e\n  \u003cli\u003eThere is no consensus on acceptable mismatch thresholds for defining clonotypes, with some commercial assays allowing up to two base mismatches, which may affect assay specificity.\u003c/li\u003e\n  \u003cli\u003eImplementing replicates (technical or biological) can significantly improve the robustness of MRD detection, particularly in high-sensitivity applications, by mitigating stochastic dropout.\u003c/li\u003e\n  \u003cli\u003eDNA from peripheral blood, bone marrow, and ctDNA may yield differing MRD results, with bone marrow often being more reliable but ctDNA offering a less invasive alternative despite lower correlation in some malignancies.\u003c/li\u003e\n  \u003cli\u003eUse of formalin-fixed paraffin-embedded (FFPE) tissues is not recommended for high-sensitivity MRD assays due to DNA degradation and reduced sequencing coverage.\u003c/li\u003e\n  \u003cli\u003eStandardization initiatives, such as those by the EuroClonality-NGS consortium, are critical to ensure reproducibility and clinical implementation of NGS-based MRD testing across platforms and laboratories.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sensitivity thresholds for MRD detection using NGS", "label": "Sensitivity thresholds for MRD detection using NGS", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clonal gene rearrangements as MRD targets (IGH, TCR)", "label": "Clonal gene rearrangements as MRD targets (IGH, TCR)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Technical limitations: DNA input, sequencing depth, and PCR efficiency", "label": "Technical limitations: DNA input, sequencing depth, and PCR efficiency", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Poisson sampling and stochastic errors in low-input scenarios", "label": "Poisson sampling and stochastic errors in low-input scenarios", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Standardization and reproducibility challenges in NGS MRD assays", "label": "Standardization and reproducibility challenges in NGS MRD assays", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of NGS with qPCR, ASO-qPCR, and flow cytometry", "label": "Comparison of NGS with qPCR, ASO-qPCR, and flow cytometry", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sample type considerations (bone marrow, peripheral blood, ctDNA, RNA)", "label": "Sample type considerations (bone marrow, peripheral blood, ctDNA, RNA)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Impact of biological variables (cell sorting, clonality, rearrangement completeness)", "label": "Impact of biological variables (cell sorting, clonality, rearrangement completeness)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Use of commercial platforms (LymphoTrack, clonoSEQ)", "label": "Use of commercial platforms (LymphoTrack, clonoSEQ)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Recommendations for assay validation and transparent reporting", "label": "Recommendations for assay validation and transparent reporting", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Prospective evaluation of minimal residual disease in the phase II FORTE trial: a head-to-head comparison between multiparameter flow cytometry and next-generation sequencing", "label": "Prospective evaluation of minimal residual disease in the phase II FORTE trial: a head-to-head comparison between multiparameter flow cytometry and next-generation sequencing", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cb\u003eArticle Title:\u003c/b\u003e Prospective evaluation of minimal residual disease in the phase II FORTE trial: a head-to-head comparison between multiparameter flow cytometry and next-generation sequencing\u003c/p\u003e\n\u003cp\u003e\u003cb\u003eDOI:\u003c/b\u003e 10.1016/j.eclinm.2023.102016\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eIn a large sample from the FORTE trial, biological concordance between multiparameter flow cytometry (MFC) and next-generation sequencing (NGS) for MRD detection was 87% at 10\u003csup\u003e\u22125\u003c/sup\u003e sensitivity and 83% at 10\u003csup\u003e\u22126\u003c/sup\u003e, supporting their comparable utility in clinical practice.\u003c/li\u003e\n\n  \u003cli\u003ePrognostic concordance was confirmed, as MRD-negative patients showed significantly improved progression-free survival (PFS) and overall survival (OS) by both MFC and NGS: HRs for PFS were 0.29 (MFC) and 0.27 (NGS), and for OS were 0.35 (MFC) and 0.31 (NGS).\u003c/li\u003e\n\n  \u003cli\u003eSustained MRD negativity over 1 or 2 years strongly correlated with improved outcomes; 2-year MRD-negative patients had 4-year PFS rates of 99% (MFC) and 97% (NGS).\u003c/li\u003e\n\n  \u003cli\u003eThe conversion rate from MRD positivity to negativity during maintenance was significantly higher with carfilzomib-lenalidomide (KR) versus lenalidomide alone (R), as observed by both MFC (46% vs 30%) and NGS (56% vs 30%), p = 0.046.\u003c/li\u003e\n\n  \u003cli\u003eMRD-negative patients in both KRd plus ASCT and KRd12 arms showed comparable PFS, suggesting potential flexibility in treatment approach if MRD negativity is achieved.\u003c/li\u003e\n\n  \u003cli\u003eNGS detected more MRD-positive cases than MFC in discordant samples; 12% of MFC-negative samples were NGS-positive at 10\u003csup\u003e\u22125\u003c/sup\u003e, suggesting greater sensitivity and stability of the gDNA-based method.\u003c/li\u003e\n\n  \u003cli\u003eClinical benefit of MRD negativity was consistent across patient subgroups, including high-risk cytogenetics, ISS stages, and LDH levels, with no significant interactions affecting the prognostic value.\u003c/li\u003e\n\n  \u003cli\u003ePatients who converted to MRD negativity during maintenance therapy had significantly higher 4-year PFS than those with persistent MRD positivity: 73% vs 44% (MFC) and 84% vs 55% (NGS).\u003c/li\u003e\n\n  \u003cli\u003eAchieving sustained MRD negativity could potentially guide decisions on treatment deintensification or discontinuation, supporting MRD as a surrogate for long-term outcomes in MM management.\u003c/li\u003e\n\n  \u003cli\u003eThe study highlights the practicality and prognostic power of both MFC and NGS for MRD monitoring and calls for their standardized integration into treatment planning and clinical trial design for multiple myeloma.\u003c/li\u003e\n\u003c/ol\u003e\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Head-to-head comparison of MFC and NGS for MRD detection", "label": "Head-to-head comparison of MFC and NGS for MRD detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Concordance and discordance rates between MFC and NGS at 10\u207b\u2075 and 10\u207b\u2076 sensitivities", "label": "Concordance and discordance rates between MFC and NGS at 10\u207b\u2075 and 10\u207b\u2076 sensitivities", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic impact of MRD status on progression-free and overall survival", "label": "Prognostic impact of MRD status on progression-free and overall survival", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sustained MRD negativity (1-year and 2-year) as a strong predictor of long-term outcomes", "label": "Sustained MRD negativity (1-year and 2-year) as a strong predictor of long-term outcomes", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Effect of maintenance therapy (KR vs R) on MRD conversion rates", "label": "Effect of maintenance therapy (KR vs R) on MRD conversion rates", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Implications of discordant MRD results and sample handling variables", "label": "Implications of discordant MRD results and sample handling variables", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Stratification of patients by cytogenetic risk, ISS stage, and treatment arms", "label": "Stratification of patients by cytogenetic risk, ISS stage, and treatment arms", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Use of MRD status to guide therapy deintensification or discontinuation", "label": "Use of MRD status to guide therapy deintensification or discontinuation", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Feasibility and limitations of achieving 10\u207b\u2076 sensitivity with current methods", "label": "Feasibility and limitations of achieving 10\u207b\u2076 sensitivity with current methods", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Role of MRD testing in clinical trial design and regulatory approval", "label": "Role of MRD testing in clinical trial design and regulatory approval", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": " Real-life sensitivity of flow cytometry minimal residual disease assessment for plasma cell neoplasms", "label": " Real-life sensitivity of flow cytometry minimal residual disease assessment for plasma cell neoplasms", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cb\u003eArticle Title:\u003c/b\u003e Real-life sensitivity of flow cytometry minimal residual disease assessment for plasma cell neoplasms\u003c/p\u003e\n\u003cp\u003e\u003cb\u003eDOI:\u003c/b\u003e 10.1038/s41408-024-01113-8\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eThe real-world sensitivity of flow cytometry (FC) MRD assessment in plasma cell neoplasms was better than the IMWG-defined minimum threshold of 10\u003csup\u003e\u22125\u003c/sup\u003e, with a median sensitivity of 2.4 \u00d7 10\u003csup\u003e\u22126\u003c/sup\u003e for internal samples and 2.8 \u00d7 10\u003csup\u003e\u22126\u003c/sup\u003e for external samples.\u003c/li\u003e\n\n  \u003cli\u003eAmong 3137 internally collected bone marrow samples, 75.9% were MRD-negative, 0.6% were MRD-indeterminate, and 23.5% were MRD-positive; external samples had a higher MRD positivity rate of 43.3%.\u003c/li\u003e\n\n  \u003cli\u003eLarge MRD-positive clones (\u0026gt;0.1% of total events) were found in 15.7% of external samples compared to only 0.3% of internal samples, suggesting possible differences in patient selection or disease burden.\u003c/li\u003e\n\n  \u003cli\u003eSamples prescreened using a lower-sensitivity 7-color proliferation assay (PCPRO) allowed selective reflexing of PCPRO-negative cases to the high-sensitivity NGF assay, optimizing resource use without compromising detection.\u003c/li\u003e\n\n  \u003cli\u003eThe median number of collected events per sample was high (8.3 \u00d7 10\u003csup\u003e6\u003c/sup\u003e internal, 7.0 \u00d7 10\u003csup\u003e6\u003c/sup\u003e external), contributing to the improved sensitivity and highlighting the importance of event acquisition volume in FC MRD testing.\u003c/li\u003e\n\n  \u003cli\u003eThere was no statistically significant difference in hemodilution indicators (mast cell and hematogone percentages) between internal and external samples, confirming comparable specimen quality across sources.\u003c/li\u003e\n\n  \u003cli\u003eFC MRD testing was feasible up to 72 hours post-collection in external specimens, reinforcing its viability in reference lab settings with broad geographic sample intake.\u003c/li\u003e\n\n  \u003cli\u003eAlthough no direct comparison was made with NGS, the results support that FC MRD, when optimized and standardized (as with NGF), can approach NGS-level sensitivity in real-world use.\u003c/li\u003e\n\n  \u003cli\u003eThe study reinforces the clinical utility of NGF in MRD detection for multiple myeloma, particularly when paired with efficient prescreening workflows like PCPRO to reduce unnecessary testing.\u003c/li\u003e\n\n  \u003cli\u003eFindings support the continued integration of high-sensitivity FC MRD assays into routine MM monitoring, especially in settings where turnaround time, cost, and specimen freshness are practical concerns.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Real-world sensitivity of flow cytometry (FC) for MRD detection in plasma cell neoplasms", "label": "Real-world sensitivity of flow cytometry (FC) for MRD detection in plasma cell neoplasms", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Performance of EuroFlow next-generation flow (NGF) assay", "label": "Performance of EuroFlow next-generation flow (NGF) assay", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Use of prescreening with lower-sensitivity PCPRO assay", "label": "Use of prescreening with lower-sensitivity PCPRO assay", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of internal vs. external sample characteristics and outcomes", "label": "Comparison of internal vs. external sample characteristics and outcomes", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Event acquisition volume and its impact on assay sensitivity", "label": "Event acquisition volume and its impact on assay sensitivity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Rates of MRD positivity, negativity, and indeterminacy", "label": "Rates of MRD positivity, negativity, and indeterminacy", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Viability of FC MRD testing with delayed sample processing (up to 72 hours)", "label": "Viability of FC MRD testing with delayed sample processing (up to 72 hours)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Differences in clone sizes between sample sources", "label": "Differences in clone sizes between sample sources", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Lack of significant hemodilution differences between sample groups", "label": "Lack of significant hemodilution differences between sample groups", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Operational implications for integrating FC MRD testing in routine MM monitoring workflows", "label": "Operational implications for integrating FC MRD testing in routine MM monitoring workflows", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Evaluation of next-generation sequencing versus next-generation flow cytometry for minimal-residual-disease detection in Chinese patients with multiple myeloma", "label": "Evaluation of next-generation sequencing versus next-generation flow cytometry for minimal-residual-disease detection in Chinese patients with multiple myeloma", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Evaluation of next-generation sequencing versus next-generation flow cytometry for minimal-residual-disease detection in Chinese patients with multiple myeloma\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1007/s12672-024-00938-w\"\u003e10.1007/s12672-024-00938-w\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eNGS demonstrated a wide MRD detection range of 10\u207b\u2076 to 10\u207b\u00b9 with excellent linearity (R\u00b2 = 0.985) and a lower limit of detection of 10\u207b\u2076, indicating very high sensitivity for MRD detection.\u003c/li\u003e\n  \u003cli\u003eIntra- and inter-assay reproducibility of the NGS method showed 100% consistency across replicates, with low variation (coefficient of variation 6.13\u201310.85%), confirming the method\u2019s reliability.\u003c/li\u003e\n  \u003cli\u003eClonal IGH rearrangements were successfully identified in 42 of 60 patients (70%) at diagnosis using NGS, establishing clonality markers critical for MRD tracking.\u003c/li\u003e\n  \u003cli\u003eNGS-based MRD monitoring showed 79.1% concordance with NGF across 43 tested samples, but detected MRD in 9 cases where NGF did not, demonstrating its superior sensitivity.\u003c/li\u003e\n  \u003cli\u003eNone of the MRD-positive cases identified by NGF were missed by NGS, reinforcing NGS\u2019s robustness and reliability as a diagnostic tool.\u003c/li\u003e\n  \u003cli\u003ePrior to autologous stem cell transplantation (ASCT), NGS detected MRD in 6 of 7 cases, compared to only 1 case by NGF, highlighting NGS\u2019s advantage in early MRD detection.\u003c/li\u003e\n  \u003cli\u003ePost-ASCT, 4 of the 6 previously NGS-MRD-positive patients became MRD-negative, demonstrating treatment response; all post-ASCT samples were NGF-negative, despite some being NGS-positive, suggesting NGS\u2019s superior sensitivity.\u003c/li\u003e\n  \u003cli\u003eUnique IGH CDR3 sequences detected by NGS were individualized for each patient, enabling personalized and precise tracking of residual disease.\u003c/li\u003e\n  \u003cli\u003eClonal IGH rearrangements were significantly more common in heavy-chain myeloma than in light-chain cases (P = 0.0012), suggesting subtype-specific clonality patterns.\u003c/li\u003e\n  \u003cli\u003eNGS and CE (capillary electrophoresis) had a high overall concordance rate of 98.3% for detecting clonal IGH rearrangements, confirming NGS\u0027s validity against established methods.\u003c/li\u003e\n  \u003cli\u003eNGS allows MRD detection at very low tumor burdens (10\u207b\u2076), exceeding NGF\u2019s sensitivity (10\u207b\u2075), which can be clinically critical in identifying early relapse or residual disease.\u003c/li\u003e\n  \u003cli\u003eIntegration of NGS-based MRD data can enable informed treatment de-escalation or intensification, contributing to personalized treatment strategies in MM.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Minimal Residual Disease (MRD) Detection", "label": "Minimal Residual Disease (MRD) Detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Next-Generation Sequencing (NGS)", "label": "Next-Generation Sequencing (NGS)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Next-Generation Flow Cytometry (NGF)", "label": "Next-Generation Flow Cytometry (NGF)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clonal IGH Rearrangement", "label": "Clonal IGH Rearrangement", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sensitivity and Reproducibility of MRD Methods", "label": "Sensitivity and Reproducibility of MRD Methods", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of NGS vs. NGF", "label": "Comparison of NGS vs. NGF", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Autologous Stem Cell Transplantation (ASCT)", "label": "Autologous Stem Cell Transplantation (ASCT)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Multiple Myeloma (MM) Monitoring", "label": "Multiple Myeloma (MM) Monitoring", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Diagnostic Biomarkers", "label": "Diagnostic Biomarkers", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical Utility of NGS in Chinese MM Patients", "label": "Clinical Utility of NGS in Chinese MM Patients", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Role of flow cytometric measurable residual disease assessment in multiple myeloma", "label": "Role of flow cytometric measurable residual disease assessment in multiple myeloma", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Role of flow cytometric measurable residual disease assessment in multiple myeloma\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.5603/AHP.a2023.0024\"\u003e10.5603/AHP.a2023.0024\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eFlow cytometry, particularly next-generation flow (NGF) cytometry, enables MRD detection in multiple myeloma (MM) with a sensitivity up to 10\u207b\u2076, significantly improving upon earlier immunophenotypic methods.\u003c/li\u003e\n  \u003cli\u003eThe EuroFlow NGF protocol using a two-tube, 8-color antibody panel allows standardized MRD assessment with high reproducibility and sensitivity across laboratories.\u003c/li\u003e\n  \u003cli\u003eStudies show NGF detects a higher percentage of MRD-positive cases (47%) compared to conventional 8-color MFC (34%), correlating with improved progression-free survival (PFS) for MRD-negative patients (p = 0.01).\u003c/li\u003e\n  \u003cli\u003eSample quality, particularly bone marrow hemodilution, significantly affects MRD detection sensitivity, underlining the need for proper pre-analytical handling and interpretation.\u003c/li\u003e\n  \u003cli\u003eNGF performance is optimized by automated tools like Infinicyt software, which enables high-throughput analysis and reduces operator subjectivity in plasma cell detection.\u003c/li\u003e\n  \u003cli\u003eAnti-CD38 therapies such as daratumumab interfere with traditional MRD markers, requiring alternative antibodies or markers (e.g., CD229, CD319, VS38c) for accurate MRD assessment.\u003c/li\u003e\n  \u003cli\u003eNext-generation sequencing (NGS) enables MRD detection with a sensitivity of up to 10\u207b\u2077 and can be applied retrospectively on cryopreserved or archival samples, though it lacks sample quality control.\u003c/li\u003e\n  \u003cli\u003eNGS and NGF show 83\u201385% concordance at 10\u207b\u2075 sensitivity and 78% at 10\u207b\u2076, with NGS showing slightly better sensitivity but higher cost and turnaround time.\u003c/li\u003e\n  \u003cli\u003eIntegration of PET-CT imaging with MRD assessment improves detection of extramedullary disease, with discordance noted in 6.8% of MRD-negative patients who remained PET-positive.\u003c/li\u003e\n  \u003cli\u003eMeta-analyses consistently show that MRD negativity is associated with significantly longer PFS and overall survival, independent of treatment type or cytogenetic risk.\u003c/li\u003e\n  \u003cli\u003eLongitudinal MRD monitoring reveals that MRD conversion (from negative to positive) precedes biochemical or clinical relapse by several months, indicating its potential as an early relapse biomarker.\u003c/li\u003e\n  \u003cli\u003eMRD-guided therapeutic decisions (e.g., treatment intensification, de-escalation, or discontinuation) have shown promising outcomes in clinical trials, supporting its role in personalized MM management.\u003c/li\u003e\n  \u003cli\u003ePeripheral blood-based MRD detection using circulating tumor plasma cells (CTPCs) or cell-free DNA (cfDNA) is less sensitive than bone marrow-based methods but may provide complementary information on disease burden and dissemination.\u003c/li\u003e\n  \u003cli\u003eMass spectrometry (e.g., MALDI-TOF-MS) of serum M-proteins offers higher sensitivity than conventional immunofixation and may serve as a non-invasive MRD detection method with prognostic relevance.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Measurable Residual Disease (MRD) as a Prognostic Marker", "label": "Measurable Residual Disease (MRD) as a Prognostic Marker", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Flow Cytometry and Next-Generation Flow (NGF) Cytometry", "label": "Flow Cytometry and Next-Generation Flow (NGF) Cytometry", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Next-Generation Sequencing (NGS) for MRD Detection", "label": "Next-Generation Sequencing (NGS) for MRD Detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sensitivity, Specificity, and Standardization of MRD Methods", "label": "Sensitivity, Specificity, and Standardization of MRD Methods", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Bone Marrow Sample Quality and Pre-analytical Challenges", "label": "Bone Marrow Sample Quality and Pre-analytical Challenges", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Integration of PET-CT Imaging in MRD Assessment", "label": "Integration of PET-CT Imaging in MRD Assessment", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical Impact of Sustained MRD Negativity", "label": "Clinical Impact of Sustained MRD Negativity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "MRD-Guided Therapeutic Strategies and Clinical Trial Design", "label": "MRD-Guided Therapeutic Strategies and Clinical Trial Design", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Use of Peripheral Blood Techniques and Liquid Biopsy", "label": "Use of Peripheral Blood Techniques and Liquid Biopsy", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Minimal residual disease detection by next-generation sequencing of different immunoglobulin gene rearrangements in pediatric B-ALL", "label": "Minimal residual disease detection by next-generation sequencing of different immunoglobulin gene rearrangements in pediatric B-ALL", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Minimal residual disease detection by next-generation sequencing of different immunoglobulin gene rearrangements in pediatric B-ALL\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1038/s41467-023-43171-9\"\u003e10.1038/s41467-023-43171-9\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eNext-generation sequencing (NGS) detected clonal rearrangements in IGH, IGK, and IGL genes in 94.5%, 44.1%, and 17.0% of pediatric B-ALL patients, respectively, enabling broader MRD tracking than IGH alone.\u003c/li\u003e\n\n  \u003cli\u003eIGH rearrangement alone was sufficient for MRD tracking in most cases; IGK/IGL added minimal prognostic value and only identified 5.5% of cases not covered by IGH.\u003c/li\u003e\n\n  \u003cli\u003ePatients with NGS-MRD levels \u0026lt;0.01% at end of induction (EOI) and \u0026lt;0.0001% at end of consolidation (EOC) had excellent 3-year event-free survival (EFS) rates, exceeding 95%.\u003c/li\u003e\n\n  \u003cli\u003eIGH-MRD levels were significantly associated with prognosis at both EOI and EOC, while IGK and IGL MRD levels were not independently prognostic at either time point.\u003c/li\u003e\n\n  \u003cli\u003eNGS detected MRD positivity in 26.2% of patients deemed MRD-negative by multiparameter flow cytometry (MFC) at EOI, and these patients had significantly worse outcomes, supporting NGS\u0027s superior sensitivity.\u003c/li\u003e\n\n  \u003cli\u003eIGH and IGK clone clearance was faster for clones with initial frequencies \u0026lt;10% compared to those with \u226550%, indicating clone size influences clearance dynamics during therapy.\u003c/li\u003e\n\n  \u003cli\u003eAt EOI, the concordance between IGH-MRD and IGK/IGL-MRD was 79.9% using a 0.01% cutoff; this increased to 81.0% at EOC using a 0.0001% cutoff, suggesting modest alignment across loci.\u003c/li\u003e\n\n  \u003cli\u003eIntegration of IGK/IGL into IGH-based MRD tracking did not significantly improve risk stratification or predict additional relapse events, indicating limited incremental value.\u003c/li\u003e\n\n  \u003cli\u003ePatients with IGH-negative but IGK/IGL-positive MRD showed similar outcomes to those completely MRD-negative, reinforcing that IGK/IGL status does not significantly alter prognosis.\u003c/li\u003e\n\n  \u003cli\u003eNGS-based MRD monitoring enables more precise relapse risk identification and may support earlier therapeutic interventions (e.g., blinatumomab) for deeper MRD eradication.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "IGH, IGK, and IGL gene rearrangements in pediatric B-ALL", "label": "IGH, IGK, and IGL gene rearrangements in pediatric B-ALL", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clonal clearance dynamics during induction and consolidation", "label": "Clonal clearance dynamics during induction and consolidation", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic significance of MRD levels at EOI and EOC", "label": "Prognostic significance of MRD levels at EOI and EOC", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Value of integrating IGK/IGL with IGH in MRD assessment", "label": "Value of integrating IGK/IGL with IGH in MRD assessment", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Risk stratification based on MRD thresholds", "label": "Risk stratification based on MRD thresholds", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical implications for relapse prediction and immunotherapy", "label": "Clinical implications for relapse prediction and immunotherapy", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": " Validation and interpretation of IGH and TCR clonality testing by Ion Torrent S5 NGS for diagnosis and disease monitoring in B and T cell cancers", "label": " Validation and interpretation of IGH and TCR clonality testing by Ion Torrent S5 NGS for diagnosis and disease monitoring in B and T cell cancers", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Validation and interpretation of IGH and TCR clonality testing by Ion Torrent S5 NGS for diagnosis and disease monitoring in B and T cell cancers\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1016/j.plabm.2020.e00191\"\u003e10.1016/j.plabm.2020.e00191\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eThe LymphoTrack assays using Ion Torrent S5 sequencing reliably detected IGH and TCR clonality, with reproducible results across technical replicates, different reagent lots, operators, and instruments.\u003c/li\u003e\n\n  \u003cli\u003eNGS detected clonal rearrangements in specimens that failed PCR fragment-based clonality assays, highlighting the superior sensitivity of the NGS method.\u003c/li\u003e\n\n  \u003cli\u003eNGS-based TCRG results were concordant with clinical and histopathologic diagnoses in the majority of patients with T-cell leukemia/lymphoma, supporting clinical utility for diagnosis and monitoring.\u003c/li\u003e\n\n  \u003cli\u003eIn 11 patients with serial TCRG NGS tests, identical clonal sequences were detected across samples, demonstrating the method\u2019s ability to track clonal populations over time and across tissue sites.\u003c/li\u003e\n\n  \u003cli\u003eComparison of interpretive guidelines showed that a 2.5% read cutoff (Guideline A) improved sensitivity, while adding a fold-over-background rule (Guideline B) improved specificity but reduced sensitivity.\u003c/li\u003e\n\n  \u003cli\u003eNGS clonality testing demonstrated higher detection rates of IGH somatic hypermutation in CLL patients compared to PCR-based methods, resolving previously uninterpretable or ambiguous results.\u003c/li\u003e\n\n  \u003cli\u003eNGS identified clonal or polyclonal sequences in approximately 67% of PCR-inconclusive or polyclonal cases, indicating its added diagnostic value beyond conventional PCR methods.\u003c/li\u003e\n\n  \u003cli\u003eThe LymphoTrack assay\u0027s performance using Ion Torrent S5 was comparable to that of Illumina platforms, showing that sequencing platform choice does not compromise assay validity.\u003c/li\u003e\n\n  \u003cli\u003eClonal sequences detected by LymphoTrack did not always match those detected by clonoSeq, suggesting methodological differences that impact concordance across platforms.\u003c/li\u003e\n\n  \u003cli\u003eOverall, LymphoTrack with Ion Torrent S5 provides a sensitive, quantitative, and clinically relevant method for clonality testing and somatic hypermutation analysis in B- and T-cell malignancies.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Validation of NGS-based IGH and TCR clonality assays", "label": "Validation of NGS-based IGH and TCR clonality assays", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison of NGS (Ion Torrent S5) with PCR fragment analysis", "label": "Comparison of NGS (Ion Torrent S5) with PCR fragment analysis", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical utility of NGS in detecting clonal sequences in B- and T-cell malignancies", "label": "Clinical utility of NGS in detecting clonal sequences in B- and T-cell malignancies", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Diagnostic performance and reproducibility of LymphoTrack assays", "label": "Diagnostic performance and reproducibility of LymphoTrack assays", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Tracking clonal populations over time and across biopsies using NGS", "label": "Tracking clonal populations over time and across biopsies using NGS", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Interpretation guidelines for NGS clonality results (sensitivity vs. specificity)", "label": "Interpretation guidelines for NGS clonality results (sensitivity vs. specificity)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Improved detection of IGH somatic hypermutation in CLL using NGS", "label": "Improved detection of IGH somatic hypermutation in CLL using NGS", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Concordance and discrepancies between NGS platforms (e.g., LymphoTrack vs. clonoSeq)", "label": "Concordance and discrepancies between NGS platforms (e.g., LymphoTrack vs. clonoSeq)", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Integration of NGS results with clinical and histopathological diagnoses", "label": "Integration of NGS results with clinical and histopathological diagnoses", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Implications for routine clinical use of NGS in hematologic malignancy diagnostics", "label": "Implications for routine clinical use of NGS in hematologic malignancy diagnostics", "shape": "ellipse"}, {"color": "lightblue", "font": {"color": "black"}, "id": "Analytical and clinical validation of a novel in-house deep-sequencing method for minimal residual disease monitoring in a phase II trial for multiple myeloma", "label": "Analytical and clinical validation of a novel in-house deep-sequencing method for minimal residual disease monitoring in a phase II trial for multiple myeloma", "shape": "box", "title": "\u003cdiv class=\u0027custom-tooltip\u0027 style=\u0027max-height:75%; overflow-y:auto; width:80%;\u0027\u003e\u003cp\u003e\u003cstrong\u003eArticle Title:\u003c/strong\u003e Analytical and clinical validation of a novel in-house deep-sequencing method for minimal residual disease monitoring in a phase II trial for multiple myeloma\u003c/p\u003e\n\u003cp\u003e\u003cstrong\u003eDOI:\u003c/strong\u003e \u003ca href=\"https://doi.org/10.1038/leu.2017.58\"\u003e10.1038/leu.2017.58\u003c/a\u003e\u003c/p\u003e\n\n\u003col\u003e\n  \u003cli\u003eThe novel in-house deep-sequencing method using Biomed-2 primers detected clonotypes in 97% of multiple myeloma patients, demonstrating high specificity and sensitivity (down to 10\u207b\u2075), comparable or superior to existing methods like 8-color flow cytometry and commercial NGS assays.\u003c/li\u003e\n\n  \u003cli\u003eThis method successfully quantified minimal residual disease (MRD) in bone marrow samples from patients in a phase II clinical trial, providing a robust tool for molecular response assessment.\u003c/li\u003e\n\n  \u003cli\u003eClonotypes were not detected in normal control samples (tonsil and bone marrow), confirming the high specificity of the assay for pathological clonal populations.\u003c/li\u003e\n\n  \u003cli\u003eDeep sequencing results showed strong reproducibility across sequencing runs and platforms, with a 99.2% concordance rate in duplicate testing and low coefficient of variation in MRD-negative samples.\u003c/li\u003e\n\n  \u003cli\u003ePatients receiving prolonged treatment (18 cycles) had a significantly higher rate of MRD-negative responses (27% vs. 11.5%, P=0.04), supporting the clinical benefit of extended therapy duration.\u003c/li\u003e\n\n  \u003cli\u003eMRD negativity by the in-house deep-sequencing method correlated with superior progression-free survival (PFS) and overall survival (OS); MRD-negative patients had a 3-year OS rate of 100% versus 45% in MRD-positive patients (HR = 3.66, P = 0.05).\u003c/li\u003e\n\n  \u003cli\u003eComparison between the new NGS method and 8-color flow cytometry showed a strong correlation (Spearman R = 0.79, P \u0026lt; 0.0001), and 89% global concordance in MRD assessment across 175 samples.\u003c/li\u003e\n\n  \u003cli\u003ePatients with discordant MRD results between the two methods had intermediate outcomes in terms of PFS and OS, indicating that combining methods may improve prognostication.\u003c/li\u003e\n\n  \u003cli\u003eThe method can be fully automated from DNA extraction to data analysis, is platform-independent, and thus suitable for standardization across clinical laboratories with NGS capacity.\u003c/li\u003e\n\n  \u003cli\u003eThis NGS-based approach aligns with the International Myeloma Working Group\u2019s recommendations for MRD assessment and could be adopted for defining sequencing MRD negativity in future clinical criteria.\u003c/li\u003e\n\u003c/ol\u003e\n\u003c/div\u003e"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "In-house NGS method for MRD detection", "label": "In-house NGS method for MRD detection", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Sensitivity and specificity in multiple myeloma", "label": "Sensitivity and specificity in multiple myeloma", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Comparison with 8-color flow cytometry", "label": "Comparison with 8-color flow cytometry", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Prognostic value of MRD negativity", "label": "Prognostic value of MRD negativity", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Impact of treatment duration on response", "label": "Impact of treatment duration on response", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Method reproducibility and platform-independence", "label": "Method reproducibility and platform-independence", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Concordance between NGS and MFC", "label": "Concordance between NGS and MFC", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Clinical lab implementation feasibility", "label": "Clinical lab implementation feasibility", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Alignment with IMWG MRD criteria", "label": "Alignment with IMWG MRD criteria", "shape": "ellipse"}, {"color": "lightgreen", "font": {"color": "black"}, "id": "Automation and standardization potential", "label": "Automation and standardization potential", "shape": "ellipse"}]);
                  edges = new vis.DataSet([{"from": "Comparison of Measurable Residual Disease in Pediatric B-Lymphoblastic Leukemia Using Multiparametric Flow Cytometry and Next-Generation Sequencing", "to": "MRD Detection Methods"}, {"from": "Comparison of Measurable Residual Disease in Pediatric B-Lymphoblastic Leukemia Using Multiparametric Flow Cytometry and Next-Generation Sequencing", "to": "Assay Concordance"}, {"from": "Comparison of Measurable Residual Disease in Pediatric B-Lymphoblastic Leukemia Using Multiparametric Flow Cytometry and Next-Generation Sequencing", "to": "Sensitivity of NGS-MRD"}, {"from": "Comparison of Measurable Residual Disease in Pediatric B-Lymphoblastic Leukemia Using Multiparametric Flow Cytometry and Next-Generation Sequencing", "to": "Discordant MRD Results"}, {"from": "Comparison of Measurable Residual Disease in Pediatric B-Lymphoblastic Leukemia Using Multiparametric Flow Cytometry and Next-Generation Sequencing", "to": "Quantitative Correlation Metrics"}, {"from": "Comparison of Measurable Residual Disease in Pediatric B-Lymphoblastic Leukemia Using Multiparametric Flow Cytometry and Next-Generation Sequencing", "to": "Immunophenotypic Shifts"}, {"from": "Comparison of Measurable Residual Disease in Pediatric B-Lymphoblastic Leukemia Using Multiparametric Flow Cytometry and Next-Generation Sequencing", "to": "Hemodilution Effects"}, {"from": "Comparison of Measurable Residual Disease in Pediatric B-Lymphoblastic Leukemia Using Multiparametric Flow Cytometry and Next-Generation Sequencing", "to": "Hematogone Analysis"}, {"from": "Comparison of Measurable Residual Disease in Pediatric B-Lymphoblastic Leukemia Using Multiparametric Flow Cytometry and Next-Generation Sequencing", "to": "Clinical Applicability of MRD Assays"}, {"from": "Comparison of Measurable Residual Disease in Pediatric B-Lymphoblastic Leukemia Using Multiparametric Flow Cytometry and Next-Generation Sequencing", "to": "Early Detection of Relapse"}, {"from": "The effectiveness of blinatumomab in clearing measurable residual disease in pediatric B-cell acute lymphoblastic leukemia patients detected by next-generation sequencing", "to": "Blinatumomab efficacy in pediatric B-ALL"}, {"from": "The effectiveness of blinatumomab in clearing measurable residual disease in pediatric B-cell acute lymphoblastic leukemia patients detected by next-generation sequencing", "to": "Measurable residual disease (MRD) monitoring"}, {"from": "The effectiveness of blinatumomab in clearing measurable residual disease in pediatric B-cell acute lymphoblastic leukemia patients detected by next-generation sequencing", "to": "Next-generation sequencing (NGS)-based MRD detection"}, {"from": "The effectiveness of blinatumomab in clearing measurable residual disease in pediatric B-cell acute lymphoblastic leukemia patients detected by next-generation sequencing", "to": "Comparison between NGS and MFC sensitivity"}, {"from": "The effectiveness of blinatumomab in clearing measurable residual disease in pediatric B-cell acute lymphoblastic leukemia patients detected by next-generation sequencing", "to": "Prognostic significance of clonal immunoglobulin gene rearrangements (IGH, IGK, IGL)"}, {"from": "The effectiveness of blinatumomab in clearing measurable residual disease in pediatric B-cell acute lymphoblastic leukemia patients detected by next-generation sequencing", "to": "Early relapse prediction and risk stratification"}, {"from": "The effectiveness of blinatumomab in clearing measurable residual disease in pediatric B-cell acute lymphoblastic leukemia patients detected by next-generation sequencing", "to": "Short-term survival outcomes (OS/EFS) post-treatment"}, {"from": "The effectiveness of blinatumomab in clearing measurable residual disease in pediatric B-cell acute lymphoblastic leukemia patients detected by next-generation sequencing", "to": "Role of blinatumomab in MRD-directed therapy"}, {"from": "The effectiveness of blinatumomab in clearing measurable residual disease in pediatric B-cell acute lymphoblastic leukemia patients detected by next-generation sequencing", "to": "Integration of NGS-MRD into clinical decision-making"}, {"from": "The effectiveness of blinatumomab in clearing measurable residual disease in pediatric B-cell acute lymphoblastic leukemia patients detected by next-generation sequencing", "to": "Hematopoietic stem cell transplantation (HSCT) after blinatumomab"}, {"from": "Multi-target measurable residual disease assessed by error-corrected sequencing in patients with acute myeloid leukemia: An ALFA study", "to": "Measurable Residual Disease (MRD) in Acute Myeloid Leukemia (AML)"}, {"from": "Multi-target measurable residual disease assessed by error-corrected sequencing in patients with acute myeloid leukemia: An ALFA study", "to": "Error-corrected Next-Generation Sequencing (NGS-MRD)"}, {"from": "Multi-target measurable residual disease assessed by error-corrected sequencing in patients with acute myeloid leukemia: An ALFA study", "to": "DNMT3A, TET2, and ASXL1 (DTA) mutations"}, {"from": "Multi-target measurable residual disease assessed by error-corrected sequencing in patients with acute myeloid leukemia: An ALFA study", "to": "Prognostic impact of persistent mutations in remission"}, {"from": "Multi-target measurable residual disease assessed by error-corrected sequencing in patients with acute myeloid leukemia: An ALFA study", "to": "Comparison between NGS-MRD, WT1-MRD, and NPM1-MRD"}, {"from": "Multi-target measurable residual disease assessed by error-corrected sequencing in patients with acute myeloid leukemia: An ALFA study", "to": "Role of mutation burden (number of mutations) in prognosis"}, {"from": "Multi-target measurable residual disease assessed by error-corrected sequencing in patients with acute myeloid leukemia: An ALFA study", "to": "Influence of NGS-MRD on allogeneic stem cell transplantation decisions"}, {"from": "Multi-target measurable residual disease assessed by error-corrected sequencing in patients with acute myeloid leukemia: An ALFA study", "to": "Integration of molecular MRD strategies in AML clinical management"}, {"from": "Quantification of Measurable Residual Disease Detection by Next-Generation Sequencing-Based Clonality Testing in B-Cell and Plasma Cell Neoplasms", "to": "MRD Detection Technologies"}, {"from": "Quantification of Measurable Residual Disease Detection by Next-Generation Sequencing-Based Clonality Testing in B-Cell and Plasma Cell Neoplasms", "to": "Comparative Accuracy of MFC-MRD vs. NGS-MRD"}, {"from": "Quantification of Measurable Residual Disease Detection by Next-Generation Sequencing-Based Clonality Testing in B-Cell and Plasma Cell Neoplasms", "to": "Sensitivity and Detection Limits"}, {"from": "Quantification of Measurable Residual Disease Detection by Next-Generation Sequencing-Based Clonality Testing in B-Cell and Plasma Cell Neoplasms", "to": "Discordance Analysis Between Assays"}, {"from": "Quantification of Measurable Residual Disease Detection by Next-Generation Sequencing-Based Clonality Testing in B-Cell and Plasma Cell Neoplasms", "to": "Quantitative Correlation and Statistical Agreement"}, {"from": "Quantification of Measurable Residual Disease Detection by Next-Generation Sequencing-Based Clonality Testing in B-Cell and Plasma Cell Neoplasms", "to": "Effect of Immunophenotypic Changes Post-Therapy"}, {"from": "Quantification of Measurable Residual Disease Detection by Next-Generation Sequencing-Based Clonality Testing in B-Cell and Plasma Cell Neoplasms", "to": "Impact of Hemodilution and Sample Quality"}, {"from": "Quantification of Measurable Residual Disease Detection by Next-Generation Sequencing-Based Clonality Testing in B-Cell and Plasma Cell Neoplasms", "to": "Use of Hematogones as a Marker of Sample Integrity"}, {"from": "Quantification of Measurable Residual Disease Detection by Next-Generation Sequencing-Based Clonality Testing in B-Cell and Plasma Cell Neoplasms", "to": "Clinical Scenarios Favoring One Assay Over the Other"}, {"from": "Quantification of Measurable Residual Disease Detection by Next-Generation Sequencing-Based Clonality Testing in B-Cell and Plasma Cell Neoplasms", "to": "Implications for Early Detection and Monitoring Strategy"}, {"from": " Corrigendum to \u201cMeasurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways?", "to": "MRD (Measurable Residual Disease) testing thresholds"}, {"from": " Corrigendum to \u201cMeasurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways?", "to": "MPFC (Multiparameter Flow Cytometry) assay standards"}, {"from": " Corrigendum to \u201cMeasurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways?", "to": "PCR-based MRD assay performance"}, {"from": " Corrigendum to \u201cMeasurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways?", "to": "NGS (Next-Generation Sequencing) in MRD detection"}, {"from": " Corrigendum to \u201cMeasurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways?", "to": "AML (Acute Myeloid Leukemia) subtypes and MRD evaluation"}, {"from": " Corrigendum to \u201cMeasurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways?", "to": "ALL (Acute Lymphoblastic Leukemia) MRD sensitivity"}, {"from": " Corrigendum to \u201cMeasurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways?", "to": "Methodological variability across studies"}, {"from": " Corrigendum to \u201cMeasurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways?", "to": "Assay sensitivity and limit of detection (LoD)"}, {"from": " Corrigendum to \u201cMeasurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways?", "to": "Standardization of MRD thresholds in hematological cancers"}, {"from": "Measurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways", "to": "Accuracy and limitations of MRD-testing"}, {"from": "Measurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways", "to": "MRD-testing techniques (MPFC, PCR, dPCR, NGS)"}, {"from": "Measurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways", "to": "Prognostic value of MRD in AML and ALL"}, {"from": "Measurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways", "to": "Sampling challenges and assay sensitivity"}, {"from": "Measurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways", "to": "Concordance between MRD-testing methods"}, {"from": "Measurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways", "to": "Clinical utility and thresholds for MRD interpretation"}, {"from": "Measurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways", "to": "MRD-guided therapy and lack of RCT-based evidence"}, {"from": "Measurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways", "to": "Role of MRD in regulatory approval and guidelines"}, {"from": "Measurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways", "to": "Integration of MRD with multi-dimensional risk models"}, {"from": "Measurable residual disease (MRD)-testing in haematological cancers: A giant leap forward or sideways", "to": "Recommendations for cautious use of MRD in clinical decisions"}, {"from": "Comparison of next-generation sequencing (NGS) and next-generation flow (NGF) for minimal residual disease (MRD) assessment in multiple myeloma", "to": "Minimal Residual Disease (MRD) detection in multiple myeloma"}, {"from": "Comparison of next-generation sequencing (NGS) and next-generation flow (NGF) for minimal residual disease (MRD) assessment in multiple myeloma", "to": "Comparison between Next-Generation Sequencing (NGS) and Next-Generation Flow (NGF)"}, {"from": "Comparison of next-generation sequencing (NGS) and next-generation flow (NGF) for minimal residual disease (MRD) assessment in multiple myeloma", "to": "Prognostic significance of MRD status on progression-free survival (PFS) and overall survival (OS)"}, {"from": "Comparison of next-generation sequencing (NGS) and next-generation flow (NGF) for minimal residual disease (MRD) assessment in multiple myeloma", "to": "Technical and clinical performance characteristics of NGS vs. NGF"}, {"from": "Comparison of next-generation sequencing (NGS) and next-generation flow (NGF) for minimal residual disease (MRD) assessment in multiple myeloma", "to": "Influence of cytogenetics and response categories on MRD outcomes"}, {"from": "Comparison of next-generation sequencing (NGS) and next-generation flow (NGF) for minimal residual disease (MRD) assessment in multiple myeloma", "to": "Multivariate modeling of survival predictors"}, {"from": "Comparison of next-generation sequencing (NGS) and next-generation flow (NGF) for minimal residual disease (MRD) assessment in multiple myeloma", "to": "Applicability and limitations of MRD assessment techniques in clinical trials"}, {"from": "Next-Generation Sequencing in Acute Lymphoblastic Leukemia", "to": "B-cell Acute Lymphoblastic Leukemia (B-ALL) subtypes and gene fusions"}, {"from": "Next-Generation Sequencing in Acute Lymphoblastic Leukemia", "to": "T-cell Acute Lymphoblastic Leukemia (T-ALL) genetic alterations"}, {"from": "Next-Generation Sequencing in Acute Lymphoblastic Leukemia", "to": "Next-Generation Sequencing (NGS) technologies and methodologies"}, {"from": "Next-Generation Sequencing in Acute Lymphoblastic Leukemia", "to": "Minimal Residual Disease (MRD) monitoring using NGS"}, {"from": "Next-Generation Sequencing in Acute Lymphoblastic Leukemia", "to": "Third-generation sequencing (e.g., nanopore sequencing) in leukemia"}, {"from": "Next-Generation Sequencing in Acute Lymphoblastic Leukemia", "to": "Risk stratification and targeted therapy in ALL"}, {"from": "Next-Generation Sequencing in Acute Lymphoblastic Leukemia", "to": "Inherited genetic predispositions to ALL"}, {"from": "Next-Generation Sequencing in Acute Lymphoblastic Leukemia", "to": "Molecular classification and prognostic biomarkers in ALL"}, {"from": "Is Next-Generation Sequencing the Way to Go for Residual Disease Monitoring in Acute Lymphoblastic Leukemia?", "to": "Prognostic role of Minimal Residual Disease (MRD) in ALL"}, {"from": "Is Next-Generation Sequencing the Way to Go for Residual Disease Monitoring in Acute Lymphoblastic Leukemia?", "to": "Traditional MRD detection methods (e.g., RQ-PCR, flow cytometry)"}, {"from": "Is Next-Generation Sequencing the Way to Go for Residual Disease Monitoring in Acute Lymphoblastic Leukemia?", "to": "Advantages and limitations of Next-Generation Sequencing (NGS) for MRD"}, {"from": "Is Next-Generation Sequencing the Way to Go for Residual Disease Monitoring in Acute Lymphoblastic Leukemia?", "to": "Comparative sensitivity and specificity of NGS vs. RQ-PCR"}, {"from": "Is Next-Generation Sequencing the Way to Go for Residual Disease Monitoring in Acute Lymphoblastic Leukemia?", "to": "NGS-based marker identification and clonal tracking"}, {"from": "Is Next-Generation Sequencing the Way to Go for Residual Disease Monitoring in Acute Lymphoblastic Leukemia?", "to": "B- and T-cell repertoire analysis using NGS"}, {"from": "Is Next-Generation Sequencing the Way to Go for Residual Disease Monitoring in Acute Lymphoblastic Leukemia?", "to": "Technical and bioinformatic challenges in NGS (e.g., contamination, quantification)"}, {"from": "Is Next-Generation Sequencing the Way to Go for Residual Disease Monitoring in Acute Lymphoblastic Leukemia?", "to": "Standardization efforts (EuroClonality-NGS Consortium)"}, {"from": "Is Next-Generation Sequencing the Way to Go for Residual Disease Monitoring in Acute Lymphoblastic Leukemia?", "to": "Cost and scalability considerations for clinical implementation"}, {"from": "Is Next-Generation Sequencing the Way to Go for Residual Disease Monitoring in Acute Lymphoblastic Leukemia?", "to": "Future perspectives and clinical integration of NGS in MRD monitoring"}, {"from": "Simple deep sequencing-based post-remission MRD surveillance predicts clinical relapse in B-ALL", "to": "NGS-based MRD detection assay development"}, {"from": "Simple deep sequencing-based post-remission MRD surveillance predicts clinical relapse in B-ALL", "to": "Analytical sensitivity and reproducibility of MRD testing"}, {"from": "Simple deep sequencing-based post-remission MRD surveillance predicts clinical relapse in B-ALL", "to": "Comparison between NGS and flow cytometry for MRD"}, {"from": "Simple deep sequencing-based post-remission MRD surveillance predicts clinical relapse in B-ALL", "to": "Post-remission MRD surveillance in B-ALL"}, {"from": "Simple deep sequencing-based post-remission MRD surveillance predicts clinical relapse in B-ALL", "to": "Prediction of clinical relapse using CPMRD"}, {"from": "Simple deep sequencing-based post-remission MRD surveillance predicts clinical relapse in B-ALL", "to": "Clinical utility of dynamic MRD monitoring"}, {"from": "Simple deep sequencing-based post-remission MRD surveillance predicts clinical relapse in B-ALL", "to": "Implementation feasibility in routine clinical labs"}, {"from": "Simple deep sequencing-based post-remission MRD surveillance predicts clinical relapse in B-ALL", "to": "Preemptive therapy timing based on MRD trends"}, {"from": "Clinical utility of next-generation sequencing-based minimal residual disease in paediatric B-cell acute lymphoblastic leukaemia", "to": "Next-generation sequencing (NGS) for MRD detection"}, {"from": "Clinical utility of next-generation sequencing-based minimal residual disease in paediatric B-cell acute lymphoblastic leukaemia", "to": "Prognostic value of MRD at multiple treatment time points"}, {"from": "Clinical utility of next-generation sequencing-based minimal residual disease in paediatric B-cell acute lymphoblastic leukaemia", "to": "Risk stratification in pediatric B-cell ALL"}, {"from": "Clinical utility of next-generation sequencing-based minimal residual disease in paediatric B-cell acute lymphoblastic leukaemia", "to": "Comparison of NGS with traditional MRD methods (RQ-PCR, FCM)"}, {"from": "Clinical utility of next-generation sequencing-based minimal residual disease in paediatric B-cell acute lymphoblastic leukaemia", "to": "Sensitivity thresholds for MRD detection"}, {"from": "Clinical utility of next-generation sequencing-based minimal residual disease in paediatric B-cell acute lymphoblastic leukaemia", "to": "Clonal evolution in relapsed leukemia"}, {"from": "Clinical utility of next-generation sequencing-based minimal residual disease in paediatric B-cell acute lymphoblastic leukaemia", "to": "Treatment implications based on MRD status"}, {"from": "Clinical utility of next-generation sequencing-based minimal residual disease in paediatric B-cell acute lymphoblastic leukaemia", "to": "Timing and frequency of MRD monitoring"}, {"from": "Clinical utility of next-generation sequencing-based minimal residual disease in paediatric B-cell acute lymphoblastic leukaemia", "to": "Clinical outcomes and relapse prediction"}, {"from": "Clinical utility of next-generation sequencing-based minimal residual disease in paediatric B-cell acute lymphoblastic leukaemia", "to": "Utility of deep sequencing in personalized treatment planning"}, {"from": "Association of Minimal Residual Disease With Clinical Outcome in Pediatric and Adult Acute Lymphoblastic Leukemia: A Meta-analysis", "to": "Minimal residual disease (MRD) as a prognostic biomarker"}, {"from": "Association of Minimal Residual Disease With Clinical Outcome in Pediatric and Adult Acute Lymphoblastic Leukemia: A Meta-analysis", "to": "Event-free survival (EFS) and overall survival (OS) in ALL"}, {"from": "Association of Minimal Residual Disease With Clinical Outcome in Pediatric and Adult Acute Lymphoblastic Leukemia: A Meta-analysis", "to": "MRD detection methods (PCR vs flow cytometry vs NGS)"}, {"from": "Association of Minimal Residual Disease With Clinical Outcome in Pediatric and Adult Acute Lymphoblastic Leukemia: A Meta-analysis", "to": "Risk stratification in pediatric and adult ALL"}, {"from": "Association of Minimal Residual Disease With Clinical Outcome in Pediatric and Adult Acute Lymphoblastic Leukemia: A Meta-analysis", "to": "MRD cutoff thresholds and sensitivity"}, {"from": "Association of Minimal Residual Disease With Clinical Outcome in Pediatric and Adult Acute Lymphoblastic Leukemia: A Meta-analysis", "to": "Clinical trial design and drug development using MRD"}, {"from": "Association of Minimal Residual Disease With Clinical Outcome in Pediatric and Adult Acute Lymphoblastic Leukemia: A Meta-analysis", "to": "Subgroup analyses (B-cell/T-cell, Ph+/Ph\u2212, age groups)"}, {"from": "Association of Minimal Residual Disease With Clinical Outcome in Pediatric and Adult Acute Lymphoblastic Leukemia: A Meta-analysis", "to": "Role of MRD in regulatory approval and surrogate endpoints"}, {"from": "Association of Minimal Residual Disease With Clinical Outcome in Pediatric and Adult Acute Lymphoblastic Leukemia: A Meta-analysis", "to": "Adaptive trial methodologies using MRD-guided endpoints"}, {"from": "Association of Minimal Residual Disease With Clinical Outcome in Pediatric and Adult Acute Lymphoblastic Leukemia: A Meta-analysis", "to": "Limitations and future directions in MRD standardization4"}, {"from": "Measurable residual disease in acute lymphoblastic leukemia: methods and clinical context in adult patients", "to": "Prognostic value of measurable residual disease (MRD) in ALL"}, {"from": "Measurable residual disease in acute lymphoblastic leukemia: methods and clinical context in adult patients", "to": "MRD detection methods (MFC, qPCR, NGS, ddPCR)"}, {"from": "Measurable residual disease in acute lymphoblastic leukemia: methods and clinical context in adult patients", "to": "Next-generation sequencing (NGS) and ClonoSEQ"}, {"from": "Measurable residual disease in acute lymphoblastic leukemia: methods and clinical context in adult patients", "to": "MRD-informed treatment strategies and risk stratification"}, {"from": "Measurable residual disease in acute lymphoblastic leukemia: methods and clinical context in adult patients", "to": "Role of MRD in guiding allogeneic hematopoietic cell transplant (HCT)"}, {"from": "Measurable residual disease in acute lymphoblastic leukemia: methods and clinical context in adult patients", "to": "Timing and frequency of MRD monitoring"}, {"from": "Measurable residual disease in acute lymphoblastic leukemia: methods and clinical context in adult patients", "to": "MRD detection post-novel therapies (e.g., blinatumomab, inotuzumab, CAR T)"}, {"from": "Measurable residual disease in acute lymphoblastic leukemia: methods and clinical context in adult patients", "to": "Limitations in MRD tracking for T-cell ALL"}, {"from": "Measurable residual disease in acute lymphoblastic leukemia: methods and clinical context in adult patients", "to": "Emerging MRD technologies and diagnostic platforms"}, {"from": "Measurable residual disease in acute lymphoblastic leukemia: methods and clinical context in adult patients", "to": "Clinical relevance of low-level MRD and early molecular relapse"}, {"from": "A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: an European Research Initiative on CLL study", "to": "Minimal Residual Disease (MRD) detection in Chronic Lymphocytic Leukemia (CLL)"}, {"from": "A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: an European Research Initiative on CLL study", "to": "Development and validation of a six-marker core flow cytometry panel"}, {"from": "A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: an European Research Initiative on CLL study", "to": "Comparison of flow cytometry with high-throughput sequencing (HTS) for MRD"}, {"from": "A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: an European Research Initiative on CLL study", "to": "Assay sensitivity, reproducibility, and inter-laboratory standardization"}, {"from": "A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: an European Research Initiative on CLL study", "to": "Limitations and advantages of Sanger sequencing versus HTS"}, {"from": "A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: an European Research Initiative on CLL study", "to": "Application of ClonoSEQ HTS platform for IGH clonotype detection"}, {"from": "A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: an European Research Initiative on CLL study", "to": "Platform-independent reagent specifications for MRD testing"}, {"from": "A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: an European Research Initiative on CLL study", "to": "Clinical trial integration and real-world implementation feasibility"}, {"from": "A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: an European Research Initiative on CLL study", "to": "Impact of marker selection and reagent signal intensity on CLL detection"}, {"from": "A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: an European Research Initiative on CLL study", "to": "Complementary roles of flow cytometry and HTS in CLL treatment monitoring"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "MRD detection in acute lymphoblastic leukemia (ALL)"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Comparison of NGS vs. multiparameter flow cytometry (MFC)"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Clinical significance of early MRD negativity by NGS"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Prognostic implications of MRD discordance between NGS and MFC"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Risk stratification using high-sensitivity NGS assays"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Impact of MRD status on relapse and overall survival"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Role of MRD in guiding allogeneic stem cell transplantation (alloSCT) decisions"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Efficacy of low-intensity regimens combined with monoclonal antibodies"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Timing of MRD assessment and its prognostic value"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Therapeutic implications and de-escalation strategies based on MRD"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Analytical validation of the clonoSEQ Assay"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "MRD detection sensitivity, specificity, and quantification"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Precision and reproducibility across DNA input levels"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Linearity and accuracy of MRD measurements"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Comparison with multiparameter flow cytometry (mpFC)"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Sequence fidelity and tracking of clonal rearrangements"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Application in ALL, CLL, and MM"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Evaluation of detection thresholds: LoD, LoQ, and LoB"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Impact of DNA input quantity on MRD assay performance"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Standardization and reliability for clinical implementation"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Comparison of NGS (clonoSEQ) and flow cytometry (MFC) for MRD detection"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Sensitivity differences between NGS (10\u207b\u2076) and MFC (10\u207b\u2075)"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Concordance and discordance rates in MRD classification"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Statistical correlation between NGS and MFC MRD levels"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Clinical outcome tracking (relapse, death, transplant) post-MRD assessment"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Prognostic value of NGS MRD detection in adult ALL"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Implications for treatment decisions and post-remission risk stratification"}, {"from": "CDRH\u2019s Approach to Tumor Profiling Next Generation Sequencing Tests", "to": "FDA authorization of tumor profiling NGS tests"}, {"from": "CDRH\u2019s Approach to Tumor Profiling Next Generation Sequencing Tests", "to": "Three-tiered biomarker classification system"}, {"from": "CDRH\u2019s Approach to Tumor Profiling Next Generation Sequencing Tests", "to": "Companion diagnostics and therapeutic guidance"}, {"from": "CDRH\u2019s Approach to Tumor Profiling Next Generation Sequencing Tests", "to": "Cancer mutations with established clinical significance"}, {"from": "CDRH\u2019s Approach to Tumor Profiling Next Generation Sequencing Tests", "to": "Cancer mutations with potential clinical relevance"}, {"from": "CDRH\u2019s Approach to Tumor Profiling Next Generation Sequencing Tests", "to": "Regulatory flexibility and post-market variant updates"}, {"from": "ClonoSEQ assay for the detection of lymphoid malignancies", "to": "Minimal Residual Disease (MRD) detection in lymphoid malignancies"}, {"from": "ClonoSEQ assay for the detection of lymphoid malignancies", "to": "ClonoSEQ assay design and technical features"}, {"from": "ClonoSEQ assay for the detection of lymphoid malignancies", "to": "Comparison of ClonoSEQ with flow cytometry and ASO-PCR"}, {"from": "ClonoSEQ assay for the detection of lymphoid malignancies", "to": "ClonoSEQ\u0027s clinical application in B-cell acute lymphoblastic leukemia (B-ALL)"}, {"from": "ClonoSEQ assay for the detection of lymphoid malignancies", "to": "ClonoSEQ\u0027s prognostic value in multiple myeloma (MM)"}, {"from": "ClonoSEQ assay for the detection of lymphoid malignancies", "to": "Use of ClonoSEQ in chronic lymphocytic leukemia (CLL) and mantle cell lymphoma (MCL)"}, {"from": "ClonoSEQ assay for the detection of lymphoid malignancies", "to": "Non-invasive MRD detection using circulating tumor DNA (ctDNA)"}, {"from": "ClonoSEQ assay for the detection of lymphoid malignancies", "to": "Limitations and challenges of the ClonoSEQ platform"}, {"from": "ClonoSEQ assay for the detection of lymphoid malignancies", "to": "Standardization efforts in high-throughput sequencing for MRD (e.g., EuroClonality-NGS)"}, {"from": "ClonoSEQ assay for the detection of lymphoid malignancies", "to": "Clonal evolution and its implications for MRD tracking"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Analytical validation of the clonoSEQ assay"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Limit of detection (LoD), quantitation (LoQ), and blank (LoB) for MRD"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Precision and reproducibility across input levels and cancer types"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Linearity of MRD quantification across DNA concentrations and disease burdens"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Comparison of clonoSEQ assay with multiparametric flow cytometry (mpFC)"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Sequence accuracy and error rate metrics"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Tracking of multiple receptor sequences for enhanced sensitivity"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Impact of clonotype uniqueness on MRD specificity"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Standardization and robustness of NGS-based MRD assessment"}, {"from": "Analytical evaluation of the clonoSEQ Assay for establishing measurable (minimal) residual disease in acute lymphoblastic leukemia, chronic lymphocytic leukemia, and multiple myeloma", "to": "Limitations of the assay and considerations for clinical implementation"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Comparison of MRD detection methods (NGS vs. MFC)"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Sensitivity and detection thresholds of clonoSEQ and flow cytometry"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Concordance and discordance in MRD results between assays"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Clinical implications of MRD positivity and relapse prediction"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Prognostic value of NGS-based MRD testing in adult ALL"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Correlation between MRD burden measured by NGS and MFC"}, {"from": "Comparison of next-generation sequencing and flow cytometry in detecting minimal residual disease in adult acute lymphoid leukemia: Evaluating clinical outcomes in a single-center study", "to": "Use of MRD testing to guide treatment decisions and transplant timing"}, {"from": "Minimal Residual Disease Detection in Adults with Acute Lymphoblastic Leukemia By Next Generation Sequencing: Impact on Clinical Outcomes", "to": "Sensitivity differences between NGS (clonoSEQ) and flow cytometry"}, {"from": "Minimal Residual Disease Detection in Adults with Acute Lymphoblastic Leukemia By Next Generation Sequencing: Impact on Clinical Outcomes", "to": "Discordance in MRD detection and its clinical implications"}, {"from": "Minimal Residual Disease Detection in Adults with Acute Lymphoblastic Leukemia By Next Generation Sequencing: Impact on Clinical Outcomes", "to": "MRD-guided therapeutic decisions (e.g., blinatumomab, allo-HCT)"}, {"from": "Minimal Residual Disease Detection in Adults with Acute Lymphoblastic Leukemia By Next Generation Sequencing: Impact on Clinical Outcomes", "to": "Prognostic value of clonoSEQ in predicting relapse and remission"}, {"from": "Minimal Residual Disease Detection in Adults with Acute Lymphoblastic Leukemia By Next Generation Sequencing: Impact on Clinical Outcomes", "to": "Use of MRD status post-induction to stratify patients"}, {"from": "Minimal Residual Disease Detection in Adults with Acute Lymphoblastic Leukemia By Next Generation Sequencing: Impact on Clinical Outcomes", "to": "Outcomes of Ph-like B-ALL patients based on MRD status"}, {"from": "Minimal Residual Disease Detection in Adults with Acute Lymphoblastic Leukemia By Next Generation Sequencing: Impact on Clinical Outcomes", "to": "Role of NGS in deferring intensive therapies in MRD-negative patients"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "MRD detection technologies"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "Sensitivity thresholds for MRD assays"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "Multiparameter flow cytometry (MFC)"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "Real-time quantitative PCR (RQ-PCR)"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "Next-generation sequencing (NGS)"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "MRD sampling techniques and best practices"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "MRD in acute lymphoblastic leukemia (ALL)"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "MRD in acute myeloid leukemia (AML)"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "MRD in multiple myeloma (MM)"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "MRD in chronic myeloid leukemia (CML)"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "Clinical utility and prognostic value of MRD negativity"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "FDA-cleared assays (e.g., ClonoSEQ\u00ae)"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "Use of MRD in post-transplant monitoring and treatment planning"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "Limitations and challenges in MRD interpretation"}, {"from": "Facts About Measurable Residual Disease (MRD)", "to": "Guidelines from organizations like ELN and NCCN"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "NGS vs. MFC MRD Sensitivity"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Prognostic Value of Early NGS MRD Negativity"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Risk Stratification in MFC-Negative Patients"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Timing of MRD Assessment and Clinical Outcomes"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Impact of Baseline Cytogenetics on MRD Interpretation"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "NGS MRD and AlloSCT Decision-Making"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Prognostic Utility of NGS in Non-Transplanted Patients"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "MRD Response in Lower-Intensity Chemotherapy Regimens"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Limitations of Late MRD Negativity"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Reevaluating Clinical MRD Thresholds with NGS Data"}, {"from": "Interlaboratory Analytical Validation of a Next-Generation Sequencing Strategy for Clonotypic Assessment and Minimal Residual Disease Monitoring in Multiple Myeloma", "to": "LymphoTrack NGS assay validation"}, {"from": "Interlaboratory Analytical Validation of a Next-Generation Sequencing Strategy for Clonotypic Assessment and Minimal Residual Disease Monitoring in Multiple Myeloma", "to": "Clonality detection in multiple myeloma"}, {"from": "Interlaboratory Analytical Validation of a Next-Generation Sequencing Strategy for Clonotypic Assessment and Minimal Residual Disease Monitoring in Multiple Myeloma", "to": "Minimal residual disease (MRD) monitoring"}, {"from": "Interlaboratory Analytical Validation of a Next-Generation Sequencing Strategy for Clonotypic Assessment and Minimal Residual Disease Monitoring in Multiple Myeloma", "to": "Comparison with Sanger sequencing"}, {"from": "Interlaboratory Analytical Validation of a Next-Generation Sequencing Strategy for Clonotypic Assessment and Minimal Residual Disease Monitoring in Multiple Myeloma", "to": "Comparison with next-generation flow cytometry (NGF)"}, {"from": "Interlaboratory Analytical Validation of a Next-Generation Sequencing Strategy for Clonotypic Assessment and Minimal Residual Disease Monitoring in Multiple Myeloma", "to": "Interlaboratory reproducibility and concordance"}, {"from": "Interlaboratory Analytical Validation of a Next-Generation Sequencing Strategy for Clonotypic Assessment and Minimal Residual Disease Monitoring in Multiple Myeloma", "to": "Detection limits and quantification precision (LOD and LOQ)"}, {"from": "Interlaboratory Analytical Validation of a Next-Generation Sequencing Strategy for Clonotypic Assessment and Minimal Residual Disease Monitoring in Multiple Myeloma", "to": "Technical challenges in DNA amplification and spike-in normalization"}, {"from": "Interlaboratory Analytical Validation of a Next-Generation Sequencing Strategy for Clonotypic Assessment and Minimal Residual Disease Monitoring in Multiple Myeloma", "to": "Prognostic value of MRD detection and relapse prediction"}, {"from": "Interlaboratory Analytical Validation of a Next-Generation Sequencing Strategy for Clonotypic Assessment and Minimal Residual Disease Monitoring in Multiple Myeloma", "to": "Standardization of NGS methods for clinical implementation"}, {"from": "How I investigate minimal residual disease in acute lymphoblastic leukemia", "to": "Minimal Residual Disease (MRD) as a prognostic marker in Acute Lymphoblastic Leukemia (ALL)"}, {"from": "How I investigate minimal residual disease in acute lymphoblastic leukemia", "to": "Multiparametric Flow Cytometry (MFC) for MRD detection"}, {"from": "How I investigate minimal residual disease in acute lymphoblastic leukemia", "to": "Molecular assays for MRD detection (qPCR, digital PCR)"}, {"from": "How I investigate minimal residual disease in acute lymphoblastic leukemia", "to": "High-throughput sequencing (HTS/NGS) approaches in MRD monitoring"}, {"from": "How I investigate minimal residual disease in acute lymphoblastic leukemia", "to": "Comparison of MRD methodologies: sensitivity, specificity, and clinical utility"}, {"from": "How I investigate minimal residual disease in acute lymphoblastic leukemia", "to": "Immunophenotypic challenges post immunotherapy (e.g., CD19, CD20 loss)"}, {"from": "How I investigate minimal residual disease in acute lymphoblastic leukemia", "to": "Use of peripheral blood vs. bone marrow for MRD evaluation"}, {"from": "How I investigate minimal residual disease in acute lymphoblastic leukemia", "to": "MRD marker panels and gating strategies for T-ALL and B-ALL"}, {"from": "How I investigate minimal residual disease in acute lymphoblastic leukemia", "to": "Standardization and validation of MRD assays across laboratories"}, {"from": "How I investigate minimal residual disease in acute lymphoblastic leukemia", "to": "FDA approval and clinical implementation of NGS-based MRD assays (e.g., ClonoSEQ)"}, {"from": " A Novel Single-Tube Next Generation Sequencing Assay for B-Cell Receptor Clonality Testing", "to": "B-cell receptor (BCR) clonality detection"}, {"from": " A Novel Single-Tube Next Generation Sequencing Assay for B-Cell Receptor Clonality Testing", "to": "Next-generation sequencing (NGS) assay development"}, {"from": " A Novel Single-Tube Next Generation Sequencing Assay for B-Cell Receptor Clonality Testing", "to": "Multiple myeloma (MM) diagnostics"}, {"from": " A Novel Single-Tube Next Generation Sequencing Assay for B-Cell Receptor Clonality Testing", "to": "Somatic hypermutation (SHM) analysis"}, {"from": " A Novel Single-Tube Next Generation Sequencing Assay for B-Cell Receptor Clonality Testing", "to": "Intraclonal diversity and clonal evolution"}, {"from": " A Novel Single-Tube Next Generation Sequencing Assay for B-Cell Receptor Clonality Testing", "to": "IGHV gene usage bias (e.g., IGHV3-11, IGHV4-34)"}, {"from": " A Novel Single-Tube Next Generation Sequencing Assay for B-Cell Receptor Clonality Testing", "to": "Assay sensitivity and specificity"}, {"from": " A Novel Single-Tube Next Generation Sequencing Assay for B-Cell Receptor Clonality Testing", "to": "Single-tube multiplex PCR methods"}, {"from": " A Novel Single-Tube Next Generation Sequencing Assay for B-Cell Receptor Clonality Testing", "to": "Detection limits in clonality testing"}, {"from": " A Novel Single-Tube Next Generation Sequencing Assay for B-Cell Receptor Clonality Testing", "to": "Reproducibility of NGS-based clonotyping"}, {"from": " LymphoTrack Is Equally Sensitive as PCR GeneScan and Sanger Sequencing for Detection of Clonal Rearrangements in ALL Patients", "to": "Clonal rearrangement detection in acute lymphoblastic leukemia (ALL)"}, {"from": " LymphoTrack Is Equally Sensitive as PCR GeneScan and Sanger Sequencing for Detection of Clonal Rearrangements in ALL Patients", "to": "Comparison of LymphoTrack with PCR GeneScan and Sanger sequencing"}, {"from": " LymphoTrack Is Equally Sensitive as PCR GeneScan and Sanger Sequencing for Detection of Clonal Rearrangements in ALL Patients", "to": "Sensitivity and specificity of next-generation sequencing (NGS) assays"}, {"from": " LymphoTrack Is Equally Sensitive as PCR GeneScan and Sanger Sequencing for Detection of Clonal Rearrangements in ALL Patients", "to": "Validation of IGH, IGK, TRG, and TCRB rearrangements"}, {"from": " LymphoTrack Is Equally Sensitive as PCR GeneScan and Sanger Sequencing for Detection of Clonal Rearrangements in ALL Patients", "to": "Correlation between %Total reads (NGS) and RFU (PCR GeneScan)"}, {"from": " LymphoTrack Is Equally Sensitive as PCR GeneScan and Sanger Sequencing for Detection of Clonal Rearrangements in ALL Patients", "to": "Identification of ambiguous and polyclonal rearrangements"}, {"from": " LymphoTrack Is Equally Sensitive as PCR GeneScan and Sanger Sequencing for Detection of Clonal Rearrangements in ALL Patients", "to": "Limitations of current LymphoTrack assays for incomplete IGH and TCRD"}, {"from": " LymphoTrack Is Equally Sensitive as PCR GeneScan and Sanger Sequencing for Detection of Clonal Rearrangements in ALL Patients", "to": "Advantages of NGS over traditional methods in turnaround time and resolution"}, {"from": " LymphoTrack Is Equally Sensitive as PCR GeneScan and Sanger Sequencing for Detection of Clonal Rearrangements in ALL Patients", "to": "Implementation considerations for LymphoTrack in clinical diagnostics"}, {"from": " LymphoTrack Is Equally Sensitive as PCR GeneScan and Sanger Sequencing for Detection of Clonal Rearrangements in ALL Patients", "to": "Utility of ASO design for MRD monitoring in ALL"}, {"from": "Multiparameter Flow Cytometry and ClonoSEQ Correlation to Evaluate Precursor B-Lymphoblastic Leukemia Measurable Residual Disease", "to": "Measurable residual disease (MRD) monitoring in precursor B-ALL"}, {"from": "Multiparameter Flow Cytometry and ClonoSEQ Correlation to Evaluate Precursor B-Lymphoblastic Leukemia Measurable Residual Disease", "to": "Comparison of multiparameter flow cytometry (MFC) and ClonoSEQ"}, {"from": "Multiparameter Flow Cytometry and ClonoSEQ Correlation to Evaluate Precursor B-Lymphoblastic Leukemia Measurable Residual Disease", "to": "Sensitivity and specificity of NGS-based MRD detection"}, {"from": "Multiparameter Flow Cytometry and ClonoSEQ Correlation to Evaluate Precursor B-Lymphoblastic Leukemia Measurable Residual Disease", "to": "Discordance between MFC and ClonoSEQ results"}, {"from": "Multiparameter Flow Cytometry and ClonoSEQ Correlation to Evaluate Precursor B-Lymphoblastic Leukemia Measurable Residual Disease", "to": "Impact of blinatumomab immunotherapy on MRD detection"}, {"from": "Multiparameter Flow Cytometry and ClonoSEQ Correlation to Evaluate Precursor B-Lymphoblastic Leukemia Measurable Residual Disease", "to": "Clonal evolution and emergence of new dominant sequences"}, {"from": "Multiparameter Flow Cytometry and ClonoSEQ Correlation to Evaluate Precursor B-Lymphoblastic Leukemia Measurable Residual Disease", "to": "Interpretation challenges in low MRD settings"}, {"from": "Multiparameter Flow Cytometry and ClonoSEQ Correlation to Evaluate Precursor B-Lymphoblastic Leukemia Measurable Residual Disease", "to": "Phenotypic markers in B-ALL MRD evaluation (CD10, CD38, CD19)"}, {"from": "Multiparameter Flow Cytometry and ClonoSEQ Correlation to Evaluate Precursor B-Lymphoblastic Leukemia Measurable Residual Disease", "to": "Diagnostic limitations of ClonoSEQ requiring baseline samples"}, {"from": "Multiparameter Flow Cytometry and ClonoSEQ Correlation to Evaluate Precursor B-Lymphoblastic Leukemia Measurable Residual Disease", "to": "Complementary use of MFC and NGS for comprehensive MRD assessment"}, {"from": "Next-generation sequencing\u2013based MRD in adults with ALL undergoing hematopoietic cell transplantation", "to": "Next-generation sequencing (NGS) for MRD detection"}, {"from": "Next-generation sequencing\u2013based MRD in adults with ALL undergoing hematopoietic cell transplantation", "to": "Prognostic significance of pre- and post-HCT MRD in ALL"}, {"from": "Next-generation sequencing\u2013based MRD in adults with ALL undergoing hematopoietic cell transplantation", "to": "Risk stratification based on MRD levels (\u003c10\u207b\u2074, \u226510\u207b\u00b3)"}, {"from": "Next-generation sequencing\u2013based MRD in adults with ALL undergoing hematopoietic cell transplantation", "to": "IgH vs non-IgH clonotypes in relapse prediction"}, {"from": "Next-generation sequencing\u2013based MRD in adults with ALL undergoing hematopoietic cell transplantation", "to": "Peripheral blood vs bone marrow MRD monitoring"}, {"from": "Next-generation sequencing\u2013based MRD in adults with ALL undergoing hematopoietic cell transplantation", "to": "Post-HCT relapse and leukemia-specific mortality"}, {"from": "Next-generation sequencing\u2013based MRD in adults with ALL undergoing hematopoietic cell transplantation", "to": "Clinical utility of MRD in treatment and monitoring decisions"}, {"from": "Next-generation sequencing\u2013based MRD in adults with ALL undergoing hematopoietic cell transplantation", "to": "Implications for MRD-guided therapeutic strategies before HCT"}, {"from": "Next-generation sequencing for MRD monitoring in B-lineage malignancies: from bench to bedside", "to": "Minimal Residual Disease (MRD) monitoring in hematologic malignancies"}, {"from": "Next-generation sequencing for MRD monitoring in B-lineage malignancies: from bench to bedside", "to": "Next-Generation Sequencing (NGS)-based clonality assays"}, {"from": "Next-generation sequencing for MRD monitoring in B-lineage malignancies: from bench to bedside", "to": "V(D)J recombination and immunoglobulin gene rearrangements"}, {"from": "Next-generation sequencing for MRD monitoring in B-lineage malignancies: from bench to bedside", "to": "Disease-specific IG rearrangement patterns in B-lineage cancers (e.g., ALL, CLL, MCL, DLBCL, FL, MM)"}, {"from": "Next-generation sequencing for MRD monitoring in B-lineage malignancies: from bench to bedside", "to": "Clonal evolution and its clinical implications"}, {"from": "Next-generation sequencing for MRD monitoring in B-lineage malignancies: from bench to bedside", "to": "Prognostic significance of IGHV mutation status"}, {"from": "Next-generation sequencing for MRD monitoring in B-lineage malignancies: from bench to bedside", "to": "Technical methodology and sensitivity thresholds for MRD detection"}, {"from": "Next-generation sequencing for MRD monitoring in B-lineage malignancies: from bench to bedside", "to": "FDA-approved NGS-based MRD platforms (e.g., ClonoSeq)"}, {"from": "Next-generation sequencing for MRD monitoring in B-lineage malignancies: from bench to bedside", "to": "Limitations and barriers to clinical adoption of NGS MRD monitoring"}, {"from": "Next-generation sequencing for MRD monitoring in B-lineage malignancies: from bench to bedside", "to": "Integration of MRD data into risk stratification and treatment decisions"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Measurable Residual Disease (MRD) in Acute Lymphoblastic Leukemia (ALL)"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Comparison of MRD detection methods: Flow Cytometry (MFC) vs. Next-Generation Sequencing (NGS)"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Prognostic significance of MRD negativity at various time points"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Clinical utility of high-sensitivity NGS MRD assays"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Risk stratification and relapse prediction in ALL"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Impact of treatment regimens (hyper-CVAD vs. mini\u2013hyper-CVD with inotuzumab)"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Implications for allogeneic stem cell transplantation (alloSCT) decisions"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Limitations of conventional MRD thresholds and timing"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "Treatment de-escalation potential based on early deep remission"}, {"from": "High-sensitivity next-generation sequencing MRD assessment in ALL identifies patients at very low risk of relapse", "to": "MRD as a biomarker for therapy guidance in frontline ALL management"}, {"from": "Perspective: Sensitive Detection of Residual Lymphoproliferative Disease by NGS and Clonal Rearrangements\u2014How Low Can You Go?", "to": "Sensitivity thresholds for MRD detection using NGS"}, {"from": "Perspective: Sensitive Detection of Residual Lymphoproliferative Disease by NGS and Clonal Rearrangements\u2014How Low Can You Go?", "to": "Clonal gene rearrangements as MRD targets (IGH, TCR)"}, {"from": "Perspective: Sensitive Detection of Residual Lymphoproliferative Disease by NGS and Clonal Rearrangements\u2014How Low Can You Go?", "to": "Technical limitations: DNA input, sequencing depth, and PCR efficiency"}, {"from": "Perspective: Sensitive Detection of Residual Lymphoproliferative Disease by NGS and Clonal Rearrangements\u2014How Low Can You Go?", "to": "Poisson sampling and stochastic errors in low-input scenarios"}, {"from": "Perspective: Sensitive Detection of Residual Lymphoproliferative Disease by NGS and Clonal Rearrangements\u2014How Low Can You Go?", "to": "Standardization and reproducibility challenges in NGS MRD assays"}, {"from": "Perspective: Sensitive Detection of Residual Lymphoproliferative Disease by NGS and Clonal Rearrangements\u2014How Low Can You Go?", "to": "Comparison of NGS with qPCR, ASO-qPCR, and flow cytometry"}, {"from": "Perspective: Sensitive Detection of Residual Lymphoproliferative Disease by NGS and Clonal Rearrangements\u2014How Low Can You Go?", "to": "Sample type considerations (bone marrow, peripheral blood, ctDNA, RNA)"}, {"from": "Perspective: Sensitive Detection of Residual Lymphoproliferative Disease by NGS and Clonal Rearrangements\u2014How Low Can You Go?", "to": "Impact of biological variables (cell sorting, clonality, rearrangement completeness)"}, {"from": "Perspective: Sensitive Detection of Residual Lymphoproliferative Disease by NGS and Clonal Rearrangements\u2014How Low Can You Go?", "to": "Use of commercial platforms (LymphoTrack, clonoSEQ)"}, {"from": "Perspective: Sensitive Detection of Residual Lymphoproliferative Disease by NGS and Clonal Rearrangements\u2014How Low Can You Go?", "to": "Recommendations for assay validation and transparent reporting"}, {"from": "Prospective evaluation of minimal residual disease in the phase II FORTE trial: a head-to-head comparison between multiparameter flow cytometry and next-generation sequencing", "to": "Head-to-head comparison of MFC and NGS for MRD detection"}, {"from": "Prospective evaluation of minimal residual disease in the phase II FORTE trial: a head-to-head comparison between multiparameter flow cytometry and next-generation sequencing", "to": "Concordance and discordance rates between MFC and NGS at 10\u207b\u2075 and 10\u207b\u2076 sensitivities"}, {"from": "Prospective evaluation of minimal residual disease in the phase II FORTE trial: a head-to-head comparison between multiparameter flow cytometry and next-generation sequencing", "to": "Prognostic impact of MRD status on progression-free and overall survival"}, {"from": "Prospective evaluation of minimal residual disease in the phase II FORTE trial: a head-to-head comparison between multiparameter flow cytometry and next-generation sequencing", "to": "Sustained MRD negativity (1-year and 2-year) as a strong predictor of long-term outcomes"}, {"from": "Prospective evaluation of minimal residual disease in the phase II FORTE trial: a head-to-head comparison between multiparameter flow cytometry and next-generation sequencing", "to": "Effect of maintenance therapy (KR vs R) on MRD conversion rates"}, {"from": "Prospective evaluation of minimal residual disease in the phase II FORTE trial: a head-to-head comparison between multiparameter flow cytometry and next-generation sequencing", "to": "Implications of discordant MRD results and sample handling variables"}, {"from": "Prospective evaluation of minimal residual disease in the phase II FORTE trial: a head-to-head comparison between multiparameter flow cytometry and next-generation sequencing", "to": "Stratification of patients by cytogenetic risk, ISS stage, and treatment arms"}, {"from": "Prospective evaluation of minimal residual disease in the phase II FORTE trial: a head-to-head comparison between multiparameter flow cytometry and next-generation sequencing", "to": "Use of MRD status to guide therapy deintensification or discontinuation"}, {"from": "Prospective evaluation of minimal residual disease in the phase II FORTE trial: a head-to-head comparison between multiparameter flow cytometry and next-generation sequencing", "to": "Feasibility and limitations of achieving 10\u207b\u2076 sensitivity with current methods"}, {"from": "Prospective evaluation of minimal residual disease in the phase II FORTE trial: a head-to-head comparison between multiparameter flow cytometry and next-generation sequencing", "to": "Role of MRD testing in clinical trial design and regulatory approval"}, {"from": " Real-life sensitivity of flow cytometry minimal residual disease assessment for plasma cell neoplasms", "to": "Real-world sensitivity of flow cytometry (FC) for MRD detection in plasma cell neoplasms"}, {"from": " Real-life sensitivity of flow cytometry minimal residual disease assessment for plasma cell neoplasms", "to": "Performance of EuroFlow next-generation flow (NGF) assay"}, {"from": " Real-life sensitivity of flow cytometry minimal residual disease assessment for plasma cell neoplasms", "to": "Use of prescreening with lower-sensitivity PCPRO assay"}, {"from": " Real-life sensitivity of flow cytometry minimal residual disease assessment for plasma cell neoplasms", "to": "Comparison of internal vs. external sample characteristics and outcomes"}, {"from": " Real-life sensitivity of flow cytometry minimal residual disease assessment for plasma cell neoplasms", "to": "Event acquisition volume and its impact on assay sensitivity"}, {"from": " Real-life sensitivity of flow cytometry minimal residual disease assessment for plasma cell neoplasms", "to": "Rates of MRD positivity, negativity, and indeterminacy"}, {"from": " Real-life sensitivity of flow cytometry minimal residual disease assessment for plasma cell neoplasms", "to": "Viability of FC MRD testing with delayed sample processing (up to 72 hours)"}, {"from": " Real-life sensitivity of flow cytometry minimal residual disease assessment for plasma cell neoplasms", "to": "Differences in clone sizes between sample sources"}, {"from": " Real-life sensitivity of flow cytometry minimal residual disease assessment for plasma cell neoplasms", "to": "Lack of significant hemodilution differences between sample groups"}, {"from": " Real-life sensitivity of flow cytometry minimal residual disease assessment for plasma cell neoplasms", "to": "Operational implications for integrating FC MRD testing in routine MM monitoring workflows"}, {"from": "Evaluation of next-generation sequencing versus next-generation flow cytometry for minimal-residual-disease detection in Chinese patients with multiple myeloma", "to": "Minimal Residual Disease (MRD) Detection"}, {"from": "Evaluation of next-generation sequencing versus next-generation flow cytometry for minimal-residual-disease detection in Chinese patients with multiple myeloma", "to": "Next-Generation Sequencing (NGS)"}, {"from": "Evaluation of next-generation sequencing versus next-generation flow cytometry for minimal-residual-disease detection in Chinese patients with multiple myeloma", "to": "Next-Generation Flow Cytometry (NGF)"}, {"from": "Evaluation of next-generation sequencing versus next-generation flow cytometry for minimal-residual-disease detection in Chinese patients with multiple myeloma", "to": "Clonal IGH Rearrangement"}, {"from": "Evaluation of next-generation sequencing versus next-generation flow cytometry for minimal-residual-disease detection in Chinese patients with multiple myeloma", "to": "Sensitivity and Reproducibility of MRD Methods"}, {"from": "Evaluation of next-generation sequencing versus next-generation flow cytometry for minimal-residual-disease detection in Chinese patients with multiple myeloma", "to": "Comparison of NGS vs. NGF"}, {"from": "Evaluation of next-generation sequencing versus next-generation flow cytometry for minimal-residual-disease detection in Chinese patients with multiple myeloma", "to": "Autologous Stem Cell Transplantation (ASCT)"}, {"from": "Evaluation of next-generation sequencing versus next-generation flow cytometry for minimal-residual-disease detection in Chinese patients with multiple myeloma", "to": "Multiple Myeloma (MM) Monitoring"}, {"from": "Evaluation of next-generation sequencing versus next-generation flow cytometry for minimal-residual-disease detection in Chinese patients with multiple myeloma", "to": "Diagnostic Biomarkers"}, {"from": "Evaluation of next-generation sequencing versus next-generation flow cytometry for minimal-residual-disease detection in Chinese patients with multiple myeloma", "to": "Clinical Utility of NGS in Chinese MM Patients"}, {"from": "Role of flow cytometric measurable residual disease assessment in multiple myeloma", "to": "Measurable Residual Disease (MRD) as a Prognostic Marker"}, {"from": "Role of flow cytometric measurable residual disease assessment in multiple myeloma", "to": "Flow Cytometry and Next-Generation Flow (NGF) Cytometry"}, {"from": "Role of flow cytometric measurable residual disease assessment in multiple myeloma", "to": "Next-Generation Sequencing (NGS) for MRD Detection"}, {"from": "Role of flow cytometric measurable residual disease assessment in multiple myeloma", "to": "Sensitivity, Specificity, and Standardization of MRD Methods"}, {"from": "Role of flow cytometric measurable residual disease assessment in multiple myeloma", "to": "Bone Marrow Sample Quality and Pre-analytical Challenges"}, {"from": "Role of flow cytometric measurable residual disease assessment in multiple myeloma", "to": "Integration of PET-CT Imaging in MRD Assessment"}, {"from": "Role of flow cytometric measurable residual disease assessment in multiple myeloma", "to": "Clinical Impact of Sustained MRD Negativity"}, {"from": "Role of flow cytometric measurable residual disease assessment in multiple myeloma", "to": "MRD-Guided Therapeutic Strategies and Clinical Trial Design"}, {"from": "Role of flow cytometric measurable residual disease assessment in multiple myeloma", "to": "Use of Peripheral Blood Techniques and Liquid Biopsy"}, {"from": "Minimal residual disease detection by next-generation sequencing of different immunoglobulin gene rearrangements in pediatric B-ALL", "to": "Next-generation sequencing (NGS) for MRD detection"}, {"from": "Minimal residual disease detection by next-generation sequencing of different immunoglobulin gene rearrangements in pediatric B-ALL", "to": "IGH, IGK, and IGL gene rearrangements in pediatric B-ALL"}, {"from": "Minimal residual disease detection by next-generation sequencing of different immunoglobulin gene rearrangements in pediatric B-ALL", "to": "Clonal clearance dynamics during induction and consolidation"}, {"from": "Minimal residual disease detection by next-generation sequencing of different immunoglobulin gene rearrangements in pediatric B-ALL", "to": "Prognostic significance of MRD levels at EOI and EOC"}, {"from": "Minimal residual disease detection by next-generation sequencing of different immunoglobulin gene rearrangements in pediatric B-ALL", "to": "Comparison of NGS vs. multiparameter flow cytometry (MFC)"}, {"from": "Minimal residual disease detection by next-generation sequencing of different immunoglobulin gene rearrangements in pediatric B-ALL", "to": "Value of integrating IGK/IGL with IGH in MRD assessment"}, {"from": "Minimal residual disease detection by next-generation sequencing of different immunoglobulin gene rearrangements in pediatric B-ALL", "to": "Risk stratification based on MRD thresholds"}, {"from": "Minimal residual disease detection by next-generation sequencing of different immunoglobulin gene rearrangements in pediatric B-ALL", "to": "Clinical implications for relapse prediction and immunotherapy"}, {"from": " Validation and interpretation of IGH and TCR clonality testing by Ion Torrent S5 NGS for diagnosis and disease monitoring in B and T cell cancers", "to": "Validation of NGS-based IGH and TCR clonality assays"}, {"from": " Validation and interpretation of IGH and TCR clonality testing by Ion Torrent S5 NGS for diagnosis and disease monitoring in B and T cell cancers", "to": "Comparison of NGS (Ion Torrent S5) with PCR fragment analysis"}, {"from": " Validation and interpretation of IGH and TCR clonality testing by Ion Torrent S5 NGS for diagnosis and disease monitoring in B and T cell cancers", "to": "Clinical utility of NGS in detecting clonal sequences in B- and T-cell malignancies"}, {"from": " Validation and interpretation of IGH and TCR clonality testing by Ion Torrent S5 NGS for diagnosis and disease monitoring in B and T cell cancers", "to": "Diagnostic performance and reproducibility of LymphoTrack assays"}, {"from": " Validation and interpretation of IGH and TCR clonality testing by Ion Torrent S5 NGS for diagnosis and disease monitoring in B and T cell cancers", "to": "Tracking clonal populations over time and across biopsies using NGS"}, {"from": " Validation and interpretation of IGH and TCR clonality testing by Ion Torrent S5 NGS for diagnosis and disease monitoring in B and T cell cancers", "to": "Interpretation guidelines for NGS clonality results (sensitivity vs. specificity)"}, {"from": " Validation and interpretation of IGH and TCR clonality testing by Ion Torrent S5 NGS for diagnosis and disease monitoring in B and T cell cancers", "to": "Improved detection of IGH somatic hypermutation in CLL using NGS"}, {"from": " Validation and interpretation of IGH and TCR clonality testing by Ion Torrent S5 NGS for diagnosis and disease monitoring in B and T cell cancers", "to": "Concordance and discrepancies between NGS platforms (e.g., LymphoTrack vs. clonoSeq)"}, {"from": " Validation and interpretation of IGH and TCR clonality testing by Ion Torrent S5 NGS for diagnosis and disease monitoring in B and T cell cancers", "to": "Integration of NGS results with clinical and histopathological diagnoses"}, {"from": " Validation and interpretation of IGH and TCR clonality testing by Ion Torrent S5 NGS for diagnosis and disease monitoring in B and T cell cancers", "to": "Implications for routine clinical use of NGS in hematologic malignancy diagnostics"}, {"from": "Analytical and clinical validation of a novel in-house deep-sequencing method for minimal residual disease monitoring in a phase II trial for multiple myeloma", "to": "In-house NGS method for MRD detection"}, {"from": "Analytical and clinical validation of a novel in-house deep-sequencing method for minimal residual disease monitoring in a phase II trial for multiple myeloma", "to": "Sensitivity and specificity in multiple myeloma"}, {"from": "Analytical and clinical validation of a novel in-house deep-sequencing method for minimal residual disease monitoring in a phase II trial for multiple myeloma", "to": "Comparison with 8-color flow cytometry"}, {"from": "Analytical and clinical validation of a novel in-house deep-sequencing method for minimal residual disease monitoring in a phase II trial for multiple myeloma", "to": "Prognostic value of MRD negativity"}, {"from": "Analytical and clinical validation of a novel in-house deep-sequencing method for minimal residual disease monitoring in a phase II trial for multiple myeloma", "to": "Impact of treatment duration on response"}, {"from": "Analytical and clinical validation of a novel in-house deep-sequencing method for minimal residual disease monitoring in a phase II trial for multiple myeloma", "to": "Method reproducibility and platform-independence"}, {"from": "Analytical and clinical validation of a novel in-house deep-sequencing method for minimal residual disease monitoring in a phase II trial for multiple myeloma", "to": "Concordance between NGS and MFC"}, {"from": "Analytical and clinical validation of a novel in-house deep-sequencing method for minimal residual disease monitoring in a phase II trial for multiple myeloma", "to": "Clinical lab implementation feasibility"}, {"from": "Analytical and clinical validation of a novel in-house deep-sequencing method for minimal residual disease monitoring in a phase II trial for multiple myeloma", "to": "Alignment with IMWG MRD criteria"}, {"from": "Analytical and clinical validation of a novel in-house deep-sequencing method for minimal residual disease monitoring in a phase II trial for multiple myeloma", "to": "Automation and standardization potential"}]);

                  nodeColors = {};
                  allNodes = nodes.get({ returnType: "Object" });
                  for (nodeId in allNodes) {
                    nodeColors[nodeId] = allNodes[nodeId].color;
                  }
                  allEdges = edges.get({ returnType: "Object" });
                  // adding nodes and edges to the graph
                  data = {nodes: nodes, edges: edges};

                  var options = {"interaction": {"hover": false, "tooltipDelay": 999999}};

                  


                  

                  network = new vis.Network(container, data, options);

                  

                  

                  
                  // make a custom popup
                      var popup = document.createElement("div");
                      popup.className = 'popup';
                      popupTimeout = null;
                      popup.addEventListener('mouseover', function () {
                          console.log(popup)
                          if (popupTimeout !== null) {
                              clearTimeout(popupTimeout);
                              popupTimeout = null;
                          }
                      });
                      popup.addEventListener('mouseout', function () {
                          if (popupTimeout === null) {
                              hidePopup();
                          }
                      });
                      container.appendChild(popup);


                      // use the popup event to show
                      network.on("showPopup", function (params) {
                          showPopup(params);
                      });

                      // use the hide event to hide it
                      network.on("hidePopup", function (params) {
                          hidePopup();
                      });

                      // hiding the popup through css
                      function hidePopup() {
                          popupTimeout = setTimeout(function () { popup.style.display = 'none'; }, 500);
                      }

                      // showing the popup
                      function showPopup(nodeId) {
                          // get the data from the vis.DataSet
                          var nodeData = nodes.get([nodeId]);
                          popup.innerHTML = nodeData[0].title;

                          // get the position of the node
                          var posCanvas = network.getPositions([nodeId])[nodeId];

                          // get the bounding box of the node
                          var boundingBox = network.getBoundingBox(nodeId);

                          //position tooltip:
                          posCanvas.x = posCanvas.x + 0.5 * (boundingBox.right - boundingBox.left);

                          // convert coordinates to the DOM space
                          var posDOM = network.canvasToDOM(posCanvas);

                          // Give it an offset
                          posDOM.x += 10;
                          posDOM.y -= 20;

                          // show and place the tooltip.
                          popup.style.display = 'block';
                          popup.style.top = posDOM.y + 'px';
                          popup.style.left = posDOM.x + 'px';
                      }
                  


                  
                      network.on("stabilizationProgress", function(params) {
                          document.getElementById('loadingBar').removeAttribute("style");
                          var maxWidth = 496;
                          var minWidth = 20;
                          var widthFactor = params.iterations/params.total;
                          var width = Math.max(minWidth,maxWidth * widthFactor);
                          document.getElementById('bar').style.width = width + 'px';
                          document.getElementById('text').innerHTML = Math.round(widthFactor*100) + '%';
                      });
                      network.once("stabilizationIterationsDone", function() {
                          document.getElementById('text').innerHTML = '100%';
                          document.getElementById('bar').style.width = '496px';
                          document.getElementById('loadingBar').style.opacity = 0;
                          // really clean the dom element
                          setTimeout(function () {document.getElementById('loadingBar').style.display = 'none';}, 500);
                      });
                  

                  return network;

              }
              drawGraph();
        </script>
    
<script type="text/javascript">
    network.on("click", function (params) {
        if (params.nodes.length > 0) {
            var nodeId = params.nodes[0];
            var node = nodes.get(nodeId);
            if (node && node.title) {
                var tooltipDiv = document.createElement('div');
                tooltipDiv.innerHTML = `<button style="float:right; margin-bottom:5px;" onclick="this.parentElement.remove();">&#x2715;</button>` + node.title;
                tooltipDiv.style.position = 'fixed';
                tooltipDiv.style.zIndex = '9999';
                tooltipDiv.style.top = '100px';
                tooltipDiv.style.left = '100px';
                tooltipDiv.style.width = '750px';
                tooltipDiv.style.background = '#fff';
                tooltipDiv.style.maxHeight = '500px';
                tooltipDiv.style.overflowY = 'auto';
                tooltipDiv.style.fontSize = '13px';
                tooltipDiv.style.border = '1px solid #aaa';
                tooltipDiv.style.padding = '10px';
                tooltipDiv.style.boxShadow = '2px 2px 12px rgba(0,0,0,0.2)';
                tooltipDiv.className = 'pyvis-tooltip';
                document.body.appendChild(tooltipDiv);
            }
        }
    });
</script>

</body>
</html>